---
layout: page
title: People
permalink: /people/
---

## Chris Wallace {% render_orcid 0000-0001-9755-1703 %} {% render_github chr1swallace %} ##


<div style="float:left;margin:0 45px 0 0;" markdown="1">
![portait]({{ "/assets/pic_200.jpg" }})
</div>
I am a statistician who works with biological datasets to understand
the mechanisms underlying human immune-mediated diseases and identify possible
treatments.  
<!-- Such work is only possible with close collaboration with -->
<!-- biologists, and I maintain close collaborations with moved to the [Cambridge -->
<!-- Institute for Medical Research](http://www.cimr.cam.ac.uk) in 2009 to pursue these -->
<!-- collaborations, most closely with the [Diabetes and Inflammation -->
<!-- Laboratory](http://www-gene.cimr.cam.ac.uk).   -->
During
the GWAS era, we, and others, identified many genetic polymorphisms
that alter risk of human immune mediated diseases such as type 1 diabetes and
rheumatoid arthritis.  I now follow up this research in three directions:
1. jointly analysing multiple immune-mediated diseases/traits to borrow information between them and understand shared and distinct components of risk
2. identifying the cell specific mechanisms through which these variants affect disease risk
3. understanding how the immune system is dysregulated in disease, and how this may be modulated

Statistically, my current work is focused on variable selection, horizontal integration, clustering, Bayesian model averaging and empirical Bayes false discovery rates, and matrix factorisation, applied to integrated analysis of multiple related genetic and genomic datasets. How we do research is also important, and I am focused on valuing the people who do research and developing better open science practices.

<!-- My aim for my group is that, by integrating detailed biological understanding with principled statistical inference we will make a meaningful contribution to the understanding of immune mediated diseases. -->

<!-- In Feburary 2016 I moved to  as a PI with the aim of establishing new collaborations. --> I am funded by the [Wellcome Trust](http://www.wellcome.ac.uk/) as a Senior Research Fellow, Director of Research and PI in the  [Department of Medicine](http://www.med.cam.ac.uk/) (University of Cambridge) and hold a Programme Leader  position  at the [MRC Biostatistics Unit](http://www.mrc-bsu.cam.ac.uk/).  I am a member of the MRC funded stratified medicine programme [CLUSTER](https://www.clusterconsortium.org.uk/), working with partners at GOSH, QMUL, Manchester, and Liverpool on childhood arthritis.

Since 2022, I have also held a part time position in GSK, which allows me to learn how genomics research can be used to impact decisions in target discovery, and perhaps make some of that impact myself.

<!-- *Previously*  -->
<!-- - 2009-2016 :: Wellcome Trust Career Development Fellow in the Diabetes and Inflammation Laboratory, and head of the DIL stats group from 2012 -->
<!-- - 2003-2008 :: Research Fellow working with	  the [BRIGHT](http://www.brightstudy.ac.uk/) study into	  the genetics of hypertension. -->
<!-- - 1999-2003 :: PhD: [Genetic Susceptibility to leprosy: methodological issues in a linkage  analysis of extended pedigrees from Karonga District, Malawi](http://figshare.com/articles/Human_genetic_susceptibility_to_leprosy_methodological_issues_in_a_linkage_analysis_of_extended_pedigrees_from_Karonga_district_Malawi/963086),	  [LSHTM](http://www.lshtm.ac.uk), part of the	  [Karonga Prevention Study](http://www.lshtm.ac.uk/eph/ide/research/kps).  -->
<!-- - 1997-1999 :: Statistical programmer, [Imperial](http://www.imperial.ac.uk). -->

<p>&nbsp;</p>
-------------------------------------------------------------------------------
<p>&nbsp;</p>

## Current members ##

### Tobi Alegbe ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/tobi.jpeg" }})
</div>

Tobi is a Mathematical Genomics and Medicine PhD student based at the Wellcome Sanger institute. He is jointly supervised by [Carl Anderson](http://andersonlab.info/) and Chris. Tobi 's main interest lies in the overlap of complex disease and single-cell genomics. His PhD project involves using whole genome sequencing and single-cell RNA sequencing of large cohorts to better understand the variants, pathways and cell-types involved in inflammatory bowel disease.

<div style="clear:both"></div>

### Jennifer Asimit {% render_orcid 0000-0002-4857-2249 %} ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/Jennifer_small.jpg" }})
</div>
[Jenn Asimit](https://www.mrc-bsu.cam.ac.uk/people/in-alphabetical-order/a-to-g/jennifer-asimit/) is a senior investigator statistician with an interest in statistical methodology development for the analysis of multiple traits, as well as fine-mapping and rare variant association approaches. She recently held a MRC Methodology Research fellowship at the Sanger institute and has developed several methods and software for rare variant association analyses and the overlap analysis of traits. She now holds an MRC career development fellowship at the BSU where she is building her own group.

<div style="clear:both"></div>

### Émeline Favreau {% render_orcid 0000-0002-0713-500X %} ###
<div style="float:left;margin:0 5px 0 0;">
  <p><img src="/assets/emeline150.jpg" alt="portait of smiling ginger woman with pink top" /></p>
</div>
[Émeline](https://github.com/EmelineFavreau) is a Bioinformatician working on the [INTREPID project](https://www.intrepidproject.info/), providing pipelines and data handling from the lab to the cluster, as well as reports to clinicians. Her previous research focused on evolutionary biology, comparing genomes and transcriptomes of social insects.

<div style="clear:both"></div>

### Wei-Yu Lin {% render_orcid 0000-0002-9267-7988 %} ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/weiyu150.jpg" }})
</div>
Wei-Yu is a statistician with expertise in genetic association studies, who is leading the omics analysis for the CLUSTER programme in childhood arthritis.

<div style="clear:both"></div>

### Ichcha Manipur ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/ichcha_150.jpg" }})
</div>

Ichcha is a Postdoctoral Research Associate with a background in Medicine and Computational Biology. Currently, she is working towards the development of statistical methods for PheWAS data and is interested in the study of shared genetic variants among multiple phenotypic traits. 

<div style="clear:both"></div>

### Guillermo Reales {% render_orcid 0000-0001-9993-3916 %} ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/guille.jpg" }})
</div>
Guillermo is a Postdoctoral Associate with a background in Molecular Biology and Population Genetics. He's particularly interested in Evolution and using computers to answer biological questions. His current work includes building a databank of GWAS summary statistics from immune-mediated diseases to help improve our knowledge about their mechanisms and their genetic relationships.

<div style="clear:both"></div>

### Luis A. Vargas-Mieles ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/luis_150.png" }})
</div>
[Luis Vargas](https://luisvargasmieles.github.io/) is a research associate with experience in statistical computing, particularly in the development and use of Markov chain Monte Carlo methods applied to very high-dimensional problems, such as imaging inverse problems. In 2018, he got an MSc with merits in Computational Applied Mathematics and in 2022 he obtained a PhD in Applied & Computational mathematics, both from the University of Edinburgh. He is now working on computational methods for biclustering & dimension reduction techniques in transcriptomic datasets related to immune-mediated disease.

### Elena Vigorito ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/elena.jpg" }})
</div>
Elena obtained her PhD in immunology at the University of Barcelona, Spain, and worked for several years on gene regulation on the immune system at the Babraham Institute, UK. She then trained in epidemiology and now works on developing statistical and computational methods for modelling genetic regulation through transcriptomic datasets.

<div style="clear:both"></div>

### Tom Willis ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/tom.jpg" }})
</div>

Tom is a PhD student at the MRC Biostatistics Unit. He comes from a background in computer science and statistics. His research concerns the use of conditional false discovery rates in high-dimensional 'omics datasets.

<div style="clear:both"></div>

<p>&nbsp;</p>
-------------------------------------------------------------------------------
<p>&nbsp;</p>

## Alumni ##


### Alan O'Callaghan ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/alan.jpg" }})
</div>

Alan worked as a research associate at the MRC Biostatistics Unit. He comes from a background in biomedical data science. His research centred on the development and use of Bayesian models for RNA sequencing data, and he has now moved to a position in Edinburgh.

<div style="clear:both"></div>
### Kath Nicholls ###

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/Kath_150.jpg" }})
</div>

[Kath](https://nichollskc.github.io/) is a PhD student jointly supervised by [Ken Smith](http://www.med.cam.ac.uk/smith/). Kath has a background in mathematics and software engineering and is interested in applying these skills to large biological datasets to learn about the immune system. She has a particular interest in graphs, and is currently working on methods to learn gene regulatory networks in the immune system.

<div style="clear:both"></div>

### Stephen Coleman

<div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/stco.jpg" }})
</div>
[Stephen Coleman](https://stcolema.github.io/) is undertaking a MRC-funded PhD in the field of integrative clustering methods. He hopes to contribute to the understanding of human health and disease through the application of these methods to 'omics data relevant to immune mediated diseases.
<div style="clear:both"></div>

### Qing Qing Zhou (2022) ###

Qing joined the group as an MPhil student in Computational Biology at Cambridge. Previously, she was trained as a biochemist and cell biologist in HKUST, where she developed a particular interest in investigating genetic traits that have pathological implications. She worked on engineering genetic features to capture risk factors associated with immune-mediated diseases.

<div style="clear:both"></div>

### Anna Hutchinson (2018-2022) ###

[Anna](https://annahutchinson.com/) was a PhD student at the MRC Biostatistics Unit (iCase studentship in collaboration with GlaxoSmithKline) and stayed on as a post-doc researcher. Coming from a Natural Sciences background, she is interested in using Bayesian techniques to answer biological questions.

Her research involved determining the true coverage of credible sets in fine mapping genetic association studies. She also has interests in using Bayesian approaches for capture Hi-C and Hi-ChIP data to better understand the 3D conformation of the genome.

She is now working at GSK.

<div style="clear:both"></div>

### Martin Kelemen (2018-2021) ###

Martin was a PhD student on the Mathematical Genomics and Medicine programme, supervised by Carl Anderson at the Wellcome Trust Sanger Institute and jointly by Chris. His research focused on developing practical machine learning methods for GWAS data, that will improve our understanding of how genetic differences influence variation in complex disease risk.

<div style="clear:both"></div>


### Martje Bos (2022) ###

Martje, as an MPhil student in Population Health Sciences and joined the group to undertake research for her thesis on condition-specific eQTLs in rheumatoid arthritis. For this, she was supervised by Elena Vigorito and Wei-Yu Lin. She obtained her BSc. in Health and Life Sciences from the University of Groningen and hopes to study for a PhD in Genetic Epidemiology after completing her MPhil.

<div style="clear:both"></div>

### Nastasiya Grinberg {% render_orcid 0000-0002-2727-5130 %} (2017-2021) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/nfg_small.jpg" }}) -->
<!-- </div> -->
Nastasiya Grinberg's research interests revolve around applications of machine learning and data mining techniques to genetic data and in bioinformatics. She also has a special interest in scientific computing.  She worked on colocalisation and TWAS and now holds a Senior Specialist position at NIAB, Cambridge. 

<div style="clear:both"></div>

### Oliver Burren {% render_orcid 0000-0002-3388-5760 %} (2017-2019) ###

<!--- <div style="float:left;margin:0 5px 0 0;" markdown="1">
![portait]({{ "/assets/Olly_small.jpg" }})
</div> --->

[Olly Burren](http://ollyburren.github.io) undertook a PhD, researching methods to integrate genomic and genetic datasets to better understand autoimmune disease mechanisms. 

Thesis: [*Integrative statistical methods for the genomic analysis of immune-mediated disease*](https://www.repository.cam.ac.uk/handle/1810/301259). 

He was previously head of Genome Informatics in the Diabetes and Inflammation Laboratory where he was responsible for the creation and smooth running of the ImmunoBase, T1DBase and CHiCP web resources which collate data and summary information from genomic, GWAS and ImmunoChip analyses of autoimmune diseases.  He now works at Astra Zeneca.
<!-- He continues to maintain  [CHiCP](http://www.chicp.org) as a means of visualizing Capture Hi-C datasets. -->

<div style="clear:both"></div>

### Hope Watson (2018) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/hope_small.jpg" }}) -->
<!-- </div> -->

Hope Watson undertook her thesis research for her MPhil in Epidemiology. Her research involved using Bayesian credible intervals to determine coverage of credible sets in fine mapping genetic associations. She also has an interest in systems architecture and high performance computing.  ​

<div style="clear:both"></div>

### James Liley (2015-2018) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/jamesliley_small.jpg" }}) -->
<!-- </div> -->
James Liley was a NIHR/BRC funded student on the Wellcome Trust four-year PhD programme in Mathematical Genomics and Medicine. He worked on analysis of GWAS and similar data across multiple diseases and in disease subtypes, and theoretical developments to the conditional False Discovery Rate.  He is now a working at Edinburgh University.

<div style="clear:both"></div>

### Loes Rutten-Jacobs {% render_orcid 0000-0003-3223-885X %} (2016-2018) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/Loes_small.jpg" }}) -->
<!-- </div> -->
[Loes Rutten-Jacobs](http://www.neuroscience.cam.ac.uk/directory/profile.php?lr406) held a British Heart Foundation Immediate Research Fellowship in the [Stroke Research Group](http://www.neurology.cam.ac.uk/neurology-unit-research-groups/stroke-research-group/) of the Department of Clinical Neurosciences, University of Cambridge and was a visiting member of our group during this time. She has now moved to Bonn, where she continues her research on pathophysiological mechanisms underlying cerebral small vessel disease, using genetic and epidemiologic approaches.	

<div style="clear:both"></div>

### Chris Eijsbouts {% render_orcid 0000-0001-5179-0653 %} (2016-2017) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/ChrisE.jpg" }}) -->
<!-- </div> -->
Chris Eijsbouts first joined the group as an MPhil student in Computational Biology for a 3 month summer project.  He stayed on afterwards as a research assistant, to continue his work on the interpretation of Capture Hi-C data. The goal of this project was to develop a method for improved identification of chromatin contacts which regulate gene expression, thereby facilitating the interpretation of GWAS studies.
He started a PhD in Oxford in Autumn 2017.

<div style="clear:both"></div>

### Steven Kiddle (2014-2017) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/kiddle_small.jpg" }}) -->
<!-- </div> -->
[Steven Kiddle](https://kclpure.kcl.ac.uk/portal/steven.kiddle.html) held an MRC Career Development Fellow in Biostatistics based at King’s College London, on which Chris Wallace was a statistical mentor. He visited our group in Cambridge from 2015-6, working on clustering approaches for time series and biomarkers of Alzheimer’s Disease.

He now holds a new MRC Career Development Fellowship and is based in the  [MRC Biostatistics Unit](http://www.mrc-bsu.cam.ac.uk).


<div style="clear:both"></div>

### Mary Fortune {% render_orcid 0000-0002-6006-4343 %} (2013-2017) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/mary_150.jpg" }}) -->
<!-- </div> -->
[Mary Fortune](http://www.damtp.cam.ac.uk/people/mdf34/)  worked as a Wellcome Trust PhD student in the [Mathematical Genomics and Medicine](http://cato.medschl.cam.ac.uk/graduate-programmes/phd-programmes/wellcome-trust-mathematical-genomics-and-medicine-mgm/) programme on colocalisation analysis in dependent datasets and methods for fitting models of genetic causality to summary GWAS data. She stayed on in the group to complete papers arising from her PhD work and has now moved to the [MRC Biostatistics Unit](http://www.mrc-bsu.cam.ac.uk).

Thesis: [*Statistical Techniques to Fine Map the Related Genetic Aetiology of Autoimmune Diseases*](https://doi.org/10.17863/CAM.10463)

<!-- Statistical colocalization of genetic risk variants for related autoimmune diseases in the context of common controls. Fortune et al. /Nature Genetics/ 2015, vol. 47, no. 7, pp. 839-846.  [ [=ukpmc=](http://ukpmc.ac.uk/abstract/MED/26053495) ] [ [=doi=](http://dx.doi.org/10.1038/ng.3330) ] -->


<div style="clear:both"></div>


### Lucia Cilloni (Summer 2015) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/LuciaCilloni.png" }}) -->
<!-- </div> -->
Lucia did her BSc in Mathematics at the University of Newcastle Upon
Tyne and her MPhil in Computational Biology at
Cambridge University. She did her summer project in the Wallace group on
eQTL-GWAS colocalisation analysis to identify causal genes for Type 1
Diabetes. She is now studying for a PhD in Epidemiology at
Imperial College London.

<div style="clear:both"></div>

### Hui Guo (2010-2014) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/hui.jpg" }}) -->
<!-- </div> -->
Hui worked as a post doctoral researcher on gene expression data, and led the analysis of our study relating gene expression in PBMCs from children at risk of type 1 diabetes to autoantibody positivity.  She is now a lecturer in biostatistics at [University of Manchester](http://www.population-health.manchester.ac.uk/staff/HuiGuo/).

<div style="float:left;margin:0 5px 0 0;" markdown="1">

### Xin Yang (2011-2014)
<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/xin.jpg" }}) -->
<!-- </div> -->
Xin's PhD focused on analysis of all kinds of next gen sequencing data - pooled DNA, RNA
seq, and bisulfite seq.  She had a special interest in modelling the extra binomial
variation induced by pooling.  

Thesis: [*Statistical analysis of next-generation sequencing data with applications to type 1 diabetes studies*](http://figshare.com/articles/Statistical_analysis_of_next_generation_sequencing_data_with_applications_to_type_1_diabetes_studies/1598221).

<!-- *Key paper*: Extra-binomial variation approach for analysis of pooled DNA sequencing data. Yang et al. /Bioinformatics/ 2012, vol. 28, no. 22, pp. 2898-2904. [ [=doi=](http://dx.doi.org/10.1093/bioinformatics/bts553) ] [ [=ukpmc=](http://ukpmc.ac.uk/abstract/MED/22976083) ] -->

She moved to KCL in 2014, and from there to the CTTV project at Sanger/GSK.

<div style="clear:both"></div>

### Nikolas Pontikos (2011-2014) ###

<!-- <div style="float:left;margin:0 5px 0 0;" markdown="1"> -->
<!-- ![portait]({{ "/assets/NikoSubmitting_small.jpg" }}) -->
<!-- </div> -->
During Niko's PhD he worked on automated analysis of flow cytometry data and applying
clustering methods to call and impute genotypes in the KIR locus.  

Thesis: [*Normalisation and Clustering Methods Applied to Association Studies in Type 1 Diabetes*](http://figshare.com/articles/Nikolas_Pontikos_PhD_Thesis_2015/1598198)

<!-- *Key paper*: A hybrid qPCR/SNP array approach allows cost efficient assessment of KIR gene copy numbers in large samples. Pontikos et al. /BMC genomics/, 2014, vol. 15, pp. 274. [ [=ukpmc=](http://ukpmc.ac.uk/abstract/MED/24720548) ] [ [=doi=](http://dx.doi.org/10.1186/1471-2164-15-274) ] -->

He moved to UCL in 2014 and maintains notes about his [current work](http://pontikos.github.io/).  




