<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Research</title>
  <meta name="description" content="Statistics applied to genomics to understand autoimmune diseases">

  <link rel="stylesheet" href="/assets/main.css">
  <!-- <link rel="stylesheet" href="/assets/css/style.css"> -->
  <link rel="stylesheet" href="assets/css/style.css">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
￼
  <link rel="canonical" href="http://chr1swallace.github.io/research/">
  <link rel="alternate" type="application/rss+xml" title="Wallace group" href="/feed.xml">
  
  
</head>


  <body>

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Wallace group</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
           
	  <a class="page-link" href="/people/">People</a>
	  
            
          
            
            
           
	  <a class="page-link" style="font-weight:bold;" href="/research/">Research</a>
	  <!-- <a class="page-link" style="color:#099;font-weight:bold;" href="/research/">Research</a> -->
	  
            
          
            
            
           
	  <a class="page-link" href="/data/">Data</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/code/">Software</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/papers/">Papers</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/talks/">Presentations</a>
	  
            
          
            
            
          
            
            
          
            
            
          
            
            
          
        </div>
      </nav>
    
  </div>
</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <!-- <header class="post-header"> -->
  <!--   <h1 class="post-title">Research</h1> -->
  <!-- </header> -->

  <div class="post-content">
    <!-- <iframe src='https://cdn.knightlab.com/libs/timeline3/latest/embed/index.html?source=1ILkjTq2V-Nq7HAdYUwjVpZwi_0rKIBkV6C6PRTyyaJo&font=Default&lang=en&initial_zoom=2&height=650' width='150%' height='650' webkitallowfullscreen mozallowfullscreen allowfullscreen frameborder='0'></iframe> -->

<h2 id="immune-mediated-diseases-their-relationships-and-divisions">Immune-mediated diseases, their relationships and divisions</h2>

<p><img src="/assets/disease-network.png" alt="portrait" style="float:left;margin:10px;" /></p>

<p>We study the shared and distinct genetic aetiology between related
autoimmune diseases both at the genomewide and region specific level,
addressing two separate questions.  First, at a the level of the whole
genome, do associations with one disease predict associations with
another disease?  We do this using an approach which estimates
conditional false discovery rates for association of each disease
at each SNP conditional on the other disease.  Because this uses
genomewide associations, it includes information from individual loci
that are not genomewide significant, but show a trend to shared
association for two diseases.  However, because genes with similar
function are often located close to one another, and because genetic
variants also show a spatial correlation due to linkage
disequilibrium, this does not autoimattically imply that the two
diseases share the same causal variants.</p>

<div style="clear:both"></div>

<p><img src="http://europepmc.org/articles/PMC4754941/bin/emss-63402-f0001.jpg" alt="portrait" style="float:right;margin:10px;" width="50%" /></p>

<p>To address this more specific question, we therefore use a
complementary approach, applying the an adapted colocalisation method that we use for matching disease and gene expression signatures to investigate sharing of causal variants in detail.
Our analysis of four autoimmune diseases identified 90 regions that were associated with at least one disease, 33 (37%) of which were associated with 2 or more disorders. For 14 of these 33 overlapping regions, we found evidence that the causal variants differed. This shows that even for two apparently closely related autoimmune diseases which both show association to the same genetic region
(delineated by recombination hotspots), this will relate to them
sharing the same causal variant only about 50% of the time.</p>

<div style="clear:both"></div>

<p><img src="http://europepmc.org/articles/PMC5357574/bin/emss-70580-f001.jpg" alt="portrait" style="float:left;margin:10px;" width="50%" /></p>

<p>More recently, we have developed a genome-wide test for determining whether some phenotypically or clinically defined division of patients corresponds to a genetic signature of aetiological heterogeneity.  This is particularly useful when direct comparison of the two subtypes yields no individual genomewide significant signal.   We investigate subgroups of 
type 1 diabetes (T1D) cases defined by autoantibody positivity, 
establishing evidence for differential genetic architecture with thyroid
 peroxidase antibody positivity, driven generally by variants in known 
T1D associated regions.</p>

<div style="clear:both"></div>

<p><strong>Related papers</strong></p>
<ol class="bibliography"><li>

  <span class="shortref">
    

    

    Fortune M. 

    
    <em>Nat Genet</em>
    
    
    2015
    

    <a href="/papers/index.html#2015_ng_fortune"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Liley J. 

    
    <em>Nat Genet</em>
    
    
    2017
    

    <a href="/papers/index.html#2017_ng_liley"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Burren O. 

    
    <em>Genome Medicine</em>
    
    
    2020
    

    <a href="/papers/index.html#burren2020-GeneticFeatureEngineering"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Ortiz-Fernández L. 

    
    <em>American Journal of Human Genetics</em>
    
    
    2020
    

    <a href="/papers/index.html#ortiz-fernandez2020-IdentificationSusceptibilityLoci"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Onengut-Gumuscu S. 

    
    <em>Nat Genet</em>
    
    
    2015
    

    <a href="/papers/index.html#2015_ng_onengut"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Liley J. 

    
    <em>PLoS Genet</em>
    
    
    2015
    

    <a href="/papers/index.html#2015_plosgen_liley"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li></ol>

<div style="clear:both"></div>

<h2 id="causal-variants">Causal variants</h2>

<p><img src="/assets/GUESSFM_small.png" alt="portrait" style="float:left;margin:10px;" /></p>

<p>Genetic association studies have identified many DNA sequence variants
that associate with disease risk. By exploiting the known correlation
that exists between neighbouring variants in the genome, inference can
be extended beyond those individual variants tested to identify sets
within which a causal variant is likely to reside. However, this
correlation, particularly in the presence of multiple disease causing
variants in relative proximity, makes disentangling the specific
causal variants difficult. Statistical approaches to this fine mapping
problem have traditionally taken a stepwise search approach, beginning
with the most associated variant in a region, then iteratively
attempting to find additional associated variants. 
We adapted a
stochastic search approach that avoids this stepwise process and is
explicitly designed for dealing with highly correlated predictors to
the fine mapping problem. We showed in simulated data that it
outperforms its stepwise counterpart and other variable selection
strategies such as the lasso.</p>

<p>Fine mapping results are often used to generate credible sets of SNPs, which are supposed to contain the true causal variant with a specified probability.  We have shown that frequentist coverage of these sets is often higher than expected, because this procedure is generally performed conditional on a small p value being seen.  Anna Hutchinson has presented this work in various forums, with slides <a href="https://annahutchinson.com/presentations/">here</a>.</p>

<p>In more recent work, we exploited the established relationships between immune mediated diseases, borrowing information from any other available diseases to increase the resolution of fine mapping results.  We highlighted in particular the potential for <em>joint tagging</em> where a single variant, in LD with two causal variants, can be mis-identified by stepwise methods, and showed how stochastic search, and multi-disease fine mapping, can avoid this.</p>

<p>We validated these results using allele specific expression studies of <em>IL2RA</em> in CD4+ T cells. All results are made available in an <a href="https://chr1swallace.github.io/MFM-output/">interactive form</a> for six immune-mediated diseases: autoimmune thyroid disease, celiac disease, childhood arthritis, MS, RA, T1D.</p>

<div style="clear:both"></div>

<p><strong>Related papers</strong></p>
<ol class="bibliography"><li>

  <span class="shortref">
    

    

    Hutchinson A. 

    
    <em>Human Molecular Genetics</em>
    
    
    2020
    

    <a href="/papers/index.html#2020_hutchinson_credsets"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Wallace C. 

    
    <em>PLoS Genet.</em>
    
    
    2015
    

    <a href="/papers/index.html#2015_wallace_il2ra"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Asimit J. 

    
    <em>Nat. Commun.</em>
    
    
    2019
    

    <a href="/papers/index.html#2019_asimit_multifinemap"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Hernández N. 

    
    <em>Nature Communications</em>
    
    
    2021
    

    <a href="/papers/index.html#2021_natcom_hernandez"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li></ol>
<h2 id="causal-genes">Causal genes</h2>

<p>Genomewide association studies (GWAS) have been hugely successful in
identifying associations between genetic variation and risk of common
diseases.  However, genetic variation will not be a pharmaceutical
target for these disease and translating this knowledge into
understanding the genes, cells and pathways involved in disease
aetiology has been slow.  This is because the associated genetic
variants do not typically reside in genes and change the protein they
encode, but lie between genes and are presumed to regulate their
expression in some cells, perhaps under specific conditions.</p>

<p><img src="/assets/ctsh.jpg" alt="portrait" style="float:right;margin:10px;" /></p>

<p>As GWAS studies for disease have been conducted over the past decade,
so people have conducted GWAS for other, more gene specific traits,
such as parallel GWAS for the expression levels of each of the ~20,000
protein coding genes in a given cell type, called “eQTL” studies.
These produce a local GWAS trace just as a disease GWAS does.  We use
a pattern matching technique called colocalisation to determine
whether the two GWAS traces are compatible with the two traits,
disease risk and expression of a given gene in a given cell type and
condition, sharing a causal variant.</p>

<div style="clear:both"></div>

<p><img src="/assets/pchic.png" alt="portrait" style="float:left;margin:10px;" />
<!-- <img src="http://europepmc.org/articles/PMC5123897/bin/fx1.jpg" width="50%" style="float:left;margin:10px;" /> --></p>

<p>Another approach is to exploit knowledge about the 3D folding of
chromatin derived from high throughput Chromosome Conformation Capture
(Hi-C) in its targetted form: Capture Hi-C (CHi-C).  This allows us to link GWAS causal variants (mapped probabilistically) to the genes they regulate, and we have deployed this approach across 17 primary human sorted cell types , as well as a separate more detailed comparison using CD4+ T cells, both activated and non-activated <a class="citation" href="/papers/index.html#2017_genbiol_burren">(<i>1</i>)</a>.</p>

<div style="clear:both"></div>

<p><strong>Related papers</strong></p>
<ol class="bibliography"><li>

  <span class="shortref">
    

    

    Burren O. 

    
    <em>Genome Biology</em>
    
    
    2017
    

    <a href="/papers/index.html#2017_genbiol_burren"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Eijsbouts C. 

    
    <em>bioRxiv</em>
    
    
    2018
    

    <a href="/papers/index.html#Eijsbouts2018-kq"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Javierre B. 

    
    <em>Cell</em>
    
    
    2016
    

    <a href="/papers/index.html#2016_cell"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Burren O. 

    
    <em>Bioinformatics</em>
    
    
    2014
    

    <a href="/papers/index.html#2014_bioinformatics_burren"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Giambartolomei C. 

    
    <em>PLoS Genet</em>
    
    
    2014
    

    <a href="/papers/index.html#2014_plosgen_giambartolomei"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Guo H. 

    
    <em>Hum Mol Genet</em>
    
    
    2015
    

    <a href="/papers/index.html#2015_hmg_guo"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Martin P. 

    
    <em>Nat. Commun.</em>
    
    
    2015
    

    <a href="/papers/index.html#2015_nc_martin"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Wallace C. 

    
    <em>Genet Epidemiol</em>
    
    
    2013
    

    <a href="/papers/index.html#2013_genepi_coloc"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li></ol>

<div style="clear:both"></div>

<h2 id="gene-expression-in-immune-cells">Gene expression in immune cells</h2>

<p><img src="/assets/ferreira.jpg" style="float:right;margin:10px;" /></p>

<p>The BABYDIET study led by Annette Ziegler and Ezio Bonifacio collected
blood samples longitudinally from 109 children genetically at risk of,
but initially unaffected by type 1 diabetes.  We were lucky to have
access to white blood cells (PBMCs) from these samples and measured
the gene expression in them.  We were particularly interested in
expression of interferon responsive genes, given previous links
between type 1 diabetes and infection.  Indeed, we saw an upregulation
of these genes in children who went on to develop the autoantibodies
that are strongly predictive of T1D diagnosis, but, crucially, the
upregulation was transient and preceded the appearance of
autoantibodies.  It was also temporally correlated with recent upper
respiratory infection, and may represent a biomarker for the response
to infection or the mechanism by which the infection influences type 1
diabetes risk.</p>

<p>We also used these data to investigate seasonal variation in gene
expression, and found that 25% of the genes expressed in these cells
in these children varied in their expression throughout the year.  In
winter, we saw that this expression profile produced a
pro-inflammatory environment.  This might be advantageous during a
season when infectious diseases are at a peak, but is a risk factor
for other diseases associated with inflammation such as cardiovascular
also peak in winter.  We replicated this finding in multiple datasets,
including one from the Southern hemisphere when winter occurs during
June-August.</p>

<div style="clear:both"></div>

<p><strong>Related papers</strong></p>
<ol class="bibliography"><li>

  <span class="shortref">
    

    

    Ferreira R. 

    
    <em>Diabetes</em>
    
    
    2014
    

    <a href="/papers/index.html#2014_diabetes_ifnsig"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li>
<li>

  <span class="shortref">
    

    

    Castro Dopico X. 

    
    <em>Nat Commun</em>
    
    
    2015
    

    <a href="/papers/index.html#2015_castro"><i class="fa fa-link" aria-hidden="true"></i></a>
  </span>
</li></ol>


  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <!-- <h2 class="footer-heading">Wallace group</h2> -->

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            
              Chris Wallace
            
<a href="https://orcid.org/0000-0001-9755-1703"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon"></a>
</li>
	  <li><a href="/contact/">Contact details</a>
            <!--  -->
            <!-- <li><a href="mailto:cew54-at-cam-dot-ac-dot-uk">cew54-at-cam-dot-ac-dot-uk</a></li> -->
            <!--  -->
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/chr1swallace"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/chr1swallace"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

        </ul>
      </div>

      <div class="footer-col footer-col-1">
        <p><b>Statistics</b> applied to <b>genomics</b> to understand <b>autoimmune diseases</b></p>
	<!-- &lt;b&gt;Statistics&lt;/b&gt; applied to genomics to understand autoimmune diseases</p> -->
      </div>
    </div>

  </div>

</footer>

    <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-134236872-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-134236872-1');
</script>



  </body>

</html>
