<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Software</title>
  <meta name="description" content="Statistics applied to genomics to understand autoimmune diseases">

  <link rel="stylesheet" href="/assets/main.css">
  <!-- <link rel="stylesheet" href="/assets/css/style.css"> -->
  <link rel="stylesheet" href="/assets/css/beautifuljekyll.css">
  <link rel="stylesheet" href="/assets/css/style.css">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
￼
  <link rel="canonical" href="http://chr1swallace.github.io/code/">
  <link rel="alternate" type="application/rss+xml" title="Wallace group" href="/feed.xml">
  
  
</head>


  <body>

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Wallace group</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
           
	  <a class="page-link" href="/people/">People</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/research/">Research</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/data/">Data</a>
	  
            
          
            
            
           
	  <a class="page-link" style="font-weight:bold;" href="/code/">Software</a>
	  <!-- <a class="page-link" style="color:#099;font-weight:bold;" href="/code/">Software</a> -->
	  
            
          
            
            
           
	  <a class="page-link" href="/papers/">Papers</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/talks/">Presentations</a>
	  
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
        </div>
      </nav>
    
  </div>
</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <!-- <header class="post-header"> -->
  <!--   <h1 class="post-title">Software</h1> -->
  <!-- </header> -->

  <div class="post-content">
    <p>This page collects released software packages.<br />
The results in papers are generated by analysis scripts that are separate from released and carefully documented software packages.  It may not be as pretty, or unit tested, or documented, but nonetheless, in an effort towards best reproducible research, we are trying to release the code supporting published results.  Where available, these will be linked from the individual <a href="/papers">papers</a>.</p>

<h2 id="r-packages">R packages</h2>
<p>To install R packages directly from github, do, in R:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>install.packages("devtools")
library(devtools)
install_github("chr1swallace/package.name")
</code></pre></div></div>
<h1 id="coloc">coloc</h1>
<p><a href="/coloc">package website</a> // 
<a href="http://cran.r-project.org/web/packages/coloc/index.html">CRAN</a> // <a href="https://github.com/chr1swallace/coloc">github</a> // 
A package of R functions used to test for
colocalisation of two genetic traits.  For more detail or
the statistical background and applications see:
<a class="citation" href="/papers/index.html#2013_genepi_coloc">(<i>1</i>)</a>
<a class="citation" href="/papers/index.html#2014_plosgen_giambartolomei">(<i>2</i>)</a>.</p>

<p>New in v3.1 which is now up to date on CRAN: <code class="language-plaintext highlighter-rouge">finemap.abf()</code> offers finemapping of single traits under a single causal variant assumption.  See <a href="http://chr1swallace.github.io/coloc/articles/vignette.html">vignette</a> for details.</p>

<h1 id="guessfm">GUESSFM</h1>
<p><a href="/GUESSFM">package website</a> // 
<a href="https://github.com/chr1swallace/GUESSFM">github</a> //
A wrapper around the GUESS stochastic model search method for fine
mapping causal variants in genetic data.   See (missing reference) <a href="/research">more detail here</a>.</p>

<h1 id="simgwas">simGWAS</h1>
<p><a href="/simGWAS">package website</a> // 
<a href="https://github.com/chr1swallace/simGWAS">github</a> //
Fast and accurate simulation of case-control GWAS summary statistics (missing reference)</p>

<h1 id="coloccommoncontrol">colocCommonControl</h1>
<p><a href="http://cran.r-project.org/web/packages/colocCommoncontrol/index.html">CRAN</a> // <a href="https://github.com/mdfortune/colocCommonControl">github</a> //
Adaptation of coloc for comparing two GWAS studies which share controls 
<a class="citation" href="/papers/index.html#2015_ng_fortune">(<i>3</i>)</a>.</p>

<h1 id="annotsnpstats">annotSnpStats</h1>
<p><a href="https://github.com/chr1swallace/annotSnpStats">github</a> //
extends SnpMatrix objects from the snpStats package by tying in
sample and snp annotation data.frames.  It allows the joint
manipulation, and binding, of such objects, and includes a few
functions I use for finding duplicate samples, applying QC, and
aligning alleles between different datasets.  It seems to work,
and I intend to push it to CRAN in the future.</p>

<h1 id="snpstatswriter">snpStatsWriter</h1>
<p><a href="https://github.com/mdfortune/colocCommonControl">github</a> //
allows writing snpStats objects to disk in formats suitable
for reading by snphap, phase, mach, IMPUTE, beagle, and
(almost) anything else that expects a rectangular format.</p>

<h1 id="wgsea">wgsea</h1>
<p><a href="http://cran.r-project.org/web/packages/wgsea/index.html">CRAN</a> //
implements a
Wilcoxon-based test for Gene Set Enrichment Analysis when
testing regions of the genome according to GWAS derived
evidence for SNP-trait association.  For more detail, see
the supplementary methods of <a class="citation" href="/papers/index.html#2010_nat_heinig">(<i>4</i>)</a>.</p>

<h1 id="extrabinomial">extraBinomial</h1>
<p><a href="http://cran.r-project.org/web/packages/extraBinomial">CRAN</a> //
(author Xin Yang, listed here because Xin doesn’t have her own website)
uses an extra binomial model to test for case-control
allele frequency differences in pooled sequence data <a class="citation" href="/papers/index.html#2012_bioinformatics_yang">(<i>5</i>)</a>.</p>

<h1 id="origin">origin</h1>
<p><a href="https://github.com/chr1swallace/origin">github</a> //
implements an   extension to the Weinberg method for testing for parent of origin
effects to allow for multiple affected offspring in a nuclear
family.  I haven’t used it in a while, and it requires some checking
just to get it to pass the CRAN tests.  Unless I need to revisit it
for another reason, I am unlikely to push it to CRAN, but it is a
finished package.
For more detail see:</p>

<p>Weinberg et al. Methods for detection of parent-of-origin effects in
    genetic studies of case-parents triads. /Am J Hum Genet/
    65:229-235. 1999.</p>

<p>Wallace et al. The imprinted DLK1-MEG3 gene region on chromosome
14q32.2 alters susceptibility to type 1 diabetes. /Nat
Genet/ 2010.
[ <a href="http://europepmc.org/abstract/MED/19966805">Europe PMC</a> ]</p>

<h2 id="non-r-packages">Non-R packages</h2>
<h1 id="vseams-olly-burren">VSEAMS (Olly Burren)</h1>
<p>Variant set enrichment analysis (of association summary statistics) using multivariate sampling. Please see the wiki available at <a href="http://github.com/ollyburren/vseams/wiki">this wiki</a> for further details.  Method described in 
<a class="citation" href="/papers/index.html#2014_bioinformatics_burren">(<i>6</i>)</a>.</p>

<!-- ### IBDSCORE  -->

<!-- implements a score test for the Rice-Holmans IBD -->
<!-- 	regression model. -->
<!-- 	The model is described by -->

<!--  Holmans, P. /Human Heredity/ 2002. 53:92-102 -->

<!-- Rice et al. /Genet Epidemiol/ 1999. 17:S691-S695 -->

<!-- and this score test by -->

<!-- Wallace et al. /Am J Hum Genet/ 2006. 79:323-31 [ [Europe PMC](http://europepmc.org/abstract/MED/16826522) ] } -->

<!-- [ [download](http://www-gene.cimr.cam.ac.uk/staff/wallace/soft/ibdscore.tgz) ] -->

<h2 id="see-also">See also…</h2>

<p><a href="http://crantastic.org/authors/2111">published R packages on CRAN</a> // <a href="http://github.com/chr1swallace">all code at github</a> // <a href="http://cwcode.wordpress.com">code blog</a></p>

  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <!-- <h2 class="footer-heading">Wallace group</h2> -->

    <div class="footer-col-wrapper">
    <!-- <div class="container-md beautiful-jekyll-footer"> -->
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            <!--  -->
              Chris Wallace
            <!--  -->
<a href="https://orcid.org/0000-0001-9755-1703"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon"></a>
</li>
	  <li><a href="/contact/">Contact details</a>
            <!--  -->
            <!-- <li><a href="mailto:cew54-at-cam-dot-ac-dot-uk">cew54-at-cam-dot-ac-dot-uk</a></li> -->
            <!--  -->
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/chr1swallace"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/chr1swallace"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

        </ul>
      </div>

      <div class="footer-col footer-col-1">
        <p><b>Statistics</b> applied to <b>genomics</b> to understand <b>autoimmune diseases</b></p>
	<!-- &lt;b&gt;Statistics&lt;/b&gt; applied to genomics to understand autoimmune diseases</p> -->
      </div>
    </div>

  </div>

</footer>

    <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-134236872-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-134236872-1');
</script>



  </body>

</html>
