<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Papers</title>
  <meta name="description" content="Statistics applied to genomics to understand autoimmune diseases">

  <link rel="stylesheet" href="/assets/main.css">
  <!-- <link rel="stylesheet" href="/assets/css/style.css"> -->
  <link rel="stylesheet" href="assets/css/style.css">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
￼
  <link rel="canonical" href="http://chr1swallace.github.io/papers/">
  <link rel="alternate" type="application/rss+xml" title="Wallace group" href="/feed.xml">
  
  
</head>


  <body>

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Wallace group</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
           
	  <a class="page-link" href="/people/">People</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/research/">Research</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/data/">Data</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/code/">Software</a>
	  
            
          
            
            
           
	  <a class="page-link" style="font-weight:bold;" href="/papers/">Papers</a>
	  <!-- <a class="page-link" style="color:#099;font-weight:bold;" href="/papers/">Papers</a> -->
	  
            
          
            
            
           
	  <a class="page-link" href="/talks/">Presentations</a>
	  
            
          
            
            
          
            
            
          
            
            
          
            
            
          
        </div>
      </nav>
    
  </div>
</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <!-- <header class="post-header"> -->
  <!--   <h1 class="post-title">Papers</h1> -->
  <!-- </header> -->

  <div class="post-content">
    <p>Our papers, ordered by year.</p>

<p />

<p><b>Key</b><br /></p>

<p><a href="#" class="pure-button doi">R <i class="fa fa-cubes"></i></a> Link to released and documented software package, where the paper describes a method with accompanying software.</p>

<p />

<p><a href="#" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a> The results in papers are generated by analysis scripts that are separate from any associated software.  It may not be as pretty, or unit tested, or well documented, but nonetheless, in an effort towards best reproducible research, we are trying to release the code supporting published results.</p>

<!-- Papers ordered by year -->

<h2 class="bibliography">2021</h2>
<ol class="bibliography"><li>

<div id="grinberg2021-MultitissueTranscriptomewideAssociation">
  <span class="title">Multi-Tissue Transcriptome-Wide Association Studies</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Grinberg N</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genetic Epidemiology</em>
    
    
    2021
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1002/gepi.22374" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2020/07/14/2020.07.13.201111.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>A transcriptome-wide association study (TWAS) attempts to identify disease associated genes by imputing gene expression into a genome-wide association study (GWAS) using an expression quantitative trait loci (eQTL) data set and then testing for associations with a trait of interest. Regulatory processes may be shared across related tissues and one natural extension of TWAS is harnessing cross-tissue correlation in gene expression to improve prediction accuracy. Here, we studied multi-tissue extensions of lasso regression and random forests (RF), joint lasso and RF-MTL (multi-task learning RF), respectively. We found that, on our chosen eQTL data set, multi-tissue methods were generally more accurate than their single-tissue counterparts, with RF-MTL performing the best. Simulations showed that these benefits generally translated into more associated genes identified, although highlighted that joint lasso had a tendency to erroneously identify genes in one tissue if there existed an eQTL signal for that gene in another. Applying the four methods to a type 1 diabetes GWAS, we found that multi-tissue methods found more unique associated genes for most of the tissues considered. We conclude that multi-tissue methods are competitive and, for some cell types, superior to single-tissue approaches and hold much promise for TWAS studies.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2021_liley">
  <span class="title">Accurate Error Control in High-Dimensional Association Testing Using Conditional False Discovery Rates</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Biometrical Journal</em>
    
    
    2021
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1002/bimj.201900254" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>High-dimensional hypothesis testing is ubiquitous in the biomedical sciences, and informative covariates may be employed to improve power. The conditional false discovery rate (cFDR) is a widely used approach suited to the setting where the covariate is a set of p-values for the equivalent hypotheses for a second trait. Although related to the Benjamini–Hochberg procedure, it does not permit any easy control of type-1 error rate and existing methods are over-conservative. We propose a new method for type-1 error rate control based on identifying mappings from the unit square to the unit interval defined by the estimated cFDR and splitting observations so that each map is independent of the observations it is used to test. We also propose an adjustment to the existing cFDR estimator which further improves power. We show by simulation that the new method more than doubles potential improvement in power over unconditional analyses compared to existing methods. We demonstrate our method on transcriptome-wide association studies and show that the method can be used in an iterative way, enabling the use of multiple covariates successively. Our methods substantially improve the power and applicability of cFDR analysis.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2021_natcompsci_vigorito">
  <span class="title">Detection of quantitative trait loci from RNA-seq data with or without genotypes using BaseQTL</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Vigorito E</b>, 
    
    
    
    
    
    

    

    Lin W, 
    
    
    
    
    
    

    

    Starr C, 
    
    
    
    
    
    

    

    Kirk P, 
    
    
    
    
    
    

    

    White S, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Comp Sci</em>
    
    
    2021
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/s43588-021-00087-y" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2020/07/16/2020.07.16.203851.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Available methods to detect molecular quantitative trait loci (QTL) require study individuals to be genotyped. Here, we describe BaseQTL, a Bayesian method that exploits allele-specific expression to map molecular QTL from sequencing reads even when no genotypes are available. When used with genotypes, BaseQTL has lower error rates and increased power compared with existing QTL mapping methods. Running without genotypes limits how many tests can be performed, but due to the proximity of QTL variants to gene bodies, the 2.8% of variants within a 100kB-window that could be tested, contained 26% of QTL variants detectable with genotypes. eQTL effect estimates were invariably consistent between analyses performed with and without genotypes. Often, sequencing data may be generated in absence of genotypes on patients and controls in differential expression studies, and we identified an apparent psoriasis-specific effect for GSTP1 in one such dataset, providing new insights into disease-dependent gene regulation.Competing Interest StatementThe authors have declared no competing interest.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Nicholls2021">
  <span class="title">Comparison of sparse biclustering algorithms for gene expression datasets</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Nicholls K</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Briefings in Bioinformatics</em>
    
    
    2021
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/bib/bbab140" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Gene clustering and sample clustering are commonly used to find patterns in gene expression datasets. However, genes may cluster differently in heterogeneous samples (e.g. different tissues or disease states), whilst traditional methods assume that clusters are consistent across samples. Biclustering algorithms aim to solve this issue by performing sample clustering and gene clustering simultaneously. Existing reviews of biclustering algorithms have yet to include a number of more recent algorithms and have based comparisons on simplistic simulated datasets without specific evaluation of biclusters in real datasets, using less robust metrics. We compared four classes of sparse biclustering algorithms on a range of simulated and real datasets. All algorithms generally struggled on simulated datasets with a large number of genes or implanted biclusters. We found that Bayesian algorithms with strict sparsity constraints had high accuracy on the simulated datasets and did not require any post-processing, but were considerably slower than other algorithm classes. We found that non-negative matrix factorisation algorithms performed poorly, but could be re-purposed for biclustering through a sparsity-inducing post-processing procedure we introduce; one such algorithm was one of the most highly ranked on real datasets. In a multi-tissue knockout mouse RNA-seq dataset, the algorithms rarely returned clusters containing samples from multiple different tissues, whilst such clusters were identified in a human dataset of more closely related cell types (sorted blood cell subsets). This highlights the need for further thought in the design and analysis of multi-tissue studies to avoid differences between tissues dominating the analysis. Code to run the analysis is available at https://github.com/nichollskc/biclust_comp, including wrappers for each algorithm, implementations of evaluation metrics, and code to simulate datasets and perform pre- and post-processing. The full tables of results are available at https://doi.org/10.5281/zenodo.4581206.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2021_plosgen">
  <span class="title">A more accurate method for colocalisation analysis allowing for multiple causal variants</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em></em>
    
    
    2021
    
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1009440" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="Reales_2020">
  <span class="title">RapidoPGS: A rapid polygenic score calculator for summary GWAS data without a test dataset</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Reales G</b>, 
    
    
    
    
    
    

    

    <b>Vigorito E</b>, 
    
    
    
    
    
    

    

    <b>Kelemen M</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Bioinformatics</em>
    
    
    2021
    
  <!--  -->
   
  <a href="http://doi.org/10.1101/2020.07.24.220392" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2021_dopico">
  <span class="title">Seropositivity in blood donors and pregnant women during the first year of SARS-CoV-2 transmission in Stockholm, Sweden</span>
  <span class="author">
    
    
    
    
    
    

    

    Castro Dopico X, 
    
    
    
    
    
    

    

    Muschiol S, 
    
    
    
    
    
    

    

    Christian M, 
    
    
    
    
    
    

    

    Hanke L, 
    
    
    
    
    
    

    

    Sheward D, 
    
    
    
    
    
    

    

    <b>Grinberg N</b>, 
    
    
    
    
    
    

    

    Rorbach J, 
    
    
    
    
    
    

    

    Bogdanovic G, 
    
    
    
    
    
    

    

    Mcinerney G, 
    
    
    
    
    
    

    

    Allander T, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Murrell B, 
    
    
    
    
    
    

    

    Albert J, 
    
    
    
    
    
    

    

    Karlsson Hedestam G. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Journal of Internal Medicine</em>
    
    
    2021
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/https://doi.org/10.1111/joim.13304" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/joim.13304" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Abstract Background In Sweden, social restrictions to contain SARS-CoV-2 have primarily relied upon voluntary adherence to a set of recommendations. Strict lockdowns have not been enforced, potentially affecting viral dissemination. To understand the levels of past SARS-CoV-2 infection in the Stockholm population before the start of mass vaccinations, healthy blood donors and pregnant women (n = 5,100) were sampled at random between 14 March 2020 and 28 February 2021. Methods In this cross-sectional prospective study, otherwise-healthy blood donors (n = 2,600) and pregnant women (n = 2,500) were sampled for consecutive weeks (at four intervals) throughout the study period. Sera from all participants and a cohort of historical (negative) controls (n = 595) were screened for IgG responses against stabilized trimers of the SARS-CoV-2 spike (S) glycoprotein and the smaller receptor-binding domain (RBD). As a complement to standard analytical approaches, a probabilistic (cut-off independent) Bayesian framework that assigns likelihood of past infection was used to analyse data over time. Setting Healthy participant samples were randomly selected from their respective pools through Karolinska University Hospital. The study was carried out in accordance with Swedish Ethical Review Authority: registration number 2020–01807. Participants No participants were symptomatic at sampling, and blood donors were all over the age of 18. No additional metadata were available from the participants. Results Blood donors and pregnant women showed a similar seroprevalence. After a steep rise at the start of the pandemic, the seroprevalence trajectory increased steadily in approach to the winter second wave of infections, approaching 15% of all individuals surveyed by 13 December 2020. By the end of February 2021, 19% of the population tested seropositive. Notably, 96% of seropositive healthy donors screened (n = 56) developed neutralizing antibody responses at titres comparable to or higher than those observed in clinical trials of SARS-CoV-2 spike mRNA vaccination, supporting that mild infection engenders a competent B-cell response. Conclusions These data indicate that in the first year since the start of community transmission, seropositivity levels in metropolitan in Stockholm had reached approximately one in five persons, providing important baseline seroprevalence information prior to the start of vaccination.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Freire-Pritchett2021">
  <span class="title">Detecting chromosomal interactions in Capture Hi-C data with CHiCAGO and companion tools</span>
  <span class="author">
    
    
    
    
    
    

    

    Freire-Pritchett P, 
    
    
    
    
    
    

    

    Ray-Jones H, 
    
    
    
    
    
    

    

    Della Rosa M, 
    
    
    
    
    
    

    

    <b>Eijsbouts C</b>, 
    
    
    
    
    
    

    

    Orchard W, 
    
    
    
    
    
    

    

    Wingett S, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Cairns J, 
    
    
    
    
    
    

    

    Spivakov M, 
    
    
    
    
    
    

    

    Malysheva V. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nature Protocols</em>
    
    
    2021
    
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="Ruffieux2021">
  <span class="title">EPISPOT: an epigenome-driven approach for detecting and interpreting hotspots in molecular QTL studies</span>
  <span class="author">
    
    
    
    
    
    

    

    Ruffieux H, 
    
    
    
    
    
    

    

    Fairfax B, 
    
    
    
    
    
    

    

    Nassiri I, 
    
    
    
    
    
    

    

    <b>Vigorito E</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Richardson S, 
    
    
    
    
    
    

    

    Bottolo L. 
    
  </span>
  
  <div class="jblocks">
    
    <em>The American Journal of Human Genetics</em>
    
    
    2021
    
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2021_natcom_hernandez">
  <span class="title">The flashfm approach for fine-mapping multiple quantitative traits</span>
  <span class="author">
    
    
    
    
    
    

    

    Hernández N, 
    
    
    
    
    
    

    

    Soenksen J, 
    
    
    
    
    
    

    

    Newcombe P, 
    
    
    
    
    
    

    

    Sandhu M, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    <b>Asimit J</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Commun</em>
    
    
    2021
    
  <!--  -->
   
  <a href="http://doi.org/10.1038/s41467-021-26364-y" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2021/04/11/2021.04.09.439186.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2021_plosgen_hutchinson">
  <span class="title">Leveraging auxiliary data from arbitrary distributions to boost GWAS discovery with Flexible cFDR</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Hutchinson A</b>, 
    
    
    
    
    
    

    

    <b>Reales G</b>, 
    
    
    
    
    
    

    

    Willis T, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLOS Genetics</em>
    
    
    2021
    
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1009853" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2021/04/23/2020.12.04.411710.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2020</h2>
<ol class="bibliography"><li>

<div id="bourges2020-ResolvingMechanismsImmunemediated">
  <span class="title">Resolving Mechanisms of Immune-Mediated Disease in Primary CD4 T Cells</span>
  <span class="author">
    
    
    
    
    
    

    

    Bourges C, 
    
    
    
    
    
    

    

    Groff A, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Gerhardinger C, 
    
    
    
    
    
    

    

    Mattioli K, 
    
    
    
    
    
    

    

    <b>Hutchinson A</b>, 
    
    
    
    
    
    

    

    Hu T, 
    
    
    
    
    
    

    

    Anand T, 
    
    
    
    
    
    

    

    Epping M, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Smith K, 
    
    
    
    
    
    

    

    Rinn J, 
    
    
    
    
    
    

    

    Lee J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>EMBO Molecular Medicine</em>
    
    
    2020
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.15252/emmm.202012112" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2020/01/17/2020.01.16.908988.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Abstract Deriving mechanisms of immune-mediated disease from GWAS data remains a formidable challenge, with attempts to identify causal variants being frequently hampered by strong linkage disequilibrium. To determine whether causal variants could be identified from their functional effects, we adapted a massively parallel reporter assay for use in primary CD4 T cells, the cell type whose regulatory DNA is most enriched for immune-mediated disease SNPs. This enabled the effects of candidate SNPs to be examined in a relevant cellular context and generated testable hypotheses into disease mechanisms. To illustrate the power of this approach, we investigated a locus that has been linked to six immune-mediated diseases but cannot be fine-mapped. By studying the lead expression-modulating SNP, we uncovered an NF-?B-driven regulatory circuit which constrains T-cell activation through the dynamic formation of a super-enhancer that upregulates TNFAIP3 (A20), a key NF-?B inhibitor. In activated T cells, this feedback circuit is disrupted?and super-enhancer formation prevented?by the risk variant at the lead SNP, leading to unrestrained T-cell activation via a molecular mechanism that appears to broadly predispose to human autoimmunity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="burren2020-GeneticFeatureEngineering">
  <span class="title">Genetic Feature Engineering Enables Characterisation of Shared Risk Factors in Immune-Mediated Diseases</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    <b>Reales G</b>, 
    
    
    
    
    
    

    

    Wong L, 
    
    
    
    
    
    

    

    Bowes J, 
    
    
    
    
    
    

    

    Lee J, 
    
    
    
    
    
    

    

    Barton A, 
    
    
    
    
    
    

    

    Lyons P, 
    
    
    
    
    
    

    

    Smith K, 
    
    
    
    
    
    

    

    Thomson W, 
    
    
    
    
    
    

    

    Kirk P, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genome Medicine</em>
    
    
    2020
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1186/s13073-020-00797-4" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2020/01/17/2020.01.14.905869.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome-wide association studies (GWAS) have identified pervasive sharing of genetic architectures across multiple immune-mediated diseases (IMD). By learning the genetic basis of IMD risk from common diseases, this sharing can be exploited to enable analysis of less frequent IMD where, due to limited sample size, traditional GWAS techniques are challenging.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2020_hutchinson_credsets">
  <span class="title">Fine-Mapping Genetic Associations</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Hutchinson A</b>, 
    
    
    
    
    
    

    

    <b>Asimit J</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Human Molecular Genetics</em>
    
    
    2020
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/hmg/ddaa148" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Whilst thousands of genetic variants have been associated with human traits, identifying the subset of those variants that are causal requires a further ‘fine-mapping’ step. We review the basic fine-mapping approach, which is computationally fast and requires only summary data, but depends on an assumption of a single causal variant per associated region which is recognized as biologically unrealistic. We discuss different ways that the approach has been built upon to accommodate multiple causal variants in a region and to incorporate additional layers of functional annotation data. We further review methods for simultaneous fine-mapping of multiple datasets, either exploiting different linkage disequilibrium (LD) structures across ancestries or borrowing information between distinct but related traits. Finally, we look to the future and the opportunities that will be offered by increasingly accurate maps of causal variants for a multitude of human traits.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="ortiz-fernandez2020-IdentificationSusceptibilityLoci">
  <span class="title">Identification of Susceptibility Loci for Takayasu Arteritis through a Large Multi-Ancestral Genome-Wide Association Study</span>
  <span class="author">
    
    
    
    
    
    

    

    Ortiz-Fernández L, 
    
    
    
    
    
    

    

    Saruhan-Direskeneli G, 
    
    
    
    
    
    

    

    Alibaz-Oner F, 
    
    
    
    
    
    

    

    Kaymaz-Tahra S, 
    
    
    
    
    
    

    

    Coit P, 
    
    
    
    
    
    

    

    Kong X, 
    
    
    
    
    
    

    

    Kiprianos A, 
    
    
    
    
    
    

    

    Maughan R, 
    
    
    
    
    
    

    

    Aydin S, 
    
    
    
    
    
    

    

    Aksu K, 
    
    
    
    
    
    

    

    Keser G, 
    
    
    
    
    
    

    

    Kamali S, 
    
    
    
    
    
    

    

    Inanc M, 
    
    
    
    
    
    

    

    Springer J, 
    
    
    
    
    
    

    

    Akar S, 
    
    
    
    
    
    

    

    Onen F, 
    
    
    
    
    
    

    

    Akkoc N, 
    
    
    
    
    
    

    

    Khalidi N, 
    
    
    
    
    
    

    

    Koening C, 
    
    
    
    
    
    

    

    Karadag O, 
    
    
    
    
    
    

    

    Kiraz S, 
    
    
    
    
    
    

    

    Forbess L, 
    
    
    
    
    
    

    

    Langford C, 
    
    
    
    
    
    

    

    McAlear C, 
    
    
    
    
    
    

    

    Ozbalkan Z, 
    
    
    
    
    
    

    

    Yavuz S, 
    
    
    
    
    
    

    

    Çetin G, 
    
    
    
    
    
    

    

    Alpay-Kanitez N, 
    
    
    
    
    
    

    

    Chung S, 
    
    
    
    
    
    

    

    Ates A, 
    
    
    
    
    
    

    

    Karaaslan Y, 
    
    
    
    
    
    

    

    McKinnon-Maksimowicz K, 
    
    
    
    
    
    

    

    Monach P, 
    
    
    
    
    
    

    

    Ozer H, 
    
    
    
    
    
    

    

    Seyahi E, 
    
    
    
    
    
    

    

    Fresko I, 
    
    
    
    
    
    

    

    Cefle A, 
    
    
    
    
    
    

    

    Seo P, 
    
    
    
    
    
    

    

    Warrington K, 
    
    
    
    
    
    

    

    Ozturk M, 
    
    
    
    
    
    

    

    Ytterberg S, 
    
    
    
    
    
    

    

    Cobankara V, 
    
    
    
    
    
    

    

    Onat A, 
    
    
    
    
    
    

    

    Duzgun N, 
    
    
    
    
    
    

    

    Bıcakcıgil M, 
    
    
    
    
    
    

    

    Yentür S, 
    
    
    
    
    
    

    

    Lally L, 
    
    
    
    
    
    

    

    Manfredi A, 
    
    
    
    
    
    

    

    Baldissera E, 
    
    
    
    
    
    

    

    Erken E, 
    
    
    
    
    
    

    

    Yazici A, 
    
    
    
    
    
    

    

    Kısacık B, 
    
    
    
    
    
    

    

    Kaşifoğlu T, 
    
    
    
    
    
    

    

    Dalkilic E, 
    
    
    
    
    
    

    

    Cuthbertson D, 
    
    
    
    
    
    

    

    Pagnoux C, 
    
    
    
    
    
    

    

    Sreih A, 
    
    
    
    
    
    

    

    <b>Reales G</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wren J, 
    
    
    
    
    
    

    

    Cunninghame-Graham D, 
    
    
    
    
    
    

    

    Vyse T, 
    
    
    
    
    
    

    

    Sun Y, 
    
    
    
    
    
    

    

    Chen H, 
    
    
    
    
    
    

    

    Grayson P, 
    
    
    
    
    
    

    

    Tombetti E, 
    
    
    
    
    
    

    

    Jiang L, 
    
    
    
    
    
    

    

    Mason J, 
    
    
    
    
    
    

    

    Merkel P, 
    
    
    
    
    
    

    

    Direskeneli H, 
    
    
    
    
    
    

    

    Sawalha A. 
    
  </span>
  
  <div class="jblocks">
    
    <em>American Journal of Human Genetics</em>
    
    
    2020
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1016/j.ajhg.2020.11.014" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Takayasu arteritis is a rare inflammatory disease of large arteries. We performed a genetic study in Takayasu arteritis comprising 6,670 individuals (1,226 affected individuals) from five different populations. We discovered HLA risk factors and four non-HLA susceptibility loci in VPS8, SVEP1, CFL2, and chr13q21 and reinforced IL12B, PTK2B, and chr21q22 as robust susceptibility loci shared across ancestries. Functional analysis proposed plausible underlying disease mechanisms and pinpointed ETS2 as a potential causal gene for chr21q22 association. We also identified >60 candidate loci with suggestive association (p < 5 \texttimes 10-5) and devised a genetic risk score for Takayasu arteritis. Takayasu arteritis was compared to hundreds of other traits, revealing the closest genetic relatedness to inflammatory bowel disease. Epigenetic patterns within risk loci suggest roles for monocytes and B cells in Takayasu arteritis. This work enhances understanding of the genetic basis and pathophysiology of Takayasu arteritis and provides clues for potential new therapeutic targets.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="wallace2020-ElicitingPriorsRelaxing">
  <span class="title">Eliciting Priors and Relaxing the Single Causal Variant Assumption in Colocalisation Analyses</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLOS Genetics</em>
    
    
    2020
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1008720" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Horizontal integration of summary statistics from different GWAS traits can be used to evaluate evidence for their shared genetic causality. One popular method to do this is a Bayesian method, coloc, which is attractive in requiring only GWAS summary statistics and no linkage disequilibrium estimates and is now being used routinely to perform thousands of comparisons between traits. Here we show that while most users do not adjust default software values, misspecification of prior parameters can substantially alter posterior inference. We suggest data driven methods to derive sensible prior values, and demonstrate how sensitivity analysis can be used to assess robustness of posterior inference. The flexibility of coloc comes at the expense of an unrealistic assumption of a single causal variant per trait. This assumption can be relaxed by stepwise conditioning, but this requires external software and an LD matrix aligned to study alleles. We have now implemented conditioning within coloc, and propose a new alternative method, masking, that does not require LD and approximates conditioning when causal variants are independent. Importantly, masking can be used in combination with conditioning where allelically aligned LD estimates are available for only a single trait. We have implemented these developments in a new version of coloc which we hope will enable more informed choice of priors and overcome the restriction of the single causal variant assumptions in coloc analysis.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Mitchelmore2020">
  <span class="title">Functional effects of variation in transcription factor binding highlight long-range gene regulation by epromoters.</span>
  <span class="author">
    
    
    
    
    
    

    

    Mitchelmore J, 
    
    
    
    
    
    

    

    <b>Grinberg N</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Spivakov M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nucleic acids research</em>
    
    
    2020
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/nar/gkaa123" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC7102942" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Identifying DNA cis-regulatory modules (CRMs) that control the expression of specific genes is crucial for deciphering the logic of transcriptional control. Natural genetic variation can point to the possible gene regulatory function of specific sequences through their allelic associations with gene expression. However, comprehensive identification of causal regulatory sequences in brute-force association testing without incorporating prior knowledge is challenging due to limited statistical power and effects of linkage disequilibrium. Sequence variants affecting transcription factor (TF) binding at CRMs have a strong potential to influence gene regulatory function, which provides a motivation for prioritizing such variants in association testing. Here, we generate an atlas of CRMs showing predicted allelic variation in TF binding affinity in human lymphoblastoid cell lines and test their association with the expression of their putative target genes inferred from Promoter Capture Hi-C and immediate linear proximity. We reveal >1300 CRM TF-binding variants associated with target gene expression, the majority of them undetected with standard association testing. A large proportion of CRMs showing associations with the expression of genes they contact in 3D localize to the promoter regions of other genes, supporting the notion of ’epromoters’: dual-action CRMs with promoter and distal enhancer activity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Coleman_2020">
  <span class="title">Consensus clustering for Bayesian mixture models</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Coleman S</b>, 
    
    
    
    
    
    

    

    Kirk P, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2020
    
  <!--  -->
   
  <a href="http://doi.org/10.1101/2020.12.17.423244" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2019</h2>
<ol class="bibliography"><li>

<div id="2019_biorxiv_colocpriors">
  <span class="title">Eliciting Priors and Relaxing the Single Causal Variant Assumption in Colocalisation Analyses</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2019
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/838946" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2019/11/12/838946.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Horizontal integration of summary statistics from different GWAS traits can be used to evaluate evidence for their shared genetic causality. One popular method to do this is a Bayesian method, coloc, which is attractive in requiring only GWAS summary statistics and no linkage disequilibrium estimates and is now being used routinely to perform thousands of comparisons between traits. Here we show that while most users do not adjust default software values, misspecification of prior parameters can substantially alter posterior inference. We suggest data driven methods to derive sensible prior values, and demonstrate how sensitivity analysis can be used to assess robustness of posterior inference. The flexibility of coloc comes at the expense of an unrealistic assumption of a single causal variant per trait. This assumption can be relaxed by stepwise conditioning, but this requires external software and an LD matrix aligned to study alleles. We have now implemented conditioning within coloc, and propose a new alternative method, masking, that does not require LD and approximates conditioning when causal variants are independent. Importantly, masking can be used in combination with conditioning where allelically aligned LD estimates are available for only a single trait. We have implemented these developments in a new version of coloc which we hope will enable more informed choice of priors and overcome the restriction of the single causal variant assumptions in coloc analysis.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2019_biorxiv_hutchinson">
  <span class="title">Correcting the Coverage of Credible Sets in Bayesian Genetic Fine-Mapping</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Hutchinson A</b>, 
    
    
    
    
    
    

    

    Watson H, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2019
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/781062" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2019/09/25/781062.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome Wide Association Studies (GWAS) have successfully identified thousands of loci associated with human diseases. Bayesian genetic fine-mapping studies aim to identify the specific causal variants within GWAS loci responsible for each association, reporting credible sets of plausible causal variants, which are interpreted as containing the causal variant with some “coverage probability”.Here, we investigate the coverage probabilities of credible sets through simulations and find that these are systematically biased. We present a method to re-estimate the coverage of credible sets using rapid simulations based on the observed, or estimated, SNP correlation structure, we call this the “corrected coverage estimate”. This is extended to find “corrected credible sets”, which are the smallest set of variants such that their corrected coverage estimate meets the target coverage.We use our method to improve the resolution of a fine-mapping study of type 1 diabetes. We found that in 27 out of 39 associated genomic regions our method could reduce the number of potentially causal variants to consider for follow-up, and found that none of the 95% or 99% credible sets required the inclusion of more variants – a pattern matched in simulations of well powered GWAS.Crucially, our correction method requires only GWAS summary statistics and remains accurate when SNP correlations are estimated from a large reference panel. Using our method to improve the resolution of fine-mapping studies will enable more efficient expenditure of resources in the follow-up process of annotating the variants in the credible set to determine the implicated genes and pathways in human diseases.Author summary Pinpointing specific genetic variants within the genome that are causal for human diseases is difficult due to complex correlation patterns existing between variants. Consequently, researchers typically prioritise a set of plausible causal variants for functional validation - these sets of putative causal variants are called “credible sets”. We find that the probabilistic interpretation that these credible sets do indeed contain the true causal variant are systematically biased, in that the reported probabilities consistently underestimate the true coverage of the causal variant in the credible set. We have developed a method to provide researchers with a “corrected coverage estimate” that the true causal variant appears in the credible set, and this has been extended to find “corrected credible sets”, allowing for more efficient allocation of resources in the expensive follow-up laboratory experiments. We used our method to reduce the number of genetic variants to consider as causal candidates for follow-up in 27 genomic regions that are associated with type 1 diabetes.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2019_biorxiv_mitchelmore">
  <span class="title">Functional Effects of Variation in Transcription Factor Binding Highlight Long-Range Gene Regulation by Epromoters</span>
  <span class="author">
    
    
    
    
    
    

    

    Mitchelmore J, 
    
    
    
    
    
    

    

    <b>Grinberg N</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Spivakov M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2019
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/620062" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2019/04/29/620062.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Identifying DNA cis-regulatory modules (CRMs) that control the expression of specific genes is crucial for deciphering the logic of transcriptional control. Natural genetic variation can point to the possible gene regulatory function of specific sequences through their allelic associations with gene expression. However, comprehensive identification of causal regulatory sequences in brute-force association testing without incorporating prior knowledge is challenging due to limited statistical power and effects of linkage disequilibrium. Sequence variants affecting transcription factor (TF) binding at CRMs have a strong potential to influence gene regulatory function, which provides a motivation for prioritising such variants in association testing. Here, we generate an atlas of CRMs showing predicted allelic variation in TF binding affinity in human lymphoblastoid cell lines (LCLs) and test their association with the expression of their putative target genes inferred from Promoter Capture Hi-C and immediate linear proximity. We reveal over 1300 CRM TF-binding variants associated with target gene expression, the majority of them undetected with standard association testing. A large proportion of CRMs showing associations with the expression of genes they contact in 3D localise to the promoter regions of other genes, supporting the notion of epromoters: dual-action CRMs with promoter and distal enhancer activity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2019_eijsbouts_peaky">
  <span class="title">Fine Mapping Chromatin Contacts in Capture Hi-C Data.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Eijsbouts C</b>, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Newcombe P, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>BMC genomics</em>
    
    
    2019
    
  <!--  -->
   
  <a href="http://doi.org/10.1186/s12864-018-5314-5" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  <a href="https://github.com/cqgd/pky" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <a href="https://github.com/chr1swallace/peakypaper" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/chr1swallace/peakypaper" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2019_lyons_egpa">
  <span class="title">Genome-Wide Association Study of Eosinophilic Granulomatosis with Polyangiitis Reveals Genomic Loci Stratified by ANCA Status</span>
  <span class="author">
    
    
    
    
    
    

    

    Lyons P, 
    
    
    
    
    
    

    

    Peters J, 
    
    
    
    
    
    

    

    Alberici F, 
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    Coulson R, 
    
    
    
    
    
    

    

    Astle W, 
    
    
    
    
    
    

    

    Baldini C, 
    
    
    
    
    
    

    

    Bonatti F, 
    
    
    
    
    
    

    

    Cid M, 
    
    
    
    
    
    

    

    Elding H, 
    
    
    
    
    
    

    

    Emmi G, 
    
    
    
    
    
    

    

    Epplen J, 
    
    
    
    
    
    

    

    Guillevin L, 
    
    
    
    
    
    

    

    Jayne D, 
    
    
    
    
    
    

    

    Jiang T, 
    
    
    
    
    
    

    

    Gunnarsson I, 
    
    
    
    
    
    

    

    Lamprecht P, 
    
    
    
    
    
    

    

    Leslie S, 
    
    
    
    
    
    

    

    Little M, 
    
    
    
    
    
    

    

    Martorana D, 
    
    
    
    
    
    

    

    Moosig F, 
    
    
    
    
    
    

    

    Neumann T, 
    
    
    
    
    
    

    

    Ohlsson S, 
    
    
    
    
    
    

    

    Quickert S, 
    
    
    
    
    
    

    

    Ramirez G, 
    
    
    
    
    
    

    

    Rewerska B, 
    
    
    
    
    
    

    

    Schett G, 
    
    
    
    
    
    

    

    Sinico R, 
    
    
    
    
    
    

    

    Szczeklik W, 
    
    
    
    
    
    

    

    Tesar V, 
    
    
    
    
    
    

    

    Vukcevic D, 
    
    
    
    
    
    

    

    Akil M, 
    
    
    
    
    
    

    

    Barratt J, 
    
    
    
    
    
    

    

    Basu N, 
    
    
    
    
    
    

    

    Butterworth A, 
    
    
    
    
    
    

    

    Bruce I, 
    
    
    
    
    
    

    

    Clarkson M, 
    
    
    
    
    
    

    

    Conlon N, 
    
    
    
    
    
    

    

    DasGupta B, 
    
    
    
    
    
    

    

    Doulton T, 
    
    
    
    
    
    

    

    Espígol-Frigolé G, 
    
    
    
    
    
    

    

    Flossmann O, 
    
    
    
    
    
    

    

    Gabrielli A, 
    
    
    
    
    
    

    

    Gasior J, 
    
    
    
    
    
    

    

    Gregorini G, 
    
    
    
    
    
    

    

    Guida G, 
    
    
    
    
    
    

    

    Hernández-Rodríguez J, 
    
    
    
    
    
    

    

    Hruskova Z, 
    
    
    
    
    
    

    

    Hudson A, 
    
    
    
    
    
    

    

    Knight A, 
    
    
    
    
    
    

    

    Lanyon P, 
    
    
    
    
    
    

    

    Luqmani R, 
    
    
    
    
    
    

    

    Magliano M, 
    
    
    
    
    
    

    

    Manfredi A, 
    
    
    
    
    
    

    

    Marguerie C, 
    
    
    
    
    
    

    

    Maritati F, 
    
    
    
    
    
    

    

    Marvisi C, 
    
    
    
    
    
    

    

    McHugh N, 
    
    
    
    
    
    

    

    Molloy E, 
    
    
    
    
    
    

    

    Motyer A, 
    
    
    
    
    
    

    

    Mukhtyar C, 
    
    
    
    
    
    

    

    Padyukov L, 
    
    
    
    
    
    

    

    Pesci A, 
    
    
    
    
    
    

    

    Prieto-Gonzalez S, 
    
    
    
    
    
    

    

    Ramentol-Sintas M, 
    
    
    
    
    
    

    

    Reis P, 
    
    
    
    
    
    

    

    Roccatello D, 
    
    
    
    
    
    

    

    Rovere-Querini P, 
    
    
    
    
    
    

    

    Salvarani C, 
    
    
    
    
    
    

    

    Santarsia F, 
    
    
    
    
    
    

    

    Solans-Laque R, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Taylor J, 
    
    
    
    
    
    

    

    Wessels J, 
    
    
    
    
    
    

    

    Zwerina J, 
    
    
    
    
    
    

    

    Terrier B, 
    
    
    
    
    
    

    

    Watts R, 
    
    
    
    
    
    

    

    Vaglio A, 
    
    
    
    
    
    

    

    Holle J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Smith K, 
    
    
    
    
    
    

    

    The European Vasculitis Genetics Consortium . 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Commun.</em>
    
    
    2019
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/s41467-019-12515-9" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2018/12/10/491837.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Eosinophilic granulomatosis with polyangiitis (EGPA) is a rare inflammatory disease of unknown cause. 30% of patients have anti-neutrophil cytoplasmic antibodies (ANCA) specific for myeloperoxidase (MPO). Here, we describe a genome-wide association study in 676 EGPA cases and 6809 controls, that identifies 4 EGPA-associated loci through conventional case-control analysis, and 4 additional associations through a conditional false discovery rate approach. Many variants are also associated with asthma and six are associated with eosinophil count in the general population. Through Mendelian randomisation, we show that a primary tendency to eosinophilia contributes to EGPA susceptibility. Stratification by ANCA reveals that EGPA comprises two genetically and clinically distinct syndromes. MPO+ ANCA EGPA is an eosinophilic autoimmune disease sharing certain clinical features and an HLA-DQ association with MPO+ ANCA-associated vasculitis, while ANCA-negative EGPA may instead have a mucosal/barrier dysfunction origin. Four candidate genes are targets of therapies in development, supporting their exploration in EGPA.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2019_nath_cytokines">
  <span class="title">Multivariate Genome-Wide Association Analysis of a Cytokine Network Reveals Variants with Widespread Immune, Haematological, and Cardiometabolic Pleiotropy</span>
  <span class="author">
    
    
    
    
    
    

    

    Nath A, 
    
    
    
    
    
    

    

    Ritchie S, 
    
    
    
    
    
    

    

    <b>Grinberg N</b>, 
    
    
    
    
    
    

    

    Tang H, 
    
    
    
    
    
    

    

    Huang Q, 
    
    
    
    
    
    

    

    Teo S, 
    
    
    
    
    
    

    

    Ahola-Olli A, 
    
    
    
    
    
    

    

    Würtz P, 
    
    
    
    
    
    

    

    Havulinna A, 
    
    
    
    
    
    

    

    Santalahti K, 
    
    
    
    
    
    

    

    Pitkänen N, 
    
    
    
    
    
    

    

    Lehtimäki T, 
    
    
    
    
    
    

    

    Kähönen M, 
    
    
    
    
    
    

    

    Lyytikäinen L, 
    
    
    
    
    
    

    

    Raitoharju E, 
    
    
    
    
    
    

    

    Seppälä I, 
    
    
    
    
    
    

    

    Sarin A, 
    
    
    
    
    
    

    

    Ripatti S, 
    
    
    
    
    
    

    

    Palotie A, 
    
    
    
    
    
    

    

    Perola M, 
    
    
    
    
    
    

    

    Viikari J, 
    
    
    
    
    
    

    

    Jalkanen S, 
    
    
    
    
    
    

    

    Maksimow M, 
    
    
    
    
    
    

    

    Salmi M, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Raitakari O, 
    
    
    
    
    
    

    

    Salomaa V, 
    
    
    
    
    
    

    

    Abraham G, 
    
    
    
    
    
    

    

    Kettunen J, 
    
    
    
    
    
    

    

    Inouye M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Am. J. Hum. Genet.</em>
    
    
    2019
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1016/j.ajhg.2019.10.001" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Cytokines are essential regulatory components of the immune system, and their aberrant levels have been linked to many disease states. Despite increasing evidence that cytokines operate in concert, many of the physiological interactions between cytokines, and the shared genetic architecture that underlies them, remain unknown. Here, we aimed to identify and characterize genetic variants with pleiotropic effects on cytokines. Using three population-based cohorts (n = 9,263), we performed multivariate genome-wide association studies (GWAS) for a correlation network of 11 circulating cytokines, then combined our results in meta-analysis. We identified a total of eight loci significantly associated with the cytokine network, of which two (PDGFRB and ABO) had not been detected previously. In addition, conditional analyses revealed a further four secondary signals at three known cytokine loci. Integration, through the use of Bayesian colocalization analysis, of publicly available GWAS summary statistics with the cytokine network associations revealed shared causal variants between the eight cytokine loci and other traits; in particular, cytokine network variants at the ABO, SERPINE2, and ZFPM2 loci showed pleiotropic effects on the production of immune-related proteins, on metabolic traits such as lipoprotein and lipid levels, on blood-cell-related traits such as platelet count, and on disease traits such as coronary artery disease and type 2 diabetes.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2019_asimit_multifinemap">
  <span class="title">Stochastic Search and Joint Fine-Mapping Increases Accuracy and Identifies Previously Unreported Associations in Immune-Mediated Diseases</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Asimit J</b>, 
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    <b>Fortune M</b>, 
    
    
    
    
    
    

    

    <b>Grinberg N</b>, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Commun.</em>
    
    
    2019
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/s41467-019-11271-0" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC6642100" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Thousands of genetic variants are associated with human disease risk, but linkage disequilibrium (LD) hinders fine-mapping the causal variants. Both lack of power, and joint tagging of two or more distinct causal variants by a single non-causal SNP, lead to inaccuracies in fine-mapping, with stochastic search more robust than stepwise. We develop a computationally efficient multinomial fine-mapping (MFM) approach that borrows information between diseases in a Bayesian framework. We show that MFM has greater accuracy than single disease analysis when shared causal variants exist, and negligible loss of precision otherwise. MFM analysis of six immune-mediated diseases reveals causal variants undetected in individual disease analysis, including in IL2RA where we confirm functional effects of multiple causal variants using allele-specific expression in sorted CD4+ T cells from genotype-selected individuals. MFM has the potential to increase fine-mapping resolution in related diseases enabling the identification of associated cellular and molecular phenotypes.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2018</h2>
<ol class="bibliography"><li>

<div id="2018_inshaw_diab_age">
  <span class="title">The Chromosome 6q22.33 Region Is Associated with Age at Diagnosis of Type 1 Diabetes and Disease Risk in Those Diagnosed under 5 Years of Age</span>
  <span class="author">
    
    
    
    
    
    

    

    Inshaw J, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Bottolo L, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetologia</em>
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1007/s00125-017-4440-y" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The genetic risk of type 1 diabetes has been extensively studied. However, the genetic determinants of age at diagnosis (AAD) of type 1 diabetes remain relatively unexplained. Identification of AAD genes and pathways could provide insight into the earliest events in the disease process.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2018_libertini">
  <span class="title">DNA Methylation Oscillation Defines Classes of Enhancers</span>
  <span class="author">
    
    
    
    
    
    

    

    Libertini E, 
    
    
    
    
    
    

    

    Hamoudi R, 
    
    
    
    
    
    

    

    Heath S, 
    
    
    
    
    
    

    

    Lancashire L, 
    
    
    
    
    
    

    

    Garcia A, 
    
    
    
    
    
    

    

    Grassi L, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Javierre B, 
    
    
    
    
    
    

    

    Cairns J, 
    
    
    
    
    
    

    

    Wingett S, 
    
    
    
    
    
    

    

    Paul D, 
    
    
    
    
    
    

    

    Gut M, 
    
    
    
    
    
    

    

    Gut I, 
    
    
    
    
    
    

    

    Martens J, 
    
    
    
    
    
    

    

    Ivliev A, 
    
    
    
    
    
    

    

    Stunnenberg H, 
    
    
    
    
    
    

    

    Frontini M, 
    
    
    
    
    
    

    

    Spivakov M, 
    
    
    
    
    
    

    

    Fraser P, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Beck S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/262212" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2018/10/10/262212.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Understanding the regulatory landscape of human cells requires the integration of genomic and epigenomic maps, capturing combinatorial levels of cell type-specific and invariant activity states. Here, we segmented whole-genome bisulfite sequencing-derived methylomes into consecutive blocks of co-methylation (COMETs) to obtain spatial variation patterns of DNA methylation (DNAm oscillations) integrated with histone modifications and promoter-enhancer interactions derived from promoter capture Hi-C (PCHi-C) sequencing of the same purified blood cells. Mapping DNAm oscillations onto regulatory genome annotation revealed that enhancers are enriched for DNAm hyper-oscillations (&gt;30-fold), where multiple machine learning models support DNAm as predictive of enhancer location. Based on this analysis, we report overall predictive power of 99% for DNAm oscillations, 77.3% for DNaseI, 41% for CGIs, 20% for UMRs and 0% for LMRs, demonstrating the power of DNAm oscillations over other methods for enhancer prediction. Methylomes of activated and non-activated CD4+ T cells indicate that DNAm oscillations exist in both states irrespective of activation; hence they can be used to determine the location of latent enhancers. Our approach advances the identification of tissue-specific regulatory elements and outperforms previous approaches defining enhancer classes based on DNA methylation.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2018_liley_cfdr_control">
  <span class="title">Accurate Error Control in High Dimensional Association Testing Using Conditional False Discovery Rates</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/414318" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2018/09/12/414318.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/jamesliley/cfdr" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <a href="https://github.com/jamesliley/cfdr\textsubscriptcontrol\textsubscriptpipeline" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/jamesliley/cfdr\textsubscriptcontrol\textsubscriptpipeline" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>A common statistical procedure throughout the biomedical sciences is the high-dimensional association study, in which a large number of null hypotheses are tested against a trait of interest. These analyses can generally be strengthened by incorporating an informative covariate, often a set of p-values for tests against some related conditional trait. One method to incorporate such a covariate estimates a quantity called the conditional false discovery rate (cFDR) for each observation. The cFDR is the posterior probability of association given p-value thresholds for both the principal and conditional trait, and can be used quantify evidence that the observation is associated with the principal trait. The cFDR-based method is powerful and intuitive, but the two-dimensional nature of its construction means it is difficult to control the overall false- discovery rate (FDR), and existing attempts to do so are over-conservative. We propose a new method, based on identifying maps from the unit square to the unit interval defined by the estimated cFDR, and splitting observations so that each map is independent of the observations it is used to test. This allows theoretical FDR control at arbitrarily low or high numbers of hypotheses and improves power over the existing method. We demonstrate the new method on two genome-wide association studies in autoimmune disease. This method improves the power of the cFDR while maintaining rigorous FDR control, allowing it to be used in the discovery phase of omics studies with the same confidence as the Benjamini-Hochberg procedure and improving its applicability to different data types.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2018_liley_cfdr_est">
  <span class="title">Improved Consistency in Estimates of Conditional False Discovery Rates Increases Power Relative to Both Existing Methods and Parametric Estimators</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/414326" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2018/09/12/414326.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/jamesliley/cfdr" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <a href="https://github.com/jamesliley/cfdr\textsubscriptestimation\textsubscriptpipeline" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/jamesliley/cfdr\textsubscriptestimation\textsubscriptpipeline" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>A common aim in high-dimensional association studies is the identification of the subset of investigated variables associated with a trait of interest. Using association statistics on the same variables for a second related trait can improve power. An important quantity in such analyses is the conditional false-discovery rate (cFDR), the probability of non-association with the trait of interest given p-value thresholds for both traits. The cFDR can be used for hypothesis testing and as a posterior probability in its own right. In this paper, we propose new estimators for the cFDR based on kernel density estimates and mixture-Gaussian models of effect sizes, the latter also allowing estimation of a local form of cFDR (cfdr). We also propose a general non-parametric improvement to existing estimators based on estimating a posterior probability previously estimated at 1. We find that new estimators have the desirable property of smooth rejection regions, but, unexpectedly, do not improve the power of the method, even when distributional assumptions are true. Furthermore, we find that although the local cfdr represents a theoretically optimal decision boundary, noisiness in its estimation means it is less powerful than corresponding cFDR estimates. We find, however, that the non-parametric adjustment increases power for every estimator. We demonstrate the best method on transcriptome-wide association study datasets for breast and ovarian cancers. The findings from this analysis are of both theoretical and pragmatic interest, giving insight into the nature of cFDR and the behaviour of false-discovery rates in a two-dimensional setting. Our methods allow improved control over the behaviour of the cFDR estimator and improved power in high-dimensional hypothesis testing.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Eijsbouts2018-kq">
  <span class="title">Fine Mapping Chromatin Contacts in Capture Hi-C Data</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Eijsbouts C</b>, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Newcombe P, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>bioRxiv</em>
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Hi-C and capture Hi-C (CHi-C) are used to map physical contacts between chromatin regions in cell nuclei using high-throughput sequencing. Analysis typically proceeds considering the evidence for contacts between each possible pair of fragments independent \ldots</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Liley2018-df">
  <span class="title">Accurate Error Control in High Dimensional Association Testing Using Conditional False Discovery Rates</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/414318" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>A common statistical procedure throughout the biomedical sciences is the high-dimensional association study, in which a large number of null hypotheses are tested against a trait of interest. These analyses can generally be strengthened by incorporating an informative covariate, often a set of p-values for tests against some related 9conditional9 trait. One method to incorporate such a covariate estimates a quantity called the conditional false discovery rate (cFDR) for each observation. The cFDR is the posterior probability of association given p-value thresholds for both the principal and conditional trait, and can be used quantify evidence that the observation is associated with the principal trait. The cFDR-based method is powerful and intuitive, but the two-dimensional nature of its construction means it is difficult to control the overall false- discovery rate (FDR), and existing attempts to do so are over-conservative. We propose a new method, based on identifying maps from the unit square to the unit interval defined by the estimated cFDR, and splitting observations so that each map is independent of the observations it is used to test. This allows theoretical FDR control at arbitrarily low or high numbers of hypotheses and improves power over the existing method. We demonstrate the new method on two genome-wide association studies in autoimmune disease. This method improves the power of the cFDR while maintaining rigorous FDR control, allowing it to be used in the discovery phase of 9omics9 studies with the same confidence as the Benjamini-Hochberg procedure and improving its applicability to different data types.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2018_choy_natcomm">
  <span class="title">Promoter Interactome of Human Embryonic Stem Cell-Derived Cardiomyocytes Connects GWAS Regions to Cardiac Gene Networks</span>
  <span class="author">
    
    
    
    
    
    

    

    Choy M, 
    
    
    
    
    
    

    

    Javierre B, 
    
    
    
    
    
    

    

    Williams S, 
    
    
    
    
    
    

    

    Baross S, 
    
    
    
    
    
    

    

    Liu Y, 
    
    
    
    
    
    

    

    Wingett S, 
    
    
    
    
    
    

    

    Akbarov A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Freire-Pritchett P, 
    
    
    
    
    
    

    

    Rugg-Gunn P, 
    
    
    
    
    
    

    

    Spivakov M, 
    
    
    
    
    
    

    

    Fraser P, 
    
    
    
    
    
    

    

    Keavney B. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Commun.</em>
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/s41467-018-04931-0" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Long-range chromosomal interactions bring distal regulatory elements and promoters together to regulate gene expression in biological processes. By performing promoter capture Hi-C (PCHi-C) on human embryonic stem cell-derived cardiomyocytes (hESC-CMs), we show that such promoter interactions are a key mechanism by which enhancers contact their target genes after hESC-CM differentiation from hESCs. We also show that the promoter interactome of hESC-CMs is associated with expression quantitative trait loci (eQTLs) in cardiac left ventricular tissue; captures the dynamic process of genome reorganisation after hESC-CM differentiation; overlaps genome-wide association study (GWAS) regions associated with heart rate; and identifies new candidate genes in such regions. These findings indicate that regulatory elements in hESC-CMs identified by our approach control gene expression involved in ventricular conduction and rhythm of the heart. The study of promoter interactions in other hESC-derived cell types may be of utility in functional investigation of GWAS-associated regions.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2018_fortune_simgwas">
  <span class="title">simGWAS: A Fast Method for Simulation of Large Scale Case-Control GWAS Summary Statistics</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Fortune M</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Bioinformatics</em>
    
    
    2018
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/bioinformatics/bty898" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="https://www.biorxiv.org/content/early/2018/05/02/313023.full.pdf" target="_blank" class="pure-button preprint">preprint <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/chr1swallace/simgwas" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <a href="https://github.com/chr1swallace/simgwas-paper" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/chr1swallace/simgwas-paper" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Methods for analysis of GWAS summary statistics have encouraged data sharing and democratised the analysis of different diseases. Ideal validation for such methods is application to simulated data, where some "truth" is known. As GWAS increase in size, so does the computational complexity of such evaluations; standard practice repeatedly simulates and analyses genotype data for all individuals in an example study. We have developed a novel method based on an alternative approach, directly simulating GWAS summary data, without individual data as an intermediate step. We mathematically derive the expected statistics for any set of causal variants and their effect sizes, conditional upon control haplotype frequencies (available from public reference datasets). Simulation of GWAS summary output can be conducted independently of sample size by simulating random variates about these expected values. Across a range of scenarios, our method, available as an open source R package, produces very similar output to that from simulating individual genotypes with a substantial gain in speed even for modest sample sizes. Fast simulation of GWAS summary statistics will enable more complete and rapid evaluation of summary statistic methods as well as opening new potential avenues of research in fine mapping and gene set enrichment analysis.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2017</h2>
<ol class="bibliography"><li>

<div id="2017_genbiol_burren">
  <span class="title">Chromosome Contacts in Activated T Cells Identify Autoimmune Disease Candidate Genes</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Rubio García A, 
    
    
    
    
    
    

    

    Javierre B, 
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    Cairns J, 
    
    
    
    
    
    

    

    Cooper N, 
    
    
    
    
    
    

    

    Lambourne J, 
    
    
    
    
    
    

    

    Schofield E, 
    
    
    
    
    
    

    

    Castro Dopico X, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Coulson R, 
    
    
    
    
    
    

    

    Burden F, 
    
    
    
    
    
    

    

    Rowlston S, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Wingett S, 
    
    
    
    
    
    

    

    Frontini M, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Fraser P, 
    
    
    
    
    
    

    

    Spivakov M, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genome Biology</em>
    
    
    2017
    
  <!--  -->
   
  <a href="http://doi.org/10.1186/s13059-017-1285-0" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC5584004" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <a href="https://github.com/chr1swallace/cd4-pchic" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/chr1swallace/cd4-pchic" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2017_ja_ferreira">
  <span class="title">Cells with Treg-Specific FOXP3 Demethylation but Low CD25 Are Prevalent in Autoimmunity</span>
  <span class="author">
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Simons H, 
    
    
    
    
    
    

    

    Thompson W, 
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    Yang X, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Oliveira J, 
    
    
    
    
    
    

    

    Castro Dopico X, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Savinykh N, 
    
    
    
    
    
    

    

    Mashar M, 
    
    
    
    
    
    

    

    Vyse T, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Baxendale H, 
    
    
    
    
    
    

    

    Chandra A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Pekalski M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Journal of autoimmunity</em>
    
    
    2017
    
  <!--  -->
   
  <a href="http://doi.org/10.1016/j.jaut.2017.07.009" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC5656572" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2017_jci_pekalski">
  <span class="title">Neonatal and Adult Recent Thymic Emigrants Produce IL-8 and Express Complement Receptors CR1 and CR2</span>
  <span class="author">
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    García A, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Mashar M, 
    
    
    
    
    
    

    

    Brady J, 
    
    
    
    
    
    

    

    Savinykh N, 
    
    
    
    
    
    

    

    Dopico X, 
    
    
    
    
    
    

    

    Mahmood S, 
    
    
    
    
    
    

    

    Duley S, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Waldron-Lynch F, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Shannon-Lowe C, 
    
    
    
    
    
    

    

    Coles A, 
    
    
    
    
    
    

    

    Jones J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L. 
    
  </span>
  
  <div class="jblocks">
    
    <em>JCI insight</em>
    
    
    2017
    
  <!--  -->
   
  <a href="http://doi.org/10.1172/jci.insight.93739" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC5621870" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2017_ng_liley">
  <span class="title">A Method for Identifying Genetic Heterogeneity within Phenotypically-Defined Disease Subgroups</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2017
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.3751" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC5357574" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/jamesliley/subtest" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Many common diseases show wide phenotypic variation. We present a statistical method for determining whether phenotypically defined subgroups of disease cases represent different genetic pathophysiologies, in which disease-associated variants have different effect sizes in the two subgroups. Our method models the genome-wide distributions of genetic association statistics with mixture Gaussians. We apply a global test without requiring explicit identification of disease-associated variants, thus maximising power in comparison to a standard variant by variant subgroup analysis. Where evidence for genetic subgrouping is found, we present methods for post-hoc identification of the contributing genetic variants. We demonstrate the method on a range of simulated and test datasets where expected results are already known. We investigate subgroups of type 1 diabetes (T1D) cases defined by autoantibody positivity, establishing evidence for differential genetic basis with thyroid peroxidase antibody positivity, driven generally by variants in known T1D associated regions.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Burren2017-ns">
  <span class="title">Chromosome Contacts in Activated T Cells Identify Autoimmune Disease Candidate Genes</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Garcia A, 
    
    
    
    
    
    

    

    Javierre B, 
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    Cairns J, 
    
    
    
    
    
    

    

    Cooper N, 
    
    
    
    
    
    

    

    Lambourne J, 
    
    
    
    
    
    

    

    Schofield E, 
    
    
    
    
    
    

    

    Dopico X, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Coulson R, 
    
    
    
    
    
    

    

    Burden F, 
    
    
    
    
    
    

    

    Rowlston S, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Wingett S, 
    
    
    
    
    
    

    

    Frontini M, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Fraser P, 
    
    
    
    
    
    

    

    Spivakov M, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genome Biol.</em>
    
    
    2017
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1186/s13059-017-1285-0" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Background: Autoimmune disease-associated variants are preferentially found in regulatory regions in immune cells, particularly CD4(+) T cells. Linking such regulatory regions to gene promoters in disease-relevant cell contexts facilitates identification of candidate disease genes. Results: Within 4 h, activation of CD4(+) T cells invokes changes in histone modifications and enhancer RNA transcription that correspond to altered expression of the interacting genes identified by promoter capture Hi-C. By integrating promoter capture Hi-C data with genetic associations for five autoimmune diseases, we prioritised 245 candidate genes with a median distance from peak signal to prioritised gene of 153 kb. Just under half (108/245) prioritised genes related to activation-sensitive interactions. This included IL2RA, where allele-specific expression analyses were consistent with its interaction-mediated regulation, illustrating the utility of the approach. Conclusions: Our systematic experimental framework offers an alternative approach to candidate causal gene identification for variants with cell state-specific functional effects, with achievable sample sizes.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Cooper2017-qa">
  <span class="title">Type 1 Diabetes Genome-Wide Association Analysis with Imputation Identifies Five New Risk Regions</span>
  <span class="author">
    
    
    
    
    
    

    

    Cooper N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    
    2017
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/120022" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Type 1 diabetes genotype datasets have undergone several well powered genome wide analysis studies (GWAS), identifying 57 associated regions at the time of analysis. There are still many regions of smaller effect size or low frequency left to discover, and better exploitation of existing type 1 diabetes cohorts with meta analysis and imputation can precede the acquisition of new or larger cohorts. An existing dataset of 5,913 case and 8,829 control samples was analysed using genome-wide microarrays (Affymetrix GeneChip 500K and Illumina Infinium 550K) with imputation via IMPUTE2 with the 1000 Genomes Project (phase 3) reference panel. Genotyping coverage was doubled in known association regions, and increased by four fold in other regions compared to previous studies. Our analysis resulted in new index variants for 17/57 regions, an expanded set of plausible candidate SNPs for 17 regions, and five novel type 1 diabetes association regions at 1p31.3, 1q24.3, 1q31.2, 2q11.2 and 11q12.2. Candidate genes for the new loci included ITGB3BP, FASLG, RGS1, AFF3 and CD5/CD6. Further prioritisation of causal genes and causal variants will require detailed RNA and protein expression studies, in conjunction with genome annotation studies including analysis of physical promoter-enhancer interactions.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Khadjesari2017-ag">
  <span class="title">Validation of the AUDIT-C in Adults Seeking Help with Their Drinking Online</span>
  <span class="author">
    
    
    
    
    
    

    

    Khadjesari Z, 
    
    
    
    
    
    

    

    White I, 
    
    
    
    
    
    

    

    McCambridge J, 
    
    
    
    
    
    

    

    Marston L, 
    
    
    
    
    
    

    

    <b>Wallace P</b>, 
    
    
    
    
    
    

    

    Godfrey C, 
    
    
    
    
    
    

    

    Murray E. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Addict. Sci. Clin. Pract.</em>
    
    
    2017
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1186/s13722-016-0066-5" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/5209877" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>BACKGROUND: The abbreviated Alcohol Use Disorder Identification Test for Consumption (AUDIT-C) is rapidly becoming the alcohol screening tool of choice for busy practitioners in clinical settings and by researchers keen to limit assessment burden and reactivity. Cut-off scores for detecting drinking above recommended limits vary by population, setting, country and potentially format. This validation study aimed to determine AUDIT-C thresholds that indicated risky drinking among a population of people seeking help over the Internet. METHOD: The data in this study were collected in the pilot phase of the Down Your Drink trial, which recruited people seeking help over the Internet and randomised them to a web-based intervention or an information-only website. Sensitivity, specificity, and positive and negative likelihood ratios were calculated for AUDIT-C scores, relative to weekly consumption that indicated drinking above limits and higher risk drinking. Receiver-operating characteristic (ROC) curves were created to assess the performance of different cut-off scores on the AUDIT-C for men and women. Past week alcohol consumption was used as the reference-standard and was collected via the TOT-AL, a validated online measure of past week drinking. RESULTS: AUDIT-C scores were obtained from 3720 adults (2053 female and 1667 male) searching the internet for help with drinking, mostly from the UK. The area under the ROC curve for risky drinking was 0.84 (95% CI 0.80, 0.87) (female) and 0.80 (95% CI 0.76, 0.84) (male). AUDIT-C cut-off scores for detecting risky drinking that maximise the sum of sensitivity and specificity were >/=8 for women and >/=8 for men; whereas those identifying the highest proportion of correctly classified individuals were >/=4 for women and >/=5 for men. AUDIT-C cut-off scores for detecting higher risk drinking were also calculated. CONCLUSIONS: AUDIT-C cut-off scores for identifying alcohol consumption above weekly limits in this largely UK based study population were substantially higher than those reported in other validation studies. Researchers and practitioners should select AUDIT-C cut-off scores according to the purpose of identifying risky drinkers and hence the relative importance of sensitivity and/or specificity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Liley2017-od">
  <span class="title">A Method for Identifying Genetic Heterogeneity within Phenotypically Defined Disease Subgroups</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Genet.</em>
    
    
    2017
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.3751" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Many common diseases show wide phenotypic variation. We present a statistical method for determining whether phenotypically defined subgroups of disease cases represent different genetic architectures, in which disease-associated variants have different effect sizes in two subgroups. Our method models the genome-wide distributions of genetic association statistics with mixture Gaussians. We apply a global test without requiring explicit identification of disease-associated variants, thus maximizing power in comparison to standard variant-by-variant subgroup analysis. Where evidence for genetic subgrouping is found, we present methods for post hoc identification of the contributing genetic variants. We demonstrate the method on a range of simulated and test data sets, for which expected results are already known. We investigate subgroups of individuals with type 1 diabetes (T1D) defined by autoantibody positivity, establishing evidence for differential genetic architecture with positivity for thyroid-peroxidase-specific antibody, driven generally by variants in known T1D-associated genomic regions.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="marigorta_transcriptional_2017">
  <span class="title">Transcriptional Risk Scores Link GWAS to eQTLs and Predict Complications in Crohn’s Disease</span>
  <span class="author">
    
    
    
    
    
    

    

    Marigorta U, 
    
    
    
    
    
    

    

    Denson L, 
    
    
    
    
    
    

    

    Hyams J, 
    
    
    
    
    
    

    

    Mondal K, 
    
    
    
    
    
    

    

    Prince J, 
    
    
    
    
    
    

    

    Walters T, 
    
    
    
    
    
    

    

    Griffiths A, 
    
    
    
    
    
    

    

    Noe J, 
    
    
    
    
    
    

    

    Crandall W, 
    
    
    
    
    
    

    

    Rosh J, 
    
    
    
    
    
    

    

    Mack D, 
    
    
    
    
    
    

    

    Kellermayer R, 
    
    
    
    
    
    

    

    Heyman M, 
    
    
    
    
    
    

    

    Baker S, 
    
    
    
    
    
    

    

    Stephens M, 
    
    
    
    
    
    

    

    Baldassano R, 
    
    
    
    
    
    

    

    Markowitz J, 
    
    
    
    
    
    

    

    Kim M, 
    
    
    
    
    
    

    

    Dubinsky M, 
    
    
    
    
    
    

    

    Cho J, 
    
    
    
    
    
    

    

    Aronow B, 
    
    
    
    
    
    

    

    Kugathasan S, 
    
    
    
    
    
    

    

    Gibson G. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nature Genetics</em>
    
    
    2017
    
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.3936" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="Rainbow2017-es">
  <span class="title">A Rare IL2RA Haplotype Identifies SNP Rs61839660 as Causal for Autoimmunity</span>
  <span class="author">
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wicker L. 
    
  </span>
  
  <div class="jblocks">
    
    
    2017
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/108126" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>IL2RA is associated with multiple autoimmune diseases including type 1 diabetes (T1D). Higher expression of IL2RA mRNA and its protein product CD25 in T lymphocytes is associated with a T1D-protective haplotype. Here we show that a rare variation of this haplotype that loses the protective allele at a single SNP, rs61839660, reduces IL2RA expression and T1D protection, identifying it as the causal factor in disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2016</h2>
<ol class="bibliography"><li>

<div id="2016_plosmed_dilt1d">
  <span class="title">Regulatory T Cell Responses in Participants with Type 1 Diabetes after a Single Dose of Interleukin-2: A Non-Randomised, Open Label, Adaptive Dose-Finding Trial</span>
  <span class="author">
    
    
    
    
    
    

    

    Todd, John A. AND Evangelou, Marina AND Cutler, Antony J. AND Pekalski, Marcin L. AND Walker, Neil M. AND Stevens, Helen E. AND Porter, Linsey AND Smyth, Deborah J. AND Rainbow, Daniel B. AND Ferreira, Ricardo C. AND Esposito, Laura AND Hunter, Kara M. D. AND Loudon, Kevin AND Irons, Kathryn AND Yang, Jennie H. AND Bell, Charles J. M. AND Schuilenburg, Helen AND Heywood, James AND Challis, Ben AND Neupane, Sankalpa AND Clarke, Pamela AND Coleman, Gillian AND Dawson, Sarah AND Goymer, Donna AND Anselmiova, Katerina AND Kennet, Jane AND Brown, Judy AND Caddy, Sarah L. AND Lu, Jia AND Greatorex, Jane AND Goodfellow, Ian AND C Wallace AND Tree, Tim I. AND Evans, Mark AND Mander, Adrian P. AND Bond, Simon AND Wicker, Linda S. AND Waldron-Lynch, Frank . 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLOS Medicine</em>
    
    
    2016
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pmed.1002139" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Frank Waldron-Lynch and colleagues investigate immune cell biomarker responses in patients with type 1 diabetes following a single dose of recombinant interleukin-2.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Liley2016-ty">
  <span class="title">A Genetic Test for Differential Causative Pathology in Disease Subgroups</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    
    2016
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/037713" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Many common diseases show wide clinical or phenotypic variation. We present a statistical method for determining whether phenotypically defined subgroups of disease cases represent different genetic pathophysiologies, in which disease-associated variants have different effect sizes. Our method models the genomewide distributions of genetic association statistics with mixture Gaussians. We test for differential genetic bases without requiring explicit identification of disease-associated variants, maximizing power compared with standard variant-by-variant analyses. Where evidence for genetic subgroups is found, we present methods for subsequent identification of the contributing genetic variants. We demonstrate the method on simulated and test datasets where expected results are already known. We investigate subgroups of type 1 diabetes (T1D) cases defined by autoantibody positivity, establishing evidence for differential genetic basis with thyroid peroxidase antibody positivity. Our method determines the existence of a genetic basis for disease heterogeneity, enabling genomics to inform the development of precision medicine.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Todd2016-sm">
  <span class="title">Regulatory T Cell Responses in Participants with Type 1 Diabetes after a Single Dose of Interleukin-2: A Non-Randomised, Open Label, Adaptive Dose-Finding Trial</span>
  <span class="author">
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Evangelou M, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Porter L, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Esposito L, 
    
    
    
    
    
    

    

    Hunter K, 
    
    
    
    
    
    

    

    Loudon K, 
    
    
    
    
    
    

    

    Irons K, 
    
    
    
    
    
    

    

    Yang J, 
    
    
    
    
    
    

    

    Bell C, 
    
    
    
    
    
    

    

    Schuilenburg H, 
    
    
    
    
    
    

    

    Heywood J, 
    
    
    
    
    
    

    

    Challis B, 
    
    
    
    
    
    

    

    Neupane S, 
    
    
    
    
    
    

    

    Clarke P, 
    
    
    
    
    
    

    

    <b>Coleman G</b>, 
    
    
    
    
    
    

    

    Dawson S, 
    
    
    
    
    
    

    

    Goymer D, 
    
    
    
    
    
    

    

    Anselmiova K, 
    
    
    
    
    
    

    

    Kennet J, 
    
    
    
    
    
    

    

    Brown J, 
    
    
    
    
    
    

    

    Caddy S, 
    
    
    
    
    
    

    

    Lu J, 
    
    
    
    
    
    

    

    Greatorex J, 
    
    
    
    
    
    

    

    Goodfellow I, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Tree T, 
    
    
    
    
    
    

    

    Evans M, 
    
    
    
    
    
    

    

    Mander A, 
    
    
    
    
    
    

    

    Bond S, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Waldron-Lynch F. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Med.</em>
    
    
    2016
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pmed.1002139" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/5058548" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>BACKGROUND: Interleukin-2 (IL-2) has an essential role in the expansion and function of CD4+ regulatory T cells (Tregs). Tregs reduce tissue damage by limiting the immune response following infection and regulate autoreactive CD4+ effector T cells (Teffs) to prevent autoimmune diseases, such as type 1 diabetes (T1D). Genetic susceptibility to T1D causes alterations in the IL-2 pathway, a finding that supports Tregs as a cellular therapeutic target. Aldesleukin (Proleukin; recombinant human IL-2), which is administered at high doses to activate the immune system in cancer immunotherapy, is now being repositioned to treat inflammatory and autoimmune disorders at lower doses by targeting Tregs. METHODS AND FINDINGS: To define the aldesleukin dose response for Tregs and to find doses that increase Tregs physiologically for treatment of T1D, a statistical and systematic approach was taken by analysing the pharmacokinetics and pharmacodynamics of single doses of subcutaneous aldesleukin in the Adaptive Study of IL-2 Dose on Regulatory T Cells in Type 1 Diabetes (DILT1D), a single centre, non-randomised, open label, adaptive dose-finding trial with 40 adult participants with recently diagnosed T1D. The primary endpoint was the maximum percentage increase in Tregs (defined as CD3+CD4+CD25highCD127low) from the baseline frequency in each participant measured over the 7 d following treatment. There was an initial learning phase with five pairs of participants, each pair receiving one of five pre-assigned single doses from 0.04 x 106 to 1.5 x 106 IU/m2, in order to model the dose-response curve. Results from each participant were then incorporated into interim statistical modelling to target the two doses most likely to induce 10% and 20% increases in Treg frequencies. Primary analysis of the evaluable population (n = 39) found that the optimal doses of aldesleukin to induce 10% and 20% increases in Tregs were 0.101 x 106 IU/m2 (standard error [SE] = 0.078, 95% CI = -0.052, 0.254) and 0.497 x 106 IU/m2 (SE = 0.092, 95% CI = 0.316, 0.678), respectively. On analysis of secondary outcomes, using a highly sensitive IL-2 assay, the observed plasma concentrations of the drug at 90 min exceeded the hypothetical Treg-specific therapeutic window determined in vitro (0.015-0.24 IU/ml), even at the lowest doses (0.040 x 106 and 0.045 x 106 IU/m2) administered. A rapid decrease in Treg frequency in the circulation was observed at 90 min and at day 1, which was dose dependent (mean decrease 11.6%, SE = 2.3%, range 10.0%-48.2%, n = 37), rebounding at day 2 and increasing to frequencies above baseline over 7 d. Teffs, natural killer cells, and eosinophils also responded, with their frequencies rapidly and dose-dependently decreased in the blood, then returning to, or exceeding, pretreatment levels. Furthermore, there was a dose-dependent down modulation of one of the two signalling subunits of the IL-2 receptor, the beta chain (CD122) (mean decrease = 58.0%, SE = 2.8%, range 9.8%-85.5%, n = 33), on Tregs and a reduction in their sensitivity to aldesleukin at 90 min and day 1 and 2 post-treatment. Due to blood volume requirements as well as ethical and practical considerations, the study was limited to adults and to analysis of peripheral blood only. CONCLUSIONS: The DILT1D trial results, most notably the early altered trafficking and desensitisation of Tregs induced by a single ultra-low dose of aldesleukin that resolves within 2-3 d, inform the design of the next trial to determine a repeat dosing regimen aimed at establishing a steady-state Treg frequency increase of 20%-50%, with the eventual goal of preventing T1D. TRIAL REGISTRATION: ISRCTN Registry ISRCTN27852285; ClinicalTrials.gov NCT01827735.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2016_cell">
  <span class="title">Lineage-Specific Genome Architecture Links Enhancers and Non-Coding Disease Variants to Target Gene Promoters</span>
  <span class="author">
    
    
    
    
    
    

    

    Javierre B, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Wilder S, 
    
    
    
    
    
    

    

    Kreuzhuber R, 
    
    
    
    
    
    

    

    Hill S, 
    
    
    
    
    
    

    

    Sewitz S, 
    
    
    
    
    
    

    

    Cairns J, 
    
    
    
    
    
    

    

    Wingett S, 
    
    
    
    
    
    

    

    Várnai C, 
    
    
    
    
    
    

    

    Thiecke M, 
    
    
    
    
    
    

    

    Burden F, 
    
    
    
    
    
    

    

    Farrow S, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Rehnström K, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Grassi L, 
    
    
    
    
    
    

    

    Kostadima M, 
    
    
    
    
    
    

    

    Freire-Pritchett P, 
    
    
    
    
    
    

    

    Wang F, 
    
    
    
    
    
    

    

    BLUEPRINT Consortium , 
    
    
    
    
    
    

    

    Stunnenberg H, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Zerbino D, 
    
    
    
    
    
    

    

    Stegle O, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Frontini M, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Spivakov M, 
    
    
    
    
    
    

    

    Fraser P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Cell</em>
    
    
    2016
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1016/j.cell.2016.09.037" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC5123897" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <a href="https://github.com/ollyburren/CHIGP" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/ollyburren/CHIGP" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Long-range interactions between regulatory elements and gene promoters play key roles in transcriptional regulation. The vast majority of interactions are uncharted, constituting a major missing link in understanding genome control. Here, we use promoter capture Hi-C to identify interacting regions of 31,253 promoters in 17 human primary hematopoietic cell types. We show that promoter interactions are highly cell type specific and enriched for links between active promoters and epigenetically marked enhancers. Promoter interactomes reflect lineage relationships of the hematopoietic tree, consistent with dynamic remodeling of nuclear architecture during differentiation. Interacting regions are enriched in genetic variants linked with altered expression of genes they contact, highlighting their functional role. We exploit this rich resource to connect non-coding disease variants to putative target promoters, prioritizing thousands of disease-candidate genes and implicating disease pathways. Our results demonstrate the power of primary cell promoter interactomes to reveal insights into genomic regulatory mechanisms underlying common diseases.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2015</h2>
<ol class="bibliography"><li>

<div id="2015_castro">
  <span class="title">Widespread Seasonal Gene Expression Reveals Annual Differences in Human Immunity and Physiology.</span>
  <span class="author">
    
    
    
    
    
    

    

    Castro Dopico X, 
    
    
    
    
    
    

    

    Evangelou M, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Ziegler A, 
    
    
    
    
    
    

    

    Bonifacio E, 
    
    
    
    
    
    

    

    <b>*Wallace C</b>, 
    
    
    
    
    
    

    

    *Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Commun</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ncomms8000" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4432600" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Seasonal variations are rarely considered a contributing component to human tissue function or health, although many diseases and physiological process display annual periodicities. Here we find more than 4,000 protein-coding mRNAs in white blood cells and adipose tissue to have seasonal expression profiles, with inverted patterns observed between Europe and Oceania. We also find the cellular composition of blood to vary by season, and these changes, which differ between the United Kingdom and The Gambia, could explain the gene expression periodicity. With regards to tissue function, the immune system has a profound pro-inflammatory transcriptomic profile during European winter, with increased levels of soluble IL-6 receptor and C-reactive protein, risk biomarkers for cardiovascular, psychiatric and autoimmune diseases that have peak incidences in winter. Circannual rhythms thus require further exploration as contributors to various aspects of human physiology and disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_diab_yang">
  <span class="title">Natural Variation in Interleukin-2 Sensitivity Influences Regulatory T-Cell Frequency and Function in Individuals With Long-Standing Type 1 Diabetes</span>
  <span class="author">
    
    
    
    
    
    

    

    Yang J, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Reading J, 
    
    
    
    
    
    

    

    Cooper N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Clarke P, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Boyce C, 
    
    
    
    
    
    

    

    Gao G, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Tree T. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetes</em>
    
    
    2015
    
  <!--  -->
   
  <a href="http://doi.org/10.2337/db15-0516" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4975524" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_ng_fortune">
  <span class="title">Statistical Colocalization of Genetic Risk Variants for Related Autoimmune Diseases in the Context of Common Controls.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Fortune M</b>, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Schofield E, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Ban M, 
    
    
    
    
    
    

    

    Sawcer S, 
    
    
    
    
    
    

    

    Bowes J, 
    
    
    
    
    
    

    

    Worthington J, 
    
    
    
    
    
    

    

    Barton A, 
    
    
    
    
    
    

    

    Eyre S, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.3330" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4754941" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/chr1swallace/colocCommonControl" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Determining whether potential causal variants for related diseases are shared can identify overlapping etiologies of multifactorial disorders. Colocalization methods disentangle shared and distinct causal variants. However, existing approaches require independent data sets. Here we extend two colocalization methods to allow for the shared-control design commonly used in comparison of genome-wide association study results across diseases. Our analysis of four autoimmune diseases-type 1 diabetes (T1D), rheumatoid arthritis, celiac disease and multiple sclerosis-identified 90 regions that were associated with at least one disease, 33 (37%) of which were associated with 2 or more disorders. Nevertheless, for 14 of these 33 shared regions, there was evidence that the causal variants differed. We identified new disease associations in 11 regions previously associated with one or more of the other 3 disorders. Four of eight T1D-specific regions contained known type 2 diabetes (T2D) candidate genes (COBL, GLIS3, RNLS and BCAR1), suggesting a shared cellular etiology.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Bowes2015-xm">
  <span class="title">Corrigendum: Dense Genotyping of Immune-Related Susceptibility Loci Reveals New Insights into the Genetics of Psoriatic Arthritis</span>
  <span class="author">
    
    
    
    
    
    

    

    Bowes J, 
    
    
    
    
    
    

    

    Budu-Aggrey A, 
    
    
    
    
    
    

    

    Huffmeier U, 
    
    
    
    
    
    

    

    Uebe S, 
    
    
    
    
    
    

    

    Steel K, 
    
    
    
    
    
    

    

    Hebert H, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Massey J, 
    
    
    
    
    
    

    

    Bruce I, 
    
    
    
    
    
    

    

    Bluett J, 
    
    
    
    
    
    

    

    Feletar M, 
    
    
    
    
    
    

    

    Morgan A, 
    
    
    
    
    
    

    

    Marzo-Ortega H, 
    
    
    
    
    
    

    

    Donohoe G, 
    
    
    
    
    
    

    

    Morris D, 
    
    
    
    
    
    

    

    Helliwell P, 
    
    
    
    
    
    

    

    Ryan A, 
    
    
    
    
    
    

    

    Kane D, 
    
    
    
    
    
    

    

    Warren R, 
    
    
    
    
    
    

    

    Korendowych E, 
    
    
    
    
    
    

    

    Alenius G, 
    
    
    
    
    
    

    

    Giardina E, 
    
    
    
    
    
    

    

    Packham J, 
    
    
    
    
    
    

    

    McManus R, 
    
    
    
    
    
    

    

    FitzGerald O, 
    
    
    
    
    
    

    

    McHugh N, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Ho P, 
    
    
    
    
    
    

    

    Behrens F, 
    
    
    
    
    
    

    

    Burkhardt H, 
    
    
    
    
    
    

    

    Reis A, 
    
    
    
    
    
    

    

    Barton A. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Commun.</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ncomms8741" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Bowes, John Budu-Aggrey, Ashley Huffmeier, Ulrike Uebe, Steffen Steel, Kathryn Hebert, Harry L Wallace, Chris Massey, Jonathon Bruce, Ian N Bluett, James Feletar, Marie Morgan, Ann W Marzo-Ortega, Helena Donohoe, Gary Morris, Derek W Helliwell, Philip Ryan, Anthony W Kane, David Warren, Richard B Korendowych, Eleanor Alenius, Gerd-Marie Giardina, Emiliano Packham, Jonathan McManus, Ross FitzGerald, Oliver McHugh, Neil Brown, Matthew A Ho, Pauline Behrens, Frank Burkhardt, Harald Reis, Andre Barton, Anne eng Published Erratum England 2015/07/07 06:00 Nat Commun. 2015 Jul 6;6:7741. doi: 10.1038/ncomms8741.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Dopico2015-sg">
  <span class="title">Widespread Seasonal Gene Expression Reveals Annual Differences in Human Immunity and Physiology</span>
  <span class="author">
    
    
    
    
    
    

    

    Dopico X, 
    
    
    
    
    
    

    

    Evangelou M, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Cooper N, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Fulford A, 
    
    
    
    
    
    

    

    Hennig B, 
    
    
    
    
    
    

    

    Prentice A, 
    
    
    
    
    
    

    

    Ziegler A, 
    
    
    
    
    
    

    

    Bonifacio E, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Commun.</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ncomms8000" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4432600" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Seasonal variations are rarely considered a contributing component to human tissue function or health, although many diseases and physiological process display annual periodicities. Here we find more than 4,000 protein-coding mRNAs in white blood cells and adipose tissue to have seasonal expression profiles, with inverted patterns observed between Europe and Oceania. We also find the cellular composition of blood to vary by season, and these changes, which differ between the United Kingdom and The Gambia, could explain the gene expression periodicity. With regards to tissue function, the immune system has a profound pro-inflammatory transcriptomic profile during European winter, with increased levels of soluble IL-6 receptor and C-reactive protein, risk biomarkers for cardiovascular, psychiatric and autoimmune diseases that have peak incidences in winter. Circannual rhythms thus require further exploration as contributors to various aspects of human physiology and disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Fortune2015-sc">
  <span class="title">Corrigendum: Statistical Colocalization of Genetic Risk Variants for Related Autoimmune Diseases in the Context of Common Controls</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Fortune M</b>, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Schofield E, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Ban M, 
    
    
    
    
    
    

    

    Sawcer S, 
    
    
    
    
    
    

    

    Bowes J, 
    
    
    
    
    
    

    

    Worthington J, 
    
    
    
    
    
    

    

    Barton A, 
    
    
    
    
    
    

    

    Eyre S, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Genet.</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng0815-962c" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Fortune, Mary D Guo, Hui Burren, Oliver Schofield, Ellen Walker, Neil M Ban, Maria Sawcer, Stephen J Bowes, John Worthington, Jane Barton, Anne Eyre, Steve Todd, John A Wallace, Chris eng 100140/Wellcome Trust/United Kingdom Published Erratum 2015/07/30 06:00 Nat Genet. 2015 Aug;47(8):962. doi: 10.1038/ng0815-962c.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_wallace_il2ra">
  <span class="title">Dissection of a Complex Disease Susceptibility Region Using a Bayesian Stochastic Search Approach to Fine Mapping</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Pontikos N, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Garcia A, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Rich S, 
    
    
    
    
    
    

    

    Onengut-Gumuscu S, 
    
    
    
    
    
    

    

    Sawcer S, 
    
    
    
    
    
    

    

    Ban M, 
    
    
    
    
    
    

    

    Richardson S, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet.</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1005272" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4481316" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <a href="https://github.com/chr1swallace/GUESSFM" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/chr1swallace/GUESSFM" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Identification of candidate causal variants in regions associated with risk of common diseases is complicated by linkage disequilibrium (LD) and multiple association signals. Nonetheless, accurate maps of these variants are needed, both to fully exploit detailed cell specific chromatin annotation data to highlight disease causal mechanisms and cells, and for design of the functional studies that will ultimately be required to confirm causal mechanisms. We adapted a Bayesian evolutionary stochastic search algorithm to the fine mapping problem, and demonstrated its improved performance over conventional stepwise and regularised regression through simulation studies. We then applied it to fine map the established multiple sclerosis (MS) and type 1 diabetes (T1D) associations in the IL-2RA (CD25) gene region. For T1D, both stepwise and stochastic search approaches identified four T1D association signals, with the major effect tagged by the single nucleotide polymorphism, rs12722496. In contrast, for MS, the stochastic search found two distinct competing models: a single candidate causal variant, tagged by rs2104286 and reported previously using stepwise analysis; and a more complex model with two association signals, one of which was tagged by the major T1Dassociated rs12722496 and the other by rs56382813. There is low to moderate LD between rs2104286 and both rs12722496 and rs56382813 (r(2) similar or equal to 0: 3) and our two SNP model could not be recovered through a forward stepwise search after conditioning on rs2104286. Both signals in the two variant model for MS affect CD25 expression on distinct subpopulations of CD4(+) T cells, which are key cells in the autoimmune process. The results support a shared causal variant for T1D and MS. Our study illustrates the benefit of using a purposely designed model search strategy for fine mapping and the advantage of combining disease and protein expression data.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_nc_bowes">
  <span class="title">Dense Genotyping of Immune-Related Susceptibility Loci Reveals New Insights into the Genetics of Psoriatic Arthritis.</span>
  <span class="author">
    
    
    
    
    
    

    

    Bowes J, 
    
    
    
    
    
    

    

    Budu-Aggrey A, 
    
    
    
    
    
    

    

    Huffmeier U, 
    
    
    
    
    
    

    

    Uebe S, 
    
    
    
    
    
    

    

    Steel K, 
    
    
    
    
    
    

    

    Hebert H, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Massey J, 
    
    
    
    
    
    

    

    Bruce I, 
    
    
    
    
    
    

    

    Bluett J, 
    
    
    
    
    
    

    

    Feletar M, 
    
    
    
    
    
    

    

    Morgan A, 
    
    
    
    
    
    

    

    Marzo-Ortega H, 
    
    
    
    
    
    

    

    Donohoe G, 
    
    
    
    
    
    

    

    Morris D, 
    
    
    
    
    
    

    

    Helliwell P, 
    
    
    
    
    
    

    

    Ryan A, 
    
    
    
    
    
    

    

    Kane D, 
    
    
    
    
    
    

    

    Warren R, 
    
    
    
    
    
    

    

    Korendowych E, 
    
    
    
    
    
    

    

    Alenius G, 
    
    
    
    
    
    

    

    Giardina E, 
    
    
    
    
    
    

    

    Packham J, 
    
    
    
    
    
    

    

    McManus R, 
    
    
    
    
    
    

    

    FitzGerald O, 
    
    
    
    
    
    

    

    McHugh N, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Ho P, 
    
    
    
    
    
    

    

    Behrens F, 
    
    
    
    
    
    

    

    Burkhardt H, 
    
    
    
    
    
    

    

    Reis A, 
    
    
    
    
    
    

    

    Barton A. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Commun</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ncomms7046" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4327416" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <a href="https://github.com/chr1swallace/finemap-psa" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/chr1swallace/finemap-psa" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Psoriatic arthritis (PsA) is a chronic inflammatory arthritis associated with psoriasis and, despite the larger estimated heritability for PsA, the majority of genetic susceptibility loci identified to date are shared with psoriasis. Here, we present results from a case-control association study on 1,962 PsA patients and 8,923 controls using the Immunochip genotyping array. We identify eight loci passing genome-wide significance, secondary independent effects at three loci and a distinct PsA-specific variant at the IL23R locus. We report two novel loci and evidence of a novel PsA-specific association at chromosome 5q31. Imputation of classical HLA alleles, amino acids and SNPs across the MHC region highlights three independent associations to class I genes. Finally, we find an enrichment of associated variants to markers of open chromatin in CD8(+) memory primary T cells. This study identifies key insights into the genetics of PsA that could begin to explain fundamental differences between psoriasis and PsA.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_hmg_guo">
  <span class="title">Integration of Disease Association and eQTL Data Using a Bayesian Colocalisation Approach Highlights Six Candidate Causal Genes in Immune-Mediated Diseases.</span>
  <span class="author">
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    <b>Fortune M</b>, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Schofield E, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hum Mol Genet</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/hmg/ddv077" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4498151" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The genes and cells that mediate genetic associations identified through genome-wide association studies (GWAS) are only partially understood. Several studies that have investigated the genetic regulation of gene expression have shown that disease-associated variants are over-represented amongst expression quantitative trait loci (eQTL) variants. Evidence for colocalisation of eQTL and disease causal variants can suggest causal genes and cells for these genetic associations. Here, we used colocalisation analysis to investigate whether 595 genetic associations to ten immune-mediated diseases are consistent with a causal variant that regulates, in cis, gene expression in resting B cells, and in resting and stimulated monocytes. Previously published candidate causal genes were over-represented amongst genes exhibiting colocalisation (odds ratio > 1.5), and we identified evidence for colocalisation (posterior odds > 5) between cis eQTLs in at least one cell type and at least one disease for six genes: ADAM15, RGS1, CARD9, LTBR, CTSH and SYNGR1. We identified cell-specific effects, such as for CTSH, the expression of which in monocytes, but not in B cells, may mediate type 1 diabetes and narcolepsy associations in the chromosome 15q25.1 region. Our results demonstrate the utility of integrating genetic studies of disease and gene expression for highlighting causal genes and cell types.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_nc_martin">
  <span class="title">Capture Hi-C Reveals Novel Candidate Genes and Complex Long-Range Interactions with Related Autoimmune Risk Loci</span>
  <span class="author">
    
    
    
    
    
    

    

    Martin P, 
    
    
    
    
    
    

    

    McGovern A, 
    
    
    
    
    
    

    

    Orozco G, 
    
    
    
    
    
    

    

    Duffus K, 
    
    
    
    
    
    

    

    Yarwood A, 
    
    
    
    
    
    

    

    Schoenfelder S, 
    
    
    
    
    
    

    

    Cooper N, 
    
    
    
    
    
    

    

    Barton A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Fraser P, 
    
    
    
    
    
    

    

    Worthington J, 
    
    
    
    
    
    

    

    Eyre S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Commun.</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ncomms10069" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4674669" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <a href="https://github.com/chr1swallace/Martin-2015-Figure3" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/chr1swallace/Martin-2015-Figure3" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome-wide association studies have been tremendously successful in identifying genetic variants associated with complex diseases. The majority of association signals are intergenic and evidence is accumulating that a high proportion of signals lie in enhancer regions. We use Capture Hi-C to investigate, for the first time, the interactions between associated variants for four autoimmune diseases and their functional targets in B- and T-cell lines. Here we report numerous looping interactions and provide evidence that only a minority of interactions are common to both B- and T-cell lines, suggesting interactions may be highly cell-type specific; some disease-associated SNPs do not interact with the nearest gene but with more compelling candidate genes (for example, FOXO1, AZI2) often situated several megabases away; and finally, regions associated with different autoimmune diseases interact with each other and the same promoter suggesting common autoimmune gene targets (for example, PTPRC, DEXI and ZFP36L1).</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_ng_onengut">
  <span class="title">Fine Mapping of Type 1 Diabetes Susceptibility Loci and Evidence for Colocalization of Causal Variants with Lymphoid Gene Enhancers.</span>
  <span class="author">
    
    
    
    
    
    

    

    Onengut-Gumuscu S, 
    
    
    
    
    
    

    

    Chen W, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Cooper N, 
    
    
    
    
    
    

    

    Quinlan A, 
    
    
    
    
    
    

    

    Mychaleckyj J, 
    
    
    
    
    
    

    

    Farber E, 
    
    
    
    
    
    

    

    Bonnie J, 
    
    
    
    
    
    

    

    Szpak M, 
    
    
    
    
    
    

    

    Schofield E, 
    
    
    
    
    
    

    

    Achuthan P, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    <b>Fortune M</b>, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Ward L, 
    
    
    
    
    
    

    

    Kundaje A, 
    
    
    
    
    
    

    

    Kellis M, 
    
    
    
    
    
    

    

    Daly M, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Type 1 Diabetes Genetics Consortium , 
    
    
    
    
    
    

    

    *Todd J, 
    
    
    
    
    
    

    

    <b>*Wallace C</b>, 
    
    
    
    
    
    

    

    *Concannon P, 
    
    
    
    
    
    

    

    *Rich S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.3245" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4380767" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <a href="https://doi.org/10.6084/m9.figshare.827246" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://doi.org/10.6084/m9.figshare.827246" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genetic studies of type 1 diabetes (T1D) have identified 50 susceptibility regions, finding major pathways contributing to risk, with some loci shared across immune disorders. To make genetic comparisons across autoimmune disorders as informative as possible, a dense genotyping array, the Immunochip, was developed, from which we identified four new T1D-associated regions (P < 5 \texttimes 10(-8)). A comparative analysis with 15 immune diseases showed that T1D is more similar genetically to other autoantibody-positive diseases, significantly most similar to juvenile idiopathic arthritis and significantly least similar to ulcerative colitis, and provided support for three additional new T1D risk loci. Using a Bayesian approach, we defined credible sets for the T1D-associated SNPs. The associated SNPs localized to enhancer sequences active in thymus, T and B cells, and CD34(+) stem cells. Enhancer-promoter interactions can now be analyzed in these cell types to identify which particular genes and regulatory sequences are causal.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_diab_ferreira">
  <span class="title">IL-21 Production by CD4^+ Effector T Cells and Frequency of Circulating Follicular Helper T Cells Are Increased in Type 1 Diabetes Patients</span>
  <span class="author">
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Simons H, 
    
    
    
    
    
    

    

    Thompson W, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Dopico X, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Mashar M, 
    
    
    
    
    
    

    

    Schuilenburg H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Pekalski M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetologia</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1007/s00125-015-3509-8" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4351433" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Type 1 diabetes results from the autoimmune destruction of insulin-secreting pancreatic beta cells by T cells. Despite the established role of T cells in the pathogenesis of the disease, to date, with the exception of the identification of islet-specific T effector (Teff) cells, studies have mostly failed to identify reproducible alterations in the frequency or function of T cell subsets in peripheral blood from patients with type 1 diabetes.We assessed the production of the proinflammatory cytokines IL-21, IFN-γ and IL-17 in peripheral blood mononuclear cells from 69 patients with type 1 diabetes and 61 healthy donors. In an additional cohort of 30 patients with type 1 diabetes and 32 healthy donors, we assessed the frequency of circulating T follicular helper (Tfh) cells in whole blood. IL-21 and IL-17 production was also measured in peripheral blood mononuclear cells (PBMCs) from a subset of 46 of the 62 donors immunophenotyped for Tfh.We found a 21.9% (95% CI 5.8, 40.2; p = 3.9 \texttimes 10(-3)) higher frequency of IL-21(+) CD45RA(-) memory CD4(+) Teffs in patients with type 1 diabetes (geometric mean 5.92% [95% CI 5.44, 6.44]) compared with healthy donors (geometric mean 4.88% [95% CI 4.33, 5.50]). Consistent with this finding, we found a 14.9% increase in circulating Tfh cells in the patients (95% CI 2.9, 26.9; p = 0.016).These results indicate that increased IL-21 production is likely to be an aetiological factor in the pathogenesis of type 1 diabetes that could be considered as a potential therapeutic target.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_eji_rainbow">
  <span class="title">Epigenetic Analysis of Regulatory T Cells Using Multiplex Bisulfite Sequencing</span>
  <span class="author">
    
    
    
    
    
    

    

    Rainbow D, 
    
    
    
    
    
    

    

    Yang X, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Klarqvist M, 
    
    
    
    
    
    

    

    Penkett C, 
    
    
    
    
    
    

    

    Brugger K, 
    
    
    
    
    
    

    

    Martin H, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wicker L. 
    
  </span>
  
  <div class="jblocks">
    
    <em>European journal of immunology</em>
    
    
    2015
    
  <!--  -->
   
  <a href="http://doi.org/10.1002/eji.201545646" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4832369" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2015_plosgen_liley">
  <span class="title">A Pleiotropy-Informed Bayesian False Discovery Rate Adapted to a Shared Control Design Finds New Disease Associations From GWAS Summary Statistics</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Liley J</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet</em>
    
    
    2015
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1004926" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4450050" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/jamesliley/cFDR-common-controls" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome-wide association studies (GWAS) have been successful in identifying single nucleotide polymorphisms (SNPs) associated with many traits and diseases. However, at existing sample sizes, these variants explain only part of the estimated heritability. Leverage of GWAS results from related phenotypes may improve detection without the need for larger datasets. The Bayesian conditional false discovery rate (cFDR) constitutes an upper bound on the expected false discovery rate (FDR) across a set of SNPs whose p values for two diseases are both less than two disease-specific thresholds. Calculation of the cFDR requires only summary statistics and have several advantages over traditional GWAS analysis. However, existing methods require distinct control samples between studies. Here, we extend the technique to allow for some or all controls to be shared, increasing applicability. Several different SNP sets can be defined with the same cFDR value, and we show that the expected FDR across the union of these sets may exceed expected FDR in any single set. We describe a procedure to establish an upper bound for the expected FDR among the union of such sets of SNPs. We apply our technique to pairwise analysis of p values from ten autoimmune diseases with variable sharing of controls, enabling discovery of 59 SNP-disease associations which do not reach GWAS significance after genomic control in individual datasets. Most of the SNPs we highlight have previously been confirmed using replication studies or larger GWAS, a useful validation of our technique; we report eight SNP-disease associations across five diseases not previously declared. Our technique extends and strengthens the previous algorithm, and establishes robust limits on the expected FDR. This approach can improve SNP detection in GWAS, and give insight into shared aetiology between phenotypically related conditions.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2014</h2>
<ol class="bibliography"><li>

<div id="2014_cei_thompson">
  <span class="title">Multi-Parametric Flow Cytometric and Genetic Investigation of the Peripheral B Cell Compartment in Human Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Thompson W, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Simons H, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Castro-Dopico X, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Guy C, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Arif S, 
    
    
    
    
    
    

    

    Peakman M, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Ferreira R. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Clin Exp Immunol</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1111/cei.12362" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4137841" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The appearance of circulating islet-specific autoantibodies before disease diagnosis is a hallmark of human type 1 diabetes (T1D), and suggests a role for B cells in the pathogenesis of the disease. Alterations in the peripheral B cell compartment have been reported in T1D patients; however, to date, such studies have produced conflicting results and have been limited by sample size. In this study, we have performed a detailed characterization of the B cell compartment in T1D patients (n\,= 45) and healthy controls (n\,= 46), and assessed the secretion of the anti-inflammatory cytokine interleukin (IL)-10 in purified B cells from the same donors. Overall, we found no evidence for a profound alteration of the B cell compartment or in the production of IL-10 in peripheral blood of T1D patients. We also investigated age-related changes in peripheral B cell subsets and confirmed the sharp decrease with age of transitional CD19(+) CD27(-) CD24(hi) CD38(hi) B cells, a subset that has recently been ascribed a putative regulatory function. Genetic analysis of the B cell compartment revealed evidence for association of the IL2-IL21\hspace0.25emT1D locus with IL-10 production by both memory B cells (P\,= 6⋅4\,\texttimes 10(-4) ) and islet-specific CD4(+) T cells (P\,= 2⋅9\,\texttimes 10(-3) ). In contrast to previous reports, we found no evidence for an alteration of the B cell compartment in healthy individuals homozygous for the non-synonymous PTPN22\hspace0.25emTrp(620) T1D risk allele (rs2476601; Arg(620) Trp). The IL2-IL21 association we have identified, if confirmed, suggests a novel role for B cells in T1D pathogenesis through the production of IL-10, and reinforces the importance of IL-10 production by autoreactive CD4(+) T cells.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2014_diabetes_ifnsig">
  <span class="title">A Type I Interferon Transcriptional Signature Precedes Autoimmunity in Children Genetically At-Risk of Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Coulson R, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Doecke J, 
    
    
    
    
    
    

    

    Flint S, 
    
    
    
    
    
    

    

    McKinney E, 
    
    
    
    
    
    

    

    Lyons P, 
    
    
    
    
    
    

    

    Smith K, 
    
    
    
    
    
    

    

    Achenbach P, 
    
    
    
    
    
    

    

    Beyerlein A, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    *Todd J , 
    
    
    
    
    
    

    

    *Bonifacio E , 
    
    
    
    
    
    

    

    <b>*Wallace C</b>, 
    
    
    
    
    
    

    

    *Ziegler G . 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetes</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.2337/db13-1777" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4066333" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Diagnosis of the autoimmune disease type 1 diabetes (T1D) is preceded by the appearance of circulating autoantibodies to pancreatic islets. However, almost nothing is known about events leading to this islet autoimmunity. Previous epidemiological and genetic data have associated viral infections and anti-viral type I interferon (IFN) immune response genes with T1D. Here, we first used DNA microarray analysis to identify IFN-Î^2 inducible genes in vitro and then used this set of genes to define an IFN-inducible transcriptional signature in peripheral blood mononuclear cells from a group of active systemic lupus erythematosus patients (N=25). Using this predefined set of 225 IFN signature genes, we investigated expression of the signature in cohorts of healthy controls (N=87), T1D patients (N=64) and a large longitudinal birth cohort of children genetically predisposed to T1D (N=109; 454 microarrayed samples). Expression of the IFN signature was increased in genetically-predisposed children prior to the development of autoantibodies (P=0.0012), but not in established T1D patients. Upregulation of IFN-inducible genes was transient, temporally associated with a recent history of upper respiratory tract infections (P=0.0064) and marked by increased expression of SIGLEC-1 (CD169), a lectin-like receptor expressed on CD14(+) monocytes. DNA variation in IFN-inducible genes altered T1D risk (P=0.007), as exemplified by IFIH1, one of the genes in our IFN signature and for which increased expression is a known disease risk factor. These findings identify transient increased expression of type I IFN genes in pre-clinical diabetes as a risk factor for autoimmunity in children with a genetic predisposition to T1D.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2014_diabetologia_downes">
  <span class="title">Plasma Concentrations of Soluble IL-2 Receptor (CD25) Are Increased in Type 1 Diabetes and Associated with Reduced C-Peptide Levels in Young Patients.</span>
  <span class="author">
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Marcovecchio M, 
    
    
    
    
    
    

    

    Clarke P, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Jolley J, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetologia</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1007/s00125-013-3113-8" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3890035" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Type 1 diabetes is a common autoimmune disease that has genetic and environmental determinants. Variations within the IL2 and IL2RA (also known as CD25) gene regions are associated with disease risk, and variation in expression or function of these proteins is likely to be causal. We aimed to investigate if circulating concentrations of the soluble form of CD25, sCD25, an established marker of immune activation and inflammation, were increased in individuals with type 1 diabetes and if this was associated with the concentration of C-peptide, a measure of insulin production that reflects the degree of autoimmune destruction of the insulin-producing beta cells.We used immunoassays to measure sCD25 and C-peptide in peripheral blood plasma from patient and control samples.We identified that sCD25 was increased in patients with type 1 diabetes compared with controls and replicated this result in an independent set of 86 adult patient and 80 age-matched control samples (pâ=â1.17âÃâ10(-3)). In 230 patients under 20Â years of age, with median duration-of-disease of 6.1Â years, concentrations of sCD25 were negatively associated with C-peptide concentrations (pâ=â4.8âÃâ10(-3)).The 25% increase in sCD25 in patients, alongside the inverse association between sCD25 and C-peptide, probably reflect the adverse effects of an on-going, actively autoimmune and inflammatory immune system on beta cell function in patients.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2014_plosgen_zanda">
  <span class="title">A Genome-Wide Assessment of the Role of Untagged Copy Number Variants in Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Zanda M, 
    
    
    
    
    
    

    

    Onengut-Gumuscu S, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Shtir C, 
    
    
    
    
    
    

    

    Gallo D, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Hurles M, 
    
    
    
    
    
    

    

    Plagnol V, 
    
    
    
    
    
    

    

    Rich S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1004367" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4038470" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome-wide association studies (GWAS) for type 1 diabetes (T1D) have successfully identified more than 40 independent T1D associated tagging single nucleotide polymorphisms (SNPs). However, owing to technical limitations of copy number variants (CNVs) genotyping assays, the assessment of the role of CNVs has been limited to the subset of these in high linkage disequilibrium with tag SNPs. The contribution of untagged CNVs, often multi-allelic and difficult to genotype using existing assays, to the heritability of T1D remains an open question. To investigate this issue, we designed a custom comparative genetic hybridization array (aCGH) specifically designed to assay untagged CNV loci identified from a variety of sources. To overcome the technical limitations of the case control design for this class of CNVs, we genotyped the Type 1 Diabetes Genetics Consortium (T1DGC) family resource (representing 3,903 transmissions from parents to affected offspring) and used an association testing strategy that does not necessitate obtaining discrete genotypes. Our design targeted 4,309 CNVs, of which 3,410 passed stringent quality control filters. As a positive control, the scan confirmed the known T1D association at the INS locus by direct typing of the 5’ variable number of tandem repeat (VNTR) locus. Our results clarify the fact that the disease association is indistinguishable from the two main polymorphic allele classes of the INS VNTR, class I-and class III. We also identified novel technical artifacts resulting into spurious associations at the somatically rearranging loci, T cell receptor, TCRA/TCRD and TCRB, and Immunoglobulin heavy chain, IGH, loci on chromosomes 14q11.2, 7q34 and 14q32.33, respectively. However, our data did not identify novel T1D loci. Our results do not support a major role of untagged CNVs in T1D heritability.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Downes2014-tu">
  <span class="title">Plasma Concentrations of Soluble IL-2 Receptor α (CD25) Are Increased in Type 1 Diabetes and Associated with Reduced C-Peptide Levels in Young Patients</span>
  <span class="author">
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Marcovecchio M, 
    
    
    
    
    
    

    

    Clarke P, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Jolley J, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetologia</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1007/s00125-013-3113-8" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Type 1 diabetes is a common autoimmune disease that has genetic and environmental determinants. Variations within the IL2 and IL2RA (also known as CD25) gene regions are associated with disease risk, and variation in expression or function of these proteins is likely to be causal. We aimed to investigate if circulating concentrations of the soluble form of CD25, sCD25, an established marker of immune activation and inflammation, were increased in individuals with type 1 diabetes and if this was associated with the concentration of C-peptide, a measure of insulin production that reflects the degree of autoimmune destruction of the insulin-producing beta cells.We used immunoassays to measure sCD25 and C-peptide in peripheral blood plasma from patient and control samples.We identified that sCD25 was increased in patients with type 1 diabetes compared with controls and replicated this result in an independent set of 86 adult patient and 80 age-matched control samples (p = 1.17  10(-3)). In 230 patients under 20 years of age, with median duration-of-disease of 6.1 years, concentrations of sCD25 were negatively associated with C-peptide concentrations (p = 4.8  10(-3)).The 25% increase in sCD25 in patients, alongside the inverse association between sCD25 and C-peptide, probably reflect the adverse effects of an on-going, actively autoimmune and inflammatory immune system on beta cell function in patients.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2014_bmcgenomics_pontikos">
  <span class="title">A Hybrid qPCR/SNP Array Approach Allows Cost Efficient Assessment of KIR Gene Copy Numbers in Large Samples</span>
  <span class="author">
    
    
    
    
    
    

    

    Pontikos N, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Schuilenburg H, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Onengut-Gumuscu S, 
    
    
    
    
    
    

    

    Chen W, 
    
    
    
    
    
    

    

    Concannon P, 
    
    
    
    
    
    

    

    Rich S, 
    
    
    
    
    
    

    

    Jayaraman J, 
    
    
    
    
    
    

    

    Jiang W, 
    
    
    
    
    
    

    

    Traherne J, 
    
    
    
    
    
    

    

    Trowsdale J, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>BMC Genomics</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1186/1471-2164-15-274" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4029094" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>BACKGROUND: Killer Immunoglobulin-like Receptors (KIRs) are surface receptors of natural killer cells that bind to their corresponding Human Leukocyte Antigen (HLA) class I ligands, making them interesting candidate genes for HLA-associated autoimmune diseases, including type 1 diabetes (T1D). However, allelic and copy number variation in the KIR region effectively mask it from standard genome-wide association studies: single nucleotide polymorphism (SNP) probes targeting the region are often discarded by standard genotype callers since they exhibit variable cluster numbers. Quantitative Polymerase Chain Reaction (qPCR) assays address this issue. However, their cost is prohibitive at the sample sizes required for detecting effects typically observed in complex genetic diseases. RESULTS: We propose a more powerful and cost-effective alternative, which combines signals from SNPs with more than three clusters found in existing datasets, with qPCR on a subset of samples. First, we showed that noise and batch effects in multiplexed qPCR assays are addressed through normalisation and simultaneous copy number calling of multiple genes. Then, we used supervised classification to impute copy numbers of specific KIR genes from SNP signals. We applied this method to assess copy number variation in two KIR genes, KIR3DL1 and KIR3DS1, which are suitable candidates for T1D susceptibility since they encode the only KIR molecules known to bind with HLA-Bw4 epitopes. We find no association between KIR3DL1/3DS1 copy number and T1D in 6744 cases and 5362 controls; a sample size twenty-fold larger than in any previous KIR association study. Due to our sample size, we can exclude odds ratios larger than 1.1 for the common KIR3DL1/3DS1 copy number groups at the 5% significance level. CONCLUSION: We found no evidence of association of KIR3DL1/3DS1 copy number with T1D, either overall or dependent on HLA-Bw4 epitope. Five other KIR genes, KIR2DS4, KIR2DL3, KIR2DL5, KIR2DS5 and KIR2DS1, in high linkage disequilibrium with KIR3DL1 and KIR3DS1, are also unlikely to be significantly associated. Our approach could potentially be applied to other KIR genes to allow cost effective assaying of gene copy number in large samples.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2014_bioinformatics_burren">
  <span class="title">VSEAMS: A Pipeline for Variant Set Enrichment Analysis Using Summary GWAS Data Identifies IKZF3, BATF and ESRRA as Key Transcription Factors in Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Bioinformatics</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/bioinformatics/btu571" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4296156" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <a href="https://github.com/ollyburren/VSEAMS" target="_blank" class="pure-button doi">R <i class="fas fa-file-code"></i></a>
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--   [<a href="https://github.com/ollyburren/VSEAMS" target="_blank">Code</a>] -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome-wide association studies (GWAS) have identified many loci implicated in disease susceptibility. Integration of GWAS summary statistics (P-values) and functional genomic datasets should help to elucidate mechanisms.We extended a non-parametric SNP set enrichment method to test for enrichment of GWAS signals in functionally defined loci to a situation where only GWAS P-values are available. The approach is implemented in VSEAMS, a freely available software pipeline. We use VSEAMS to identify enrichment of type 1 diabetes (T1D) GWAS associations near genes that are targets for the transcription factors IKZF3, BATF and ESRRA. IKZF3 lies in a known T1D susceptibility region, while BATF and ESRRA overlap other immune disease susceptibility regions, validating our approach and suggesting novel avenues of research for T1D. Availability and implementation: VSEAMS is available for download (http://github.com/ollyburren/vseams).chris.wallace@cimr.cam.ac.uk Supplementary information: Supplementary data are available at Bioinformatics online.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2014_genepi_evangelou">
  <span class="title">A Method for Gene-Based Pathway Analysis Using Genomewide Association Study Summary Statistics Reveals Nine New Type 1 Diabetes Associations</span>
  <span class="author">
    
    
    
    
    
    

    

    Evangelou M, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    <b>Fortune M</b>, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Onengut-Gumuscu S, 
    
    
    
    
    
    

    

    Chen W, 
    
    
    
    
    
    

    

    Concannon P, 
    
    
    
    
    
    

    

    Rich S, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genet. Epidemiol.</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1002/gepi.21853" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4258092" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Pathway analysis can complement point-wise single nucleotide polymorphism (SNP) analysis in exploring genomewide association study (GWAS) data to identify specific disease-associated genes that can be candidate causal genes. We propose a straightforward methodology that can be used for conducting a gene-based pathway analysis using summary GWAS statistics in combination with widely available reference genotype data. We used this method to perform a gene-based pathway analysis of a type 1 diabetes (T1D) meta-analysis GWAS (of 7,514 cases and 9,045 controls). An important feature of the conducted analysis is the removal of the major histocompatibility complex gene region, the major genetic risk factor for T1D. Thirty-one of the 1,583 (2%) tested pathways were identified to be enriched for association with T1D at a 5% false discovery rate. We analyzed these 31 pathways and their genes to identify SNPs in or near these pathway genes that showed potentially novel association with T1D and attempted to replicate the association of 22 SNPs in additional samples. Replication P-values were skewed (P=9.85x10-11) with 12 of the 22 SNPs showing P<0.05. Support, including replication evidence, was obtained for nine T1D associated variants in genes ITGB7 (rs11170466, P=7.86x10-9), NRP1 (rs722988, 4.88x10-8), BAD (rs694739, 2.37x10-7), CTSB (rs1296023, 2.79x10-7), FYN (rs11964650, P=5.60x10-7), UBE2G1 (rs9906760, 5.08x10-7), MAP3K14 (rs17759555, 9.67x10-7), ITGB1 (rs1557150, 1.93x10-6), and IL7R (rs1445898, 2.76x10-6). The proposed methodology can be applied to other GWAS datasets for which only summary level data are available.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2014_plosgen_giambartolomei">
  <span class="title">Bayesian Test for Colocalisation between Pairs of Genetic Association Studies Using Summary Statistics.</span>
  <span class="author">
    
    
    
    
    
    

    

    Giambartolomei C, 
    
    
    
    
    
    

    

    Vukcevic D, 
    
    
    
    
    
    

    

    Schadt E, 
    
    
    
    
    
    

    

    Franke L, 
    
    
    
    
    
    

    

    Hingorani A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Plagnol V. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet</em>
    
    
    2014
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1004383" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4022491" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/chr1swallace/coloc" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genetic association studies, in particular the genome-wide association study (GWAS) design, have provided a wealth of novel insights into the aetiology of a wide range of human diseases and traits, in particular cardiovascular diseases and lipid biomarkers. The next challenge consists of understanding the molecular basis of these associations. The integration of multiple association datasets, including gene expression datasets, can contribute to this goal. We have developed a novel statistical methodology to assess whether two association signals are consistent with a shared causal variant. An application is the integration of disease scans with expression quantitative trait locus (eQTL) studies, but any pair of GWAS datasets can be integrated in this framework. We demonstrate the value of the approach by re-analysing a gene expression dataset in 966 liver samples with a published meta-analysis of lipid traits including >100,000 individuals of European ancestry. Combining all lipid biomarkers, our re-analysis supported 26 out of 38 reported colocalisation results with eQTLs and identified 14 new colocalisation results, hence highlighting the value of a formal statistical test. In three cases of reported eQTL-lipid pairs (SYPL2, IFT172, TBKBP1) for which our analysis suggests that the eQTL pattern is not consistent with the lipid association, we identify alternative colocalisation results with SORT1, GCKR, and KPNB1, indicating that these genes are more likely to be causal in these genomic intervals. A key feature of the method is the ability to derive the output statistics from single SNP summary statistics, hence making it possible to perform systematic meta-analysis type comparisons across multiple GWAS datasets (implemented online at http://coloc.cs.ucl.ac.uk/coloc/). Our methodology provides information about candidate causal genes in associated intervals and has direct implications for the understanding of complex diseases as well as the design of drugs to target disease pathways.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2013</h2>
<ol class="bibliography"><li>

<div id="Hinks2013-lm">
  <span class="title">Dense Genotyping of Immune-Related Disease Regions Identifies 14 New Susceptibility Loci for Juvenile Idiopathic Arthritis</span>
  <span class="author">
    
    
    
    
    
    

    

    Hinks A, 
    
    
    
    
    
    

    

    Cobb J, 
    
    
    
    
    
    

    

    Marion M, 
    
    
    
    
    
    

    

    Prahalad S, 
    
    
    
    
    
    

    

    Sudman M, 
    
    
    
    
    
    

    

    Bowes J, 
    
    
    
    
    
    

    

    Martin P, 
    
    
    
    
    
    

    

    Comeau M, 
    
    
    
    
    
    

    

    Sajuthi S, 
    
    
    
    
    
    

    

    Andrews R, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Chen W, 
    
    
    
    
    
    

    

    Concannon P, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Edkins S, 
    
    
    
    
    
    

    

    Eyre S, 
    
    
    
    
    
    

    

    Gaffney P, 
    
    
    
    
    
    

    

    Guthery S, 
    
    
    
    
    
    

    

    Guthridge J, 
    
    
    
    
    
    

    

    Hunt S, 
    
    
    
    
    
    

    

    James J, 
    
    
    
    
    
    

    

    Keddache M, 
    
    
    
    
    
    

    

    Moser K, 
    
    
    
    
    
    

    

    Nigrovic P, 
    
    
    
    
    
    

    

    Onengut-Gumuscu S, 
    
    
    
    
    
    

    

    Onslow M, 
    
    
    
    
    
    

    

    Rosé C, 
    
    
    
    
    
    

    

    Rich S, 
    
    
    
    
    
    

    

    Steel K, 
    
    
    
    
    
    

    

    Wakeland E, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wedderburn L, 
    
    
    
    
    
    

    

    Woo P, 
    
    
    
    
    
    

    

    Boston Children’s J I A Registry , 
    
    
    
    
    
    

    

    British Society of Paediatric , 
    
    
    
    
    
    

    

    Group A, 
    
    
    
    
    
    

    

    Childhood Arthritis Prospective Study (C A P S) , 
    
    
    
    
    
    

    

    Childhood Arthritis Response to Medication Study (C H A R M S) , 
    
    
    
    
    
    

    

    German Society For Pediatric Rheumatology (G K J R) , 
    
    
    
    
    
    

    

    J I A Gene Expression Study , 
    
    
    
    
    
    

    

    N I A M S J I A Genetic Registry , 
    
    
    
    
    
    

    

    T R E A T Study , 
    
    
    
    
    
    

    

    United Kingdom Juvenile Idiopathic Arthritis Genetics Consortium (U K J I A G C) , 
    
    
    
    
    
    

    

    Bohnsack J, 
    
    
    
    
    
    

    

    Haas J, 
    
    
    
    
    
    

    

    Glass D, 
    
    
    
    
    
    

    

    Langefeld C, 
    
    
    
    
    
    

    

    Thomson W, 
    
    
    
    
    
    

    

    Thompson S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nature Publishing Group</em>
    
    
    2013
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.2614" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3673707" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>We used the Immunochip array to analyze 2,816 individuals with juvenile idiopathic arthritis (JIA), comprising the most common subtypes (oligoarticular and rheumatoid factor-negative polyarticular JIA), and 13,056 controls. We confirmed association of 3 known JIA risk loci (the human leukocyte antigen (HLA) region, PTPN22 and PTPN2) and identified 14 loci reaching genome-wide significance (P < 5  10(-8)) for the first time. Eleven additional new regions showed suggestive evidence of association with JIA (P < 1  10(-6)). Dense mapping of loci along with bioinformatics analysis refined the associations to one gene in each of eight regions, highlighting crucial pathways, including the interleukin (IL)-2 pathway, in JIA disease pathogenesis. The entire Immunochip content, the HLA region and the top 27 loci (P < 1  10(-6)) explain an estimated 18, 13 and 6% of the risk of JIA, respectively. In summary, this is the largest collection of JIA cases investigated so far and provides new insight into the genetic basis of this childhood autoimmune disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2013_pekalski">
  <span class="title">Postthymic Expansion in Human CD4 Naive T Cells Defined by Expression of Functional High-Affinity IL-2 Receptors.</span>
  <span class="author">
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Ferreira R, 
    
    
    
    
    
    

    

    Coulson R, 
    
    
    
    
    
    

    

    Cutler A, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Dendrou C, 
    
    
    
    
    
    

    

    Castro Dopico X, 
    
    
    
    
    
    

    

    Esposito L, 
    
    
    
    
    
    

    

    <b>Coleman G</b>, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Guy C, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Tree T, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Wicker L. 
    
  </span>
  
  <div class="jblocks">
    
    <em>J Immunol</em>
    
    
    2013
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.4049/jimmunol.1202914" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3614027" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>As the thymus involutes with age, the maintenance of peripheral naive T cells in humans becomes strongly dependent on peripheral cell division. However, mechanisms that orchestrate homeostatic division remain unclear. In this study we present evidence that the frequency of naive CD4 T cells that express CD25 (IL-2 receptor Î\pm-chain) increases with age on subsets of both CD31(+) and CD31(-) naive CD4 T cells. Analyses of TCR excision circles from sorted subsets indicate that CD25(+) naive CD4 T cells have undergone more rounds of homeostatic proliferation than their CD25(-) counterparts in both the CD31(+) and CD31(-) subsets, indicating that CD25 is a marker of naive CD4 T cells that have preferentially responded to survival signals from self-Ags or cytokines. CD25 expression on CD25(-) naive CD4 T cells can be induced by IL-7 in vitro in the absence of TCR activation. Although CD25(+) naive T cells respond to lower concentrations of IL-2 as compared with their CD25(-) counterparts, IL-2 responsiveness is further increased in CD31(-) naive T cells by their expression of the signaling IL-2 receptor Î^2-chain CD122, forming with common Î^3-chain functional high-affinity IL-2 receptors. CD25 plays a role during activation: CD25(+) naive T cells stimulated in an APC-dependent manner were shown to produce increased levels of IL-2 as compared with their CD25(-) counterparts. This study establishes CD25(+) naive CD4 T cells, which are further delineated by CD31 expression, as a major functionally distinct immune cell subset in humans that warrants further characterization in health and disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2013_genepi_coloc">
  <span class="title">Statistical Testing of Shared Genetic Control for Potentially Related Traits.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genet Epidemiol</em>
    
    
    2013
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1002/gepi.21765" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC4158901" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  <a href="https://github.com/chr1swallace/coloc" target="_blank" class="pure-button doi">R <i class="fa fa-cubes"></i></a>
  
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Integration of data from genome-wide single nucleotide polymorphism (SNP) association studies of different traits should allow researchers to disentangle the genetics of potentially related traits within individually associated regions. Formal statistical colocalisation testing of individual regions requires selection of a set of SNPs summarising the association in a region. We show that the SNP selection method greatly affects type 1 error rates, with published studies having used methods expected to result in substantially inflated type 1 error rates. We show that either avoiding variable selection and instead testing the most informative principal components or integrating over variable selection using Bayesian model averaging can help control type 1 error rates. Application to data from Graves’ disease and Hashimoto’s thyroiditis reveals a common genetic signature across seven regions shared between the diseases, and indicates that in five of six regions associated with Graves’ disease and not Hashimoto’s thyroiditis, this more likely reflects genuine absence of association with the latter rather than lack of power. Our examination, by simulation, of the performance of colocalisation tests and associated software will foster more widespread adoption of formal colocalisation testing. Given the increasing availability of large expression and genetic association datasets from disease-relevant tissue and purified cell populations, coupled with identification of regulatory sequences by projects such as ENCODE, colocalisation analysis has the potential to reveal both shared genetic signatures of related traits and causal disease genes and tissues.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2012</h2>
<ol class="bibliography"><li>

<div id="2012_angelakopoulou">
  <span class="title">Comparative Analysis of Genome-Wide Association Studies Signals for Lipids, Diabetes, and Coronary Heart Disease: Cardiovascular Biomarker Genetics Collaboration.</span>
  <span class="author">
    
    
    
    
    
    

    

    Angelakopoulou A, 
    
    
    
    
    
    

    

    Shah T, 
    
    
    
    
    
    

    

    Sofat R, 
    
    
    
    
    
    

    

    Shah S, 
    
    
    
    
    
    

    

    Berry D, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Palmen J, 
    
    
    
    
    
    

    

    Tzoulaki I, 
    
    
    
    
    
    

    

    Wong A, 
    
    
    
    
    
    

    

    Jefferis B, 
    
    
    
    
    
    

    

    Maniatis N, 
    
    
    
    
    
    

    

    Drenos F, 
    
    
    
    
    
    

    

    Gigante B, 
    
    
    
    
    
    

    

    Hardy R, 
    
    
    
    
    
    

    

    Laxton R, 
    
    
    
    
    
    

    

    Leander K, 
    
    
    
    
    
    

    

    Motterle A, 
    
    
    
    
    
    

    

    Simpson I, 
    
    
    
    
    
    

    

    Smeeth L, 
    
    
    
    
    
    

    

    Thomson A, 
    
    
    
    
    
    

    

    Verzilli C, 
    
    
    
    
    
    

    

    Kuh D, 
    
    
    
    
    
    

    

    Ireland H, 
    
    
    
    
    
    

    

    Deanfield J, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Fowkes F, 
    
    
    
    
    
    

    

    Marmot M, 
    
    
    
    
    
    

    

    Whincup P, 
    
    
    
    
    
    

    

    Whittaker J, 
    
    
    
    
    
    

    

    de Faire U, 
    
    
    
    
    
    

    

    Kivimaki M, 
    
    
    
    
    
    

    

    Kumari M, 
    
    
    
    
    
    

    

    Hypponen E, 
    
    
    
    
    
    

    

    Power C, 
    
    
    
    
    
    

    

    Humphries S, 
    
    
    
    
    
    

    

    Talmud P, 
    
    
    
    
    
    

    

    Price J, 
    
    
    
    
    
    

    

    Morris R, 
    
    
    
    
    
    

    

    Ye S, 
    
    
    
    
    
    

    

    Casas J, 
    
    
    
    
    
    

    

    Hingorani A. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Eur Heart J</em>
    
    
    2012
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/eurheartj/ehr225" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3270041" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>To evaluate the associations of emergent genome-wide-association study-derived coronary heart disease (CHD)-associated single nucleotide polymorphisms (SNPs) with established and emerging risk factors, and the association of genome-wide-association study-derived lipid-associated SNPs with other risk factors and CHD events.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Jin2012-bw">
  <span class="title">Genome-Wide Association Analyses Identify 13 New Susceptibility Loci for Generalized Vitiligo</span>
  <span class="author">
    
    
    
    
    
    

    

    Jin Y, 
    
    
    
    
    
    

    

    Birlea S, 
    
    
    
    
    
    

    

    Fain P, 
    
    
    
    
    
    

    

    Ferrara T, 
    
    
    
    
    
    

    

    Ben S, 
    
    
    
    
    
    

    

    Riccardi S, 
    
    
    
    
    
    

    

    Cole J, 
    
    
    
    
    
    

    

    Gowan K, 
    
    
    
    
    
    

    

    Holland P, 
    
    
    
    
    
    

    

    Bennett D, 
    
    
    
    
    
    

    

    Luiten R, 
    
    
    
    
    
    

    

    Wolkerstorfer A, 
    
    
    
    
    
    

    

    van der Veen J, 
    
    
    
    
    
    

    

    Hartmann A, 
    
    
    
    
    
    

    

    Eichner S, 
    
    
    
    
    
    

    

    Schuler G, 
    
    
    
    
    
    

    

    van Geel N, 
    
    
    
    
    
    

    

    Lambert J, 
    
    
    
    
    
    

    

    Kemp E, 
    
    
    
    
    
    

    

    Gawkrodger D, 
    
    
    
    
    
    

    

    Weetman A, 
    
    
    
    
    
    

    

    Taïeb A, 
    
    
    
    
    
    

    

    Jouary T, 
    
    
    
    
    
    

    

    Ezzedine K, 
    
    
    
    
    
    

    

    <b>Wallace M</b>, 
    
    
    
    
    
    

    

    McCormack W, 
    
    
    
    
    
    

    

    Picardo M, 
    
    
    
    
    
    

    

    Leone G, 
    
    
    
    
    
    

    

    Overbeck A, 
    
    
    
    
    
    

    

    Silverberg N, 
    
    
    
    
    
    

    

    Spritz R. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Genet.</em>
    
    
    2012
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.2272" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3366044" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>We previously reported a genome-wide association study (GWAS) identifying 14 susceptibility loci for generalized vitiligo. We report here a second GWAS (450 individuals with vitiligo (cases) and 3,182 controls), an independent replication study (1,440 cases and 1,316 controls) and a meta-analysis (3,187 cases and 6,723 controls) identifying 13 additional vitiligo-associated loci. These include OCA2-HERC2 (combined P = 3.80  10(-8)), MC1R (P = 1.82  10(-13)), a region near TYR (P = 1.57  10(-13)), IFIH1 (P = 4.91  10(-15)), CD80 (P = 3.78  10(-10)), CLNK (P = 1.56  10(-8)), BACH2 (P = 2.53  10(-8)), SLA (P = 1.58  10(-8)), CASP7 (P = 3.56  10(-8)), CD44 (P = 1.78  10(-9)), IKZF4 (P = 2.75  10(-14)), SH2B3 (P = 3.54  10(-18)) and TOB2 (P = 6.81  10(-10)). Most vitiligo susceptibility loci encode immunoregulatory proteins or melanocyte components that likely mediate immune targeting and the relationships among vitiligo, melanoma, and eye, skin and hair coloration.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2012_bioinformatics_yang">
  <span class="title">Extra-Binomial Variation Approach for Analysis of Pooled DNA Sequencing Data.</span>
  <span class="author">
    
    
    
    
    
    

    

    Yang X, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    <b>Wallace C</b>. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Bioinformatics</em>
    
    
    2012
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/bioinformatics/bts553" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3496343" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The invention of next-generation sequencing technology has made it possible to study the rare variants that are more likely to pinpoint causal disease genes. To make such experiments financially viable, DNA samples from several subjects are often pooled before sequencing. This induces large between-pool variation which, together with other sources of experimental error, creates over-dispersed data. Statistical analysis of pooled sequencing data needs to appropriately model this additional variance to avoid inflating the false-positive rate.We propose a new statistical method based on an extra-binomial model to address the over-dispersion and apply it to pooled case-control data. We demonstrate that our model provides a better fit to the data than either a standard binomial model or a traditional extra-binomial model proposed by Williams and can analyse both rare and common variants with lower or more variable pool depths compared to the other methods.Package ’extraBinomial’ is on http://cran.r-project.org/.chris.wallace@cimr.cam.ac.uk.Supplementary data are available at Bioinformatics Online.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2012_hmg_cooper">
  <span class="title">Seven Newly Identified Loci for Autoimmune Thyroid Disease.</span>
  <span class="author">
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Simmonds M, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Brand O, 
    
    
    
    
    
    

    

    Guo H, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    <b>Coleman G</b>, 
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Franklyn J, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Gough S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hum Mol Genet</em>
    
    
    2012
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/hmg/dds357" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3490518" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Autoimmune thyroid disease (AITD), including Graves’ disease (GD) and Hashimoto’s thyroiditis (HT), is one of the most common of the immune-mediated diseases. To further investigate the genetic determinants of AITD, we conducted an association study using a custom-made single-nucleotide polymorphism (SNP) array, the ImmunoChip. The SNP array contains all known and genotype-able SNPs across 186 distinct susceptibility loci associated with one or more immune-mediated diseases. After stringent quality control, we analysed 103 875 common SNPs (minor allele frequency >0.05) in 2285 GD and 462 HT patients and 9364 controls. We found evidence for seven new AITD risk loci (P < 1.12 Ã 10(-6); a permutation test derived significance threshold), five at locations previously associated and two at locations awaiting confirmation, with other immune-mediated diseases.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2012_hmg_davison">
  <span class="title">Long-Range DNA Looping and Gene Expression Analyses Identify <i>DEXI</i> as an Autoimmune Disease Candidate Gene.</span>
  <span class="author">
    
    
    
    
    
    

    

    Davison, L , 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Cope N, 
    
    
    
    
    
    

    

    Wilson N, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Saleh N, 
    
    
    
    
    
    

    

    Al-Jeffery A, 
    
    
    
    
    
    

    

    Angus K, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Duley S, 
    
    
    
    
    
    

    

    Coulson R, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Rice C, 
    
    
    
    
    
    

    

    Cambien F, 
    
    
    
    
    
    

    

    Zeller T, 
    
    
    
    
    
    

    

    Munzel T, 
    
    
    
    
    
    

    

    Lackner K, 
    
    
    
    
    
    

    

    Blankenberg S, 
    
    
    
    
    
    

    

    Cardiogenics Consortium , 
    
    
    
    
    
    

    

    Fraser P, 
    
    
    
    
    
    

    

    Gottgens B, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hum Mol Genet</em>
    
    
    2012
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/hmg/ddr468" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3276289" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The chromosome 16p13 region has been associated with several autoimmune diseases, including type 1 diabetes (T1D) and multiple sclerosis (MS). CLEC16A has been reported as the most likely candidate gene in the region, since it contains the most disease-associated single-nucleotide polymorphisms (SNPs), as well as an imunoreceptor tyrosine-based activation motif. However, here we report that intron 19 of CLEC16A, containing the most autoimmune disease-associated SNPs, appears to behave as a regulatory sequence, affecting the expression of a neighbouring gene, DEXI. The CLEC16A alleles that are protective from T1D and MS are associated with increased expression of DEXI, and no other genes in the region, in two independent monocyte gene expression data sets. Critically, using chromosome conformation capture (3C), we identified physical proximity between the DEXI promoter region and intron 19 of CLEC16A, separated by a loop of >150 kb. In reciprocal experiments, a 20 kb fragment of intron 19 of CLEC16A, containing SNPs associated with T1D and MS, as well as with DEXI expression, interacted with the promotor region of DEXI but not with candidate DNA fragments containing other potential causal genes in the region, including CLEC16A. Intron 19 of CLEC16A is highly enriched for transcription-factor-binding events and markers associated with enhancer activity. Taken together, these data indicate that although the causal variants in the 16p13 region lie within CLEC16A, DEXI is an unappreciated autoimmune disease candidate gene, and illustrate the power of the 3C approach in progressing from genome-wide association studies results to candidate causal genes.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2012_hmg_coloc">
  <span class="title">Statistical Colocalization of Monocyte Gene Expression and Genetic Risk Variants for Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Rotival M, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Rice C, 
    
    
    
    
    
    

    

    Yang J, 
    
    
    
    
    
    

    

    McNeill M, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Niblett D, 
    
    
    
    
    
    

    

    Cambien F, 
    
    
    
    
    
    

    

    Cardiogenics Consortium , 
    
    
    
    
    
    

    

    Tiret L, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Blankenberg S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hum Mol Genet</em>
    
    
    2012
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/hmg/dds098" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3363338" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>One mechanism by which disease-associated DNA variation can alter disease risk is altering gene expression. However, linkage disequilibrium (LD) between variants, mostly single-nucleotide polymorphisms (SNPs), means it is not sufficient to show that a particular variant associates with both disease and expression, as there could be two distinct causal variants in LD. Here, we describe a formal statistical test of colocalization and apply it to type 1 diabetes (T1D)-associated regions identified mostly through genome-wide association studies and expression quantitative trait loci (eQTLs) discovered in a recently determined large monocyte expression data set from the Gutenberg Health Study (1370 individuals), with confirmation sought in an additional data set from the Cardiogenics Transcriptome Study (558 individuals). We excluded 39 out of 60 overlapping eQTLs in 49 T1D regions from possible colocalization and identified 21 coincident eQTLs, representing 21 genes in 14 distinct T1D regions. Our results reflect the importance of monocyte (and their derivatives, macrophage and dendritic cell) gene expression in human T1D and support the candidacy of several genes as causal factors in autoimmune pancreatic beta-cell destruction, including AFF3, CD226, CLECL1, DEXI, FKRP, PRKD2, RNLS, SMARCE1 and SUOX, in addition to the recently described GPR183 (EBI2) gene.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2011</h2>
<ol class="bibliography"><li>

<div id="2011_diab_cooper">
  <span class="title">Inherited Variation in Vitamin D Genes Is Associated with Predisposition to Autoimmune Disease Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Greissl C, 
    
    
    
    
    
    

    

    Ramos-Lopez E, 
    
    
    
    
    
    

    

    HyppÃnen E, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Wang T, 
    
    
    
    
    
    

    

    Badenhoop K, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetes</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.2337/db10-1656" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3292339" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Vitamin D deficiency (25-hydroxyvitamin D [25(OH)D] <50 nmol/L) is commonly reported in both children and adults worldwide, and growing evidence indicates that vitamin D deficiency is associated with many extraskeletal chronic disorders, including the autoimmune diseases type 1 diabetes and multiple sclerosis.We measured 25(OH)D concentrations in 720 case and 2,610 control plasma samples and genotyped single nucleotide polymorphisms from seven vitamin D metabolism genes in 8,517 case, 10,438 control, and 1,933 family samples. We tested genetic variants influencing 25(OH)D metabolism for an association with both circulating 25(OH)D concentrations and disease status.Type 1 diabetic patients have lower circulating levels of 25(OH)D than similarly aged subjects from the British population. Only 4.3 and 18.6% of type 1 diabetic patients reached optimal levels (â\textyen 75 nmol/L) of 25(OH)D for bone health in the winter and summer, respectively. We replicated the associations of four vitamin D metabolism genes (GC, DHCR7, CYP2R1, and CYP24A1) with 25(OH)D in control subjects. In addition to the previously reported association between type 1 diabetes and CYP27B1 (P = 1.4 Ã 10(-4)), we obtained consistent evidence of type 1 diabetes being associated with DHCR7 (P = 1.2 Ã 10(-3)) and CYP2R1 (P = 3.0 Ã 10(-3)).Circulating levels of 25(OH)D in children and adolescents with type 1 diabetes vary seasonally and are under the same genetic control as in the general population but are much lower. Three key 25(OH)D metabolism genes show consistent evidence of association with type 1 diabetes risk, indicating a genetic etiological role for vitamin D deficiency in type 1 diabetes.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_johnson">
  <span class="title">Blood Pressure Loci Identified with a Gene-Centric Array</span>
  <span class="author">
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Gaunt T, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Padmanabhan S, 
    
    
    
    
    
    

    

    Tomaszewski M, 
    
    
    
    
    
    

    

    Kumari M, 
    
    
    
    
    
    

    

    Morris R, 
    
    
    
    
    
    

    

    Tzoulaki I, 
    
    
    
    
    
    

    

    O’Brien E, 
    
    
    
    
    
    

    

    Poulter N, 
    
    
    
    
    
    

    

    Sever P, 
    
    
    
    
    
    

    

    Shields D, 
    
    
    
    
    
    

    

    Thom S, 
    
    
    
    
    
    

    

    Wannamethee S, 
    
    
    
    
    
    

    

    Whincup P, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Howard P, 
    
    
    
    
    
    

    

    Mein C, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    Shaw-Hawkins S, 
    
    
    
    
    
    

    

    Zhang Y, 
    
    
    
    
    
    

    

    Smith G, 
    
    
    
    
    
    

    

    Day I, 
    
    
    
    
    
    

    

    Lawlor D, 
    
    
    
    
    
    

    

    Goodall A, 
    
    
    
    
    
    

    

    Fowkes F, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Gateva V, 
    
    
    
    
    
    

    

    Braund P, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Nelson C, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Van Der Harst P, 
    
    
    
    
    
    

    

    Glorioso N, 
    
    
    
    
    
    

    

    Neuvrith H, 
    
    
    
    
    
    

    

    Salvi E, 
    
    
    
    
    
    

    

    Staessen J, 
    
    
    
    
    
    

    

    Stucchi A, 
    
    
    
    
    
    

    

    Devos N, 
    
    
    
    
    
    

    

    Jeunemaitre X, 
    
    
    
    
    
    

    

    Plouin P, 
    
    
    
    
    
    

    

    Tichet J, 
    
    
    
    
    
    

    

    Juhanson P, 
    
    
    
    
    
    

    

    Org E, 
    
    
    
    
    
    

    

    Putku M, 
    
    
    
    
    
    

    

    Sauber S, 
    
    
    
    
    
    

    

    Veldre G, 
    
    
    
    
    
    

    

    Viigimaa M, 
    
    
    
    
    
    

    

    Levinsson A, 
    
    
    
    
    
    

    

    Rosengren A, 
    
    
    
    
    
    

    

    Thelle D, 
    
    
    
    
    
    

    

    Hastie C, 
    
    
    
    
    
    

    

    Hedner T, 
    
    
    
    
    
    

    

    Lee W, 
    
    
    
    
    
    

    

    Melander O, 
    
    
    
    
    
    

    

    Wahlstrand B, 
    
    
    
    
    
    

    

    Hardy R, 
    
    
    
    
    
    

    

    Wong A, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Palmen J, 
    
    
    
    
    
    

    

    Chen L, 
    
    
    
    
    
    

    

    Stewart A, 
    
    
    
    
    
    

    

    Wells G, 
    
    
    
    
    
    

    

    Westra H, 
    
    
    
    
    
    

    

    Wolfs M, 
    
    
    
    
    
    

    

    Clarke R, 
    
    
    
    
    
    

    

    Franzosi M, 
    
    
    
    
    
    

    

    Goel A, 
    
    
    
    
    
    

    

    Hamsten A, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Peden J, 
    
    
    
    
    
    

    

    Seedorf U, 
    
    
    
    
    
    

    

    Watkins H, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Sambrook J, 
    
    
    
    
    
    

    

    Stephens J, 
    
    
    
    
    
    

    

    Casas J, 
    
    
    
    
    
    

    

    Drenos F, 
    
    
    
    
    
    

    

    Holmes M, 
    
    
    
    
    
    

    

    Kivimaki M, 
    
    
    
    
    
    

    

    Shah S, 
    
    
    
    
    
    

    

    Shah T, 
    
    
    
    
    
    

    

    Talmud P, 
    
    
    
    
    
    

    

    Whittaker J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Delles C, 
    
    
    
    
    
    

    

    Laan M, 
    
    
    
    
    
    

    

    Kuh D, 
    
    
    
    
    
    

    

    Humphries S, 
    
    
    
    
    
    

    

    Nyberg F, 
    
    
    
    
    
    

    

    Cusi D, 
    
    
    
    
    
    

    

    Roberts R, 
    
    
    
    
    
    

    

    Newton-Cheh C, 
    
    
    
    
    
    

    

    Franke L, 
    
    
    
    
    
    

    

    Stanton A, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Hingorani A, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>American Journal of Human Genetics</em>
    
    
    2011
    
  <!--  -->
   
  
  <a href="http://europepmc.org/articles/PMC3234370" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_nat_bpgwas">
  <span class="title">Genetic Variants in Novel Pathways Influence Blood Pressure and Cardiovascular Disease Risk.</span>
  <span class="author">
    
    
    
    
    
    

    

    for Blood Pressure Genome-Wide Association Studies I, 
    
    
    
    
    
    

    

    Ehret G, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Rice K, 
    
    
    
    
    
    

    

    Bochud M, 
    
    
    
    
    
    

    

    Johnson A, 
    
    
    
    
    
    

    

    Chasman D, 
    
    
    
    
    
    

    

    Smith A, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Verwoert G, 
    
    
    
    
    
    

    

    Hwang S, 
    
    
    
    
    
    

    

    Pihur V, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    O’Reilly P, 
    
    
    
    
    
    

    

    Amin N, 
    
    
    
    
    
    

    

    Bragg-Gresham J, 
    
    
    
    
    
    

    

    Teumer A, 
    
    
    
    
    
    

    

    Glazer N, 
    
    
    
    
    
    

    

    Launer L, 
    
    
    
    
    
    

    

    Zhao J, 
    
    
    
    
    
    

    

    Aulchenko Y, 
    
    
    
    
    
    

    

    Heath S, 
    
    
    
    
    
    

    

    SÃ\textmuber S, 
    
    
    
    
    
    

    

    Parsa A, 
    
    
    
    
    
    

    

    Luan J, 
    
    
    
    
    
    

    

    Arora P, 
    
    
    
    
    
    

    

    Dehghan A, 
    
    
    
    
    
    

    

    Zhang F, 
    
    
    
    
    
    

    

    Lucas G, 
    
    
    
    
    
    

    

    Hicks A, 
    
    
    
    
    
    

    

    Jackson A, 
    
    
    
    
    
    

    

    Peden J, 
    
    
    
    
    
    

    

    Tanaka T, 
    
    
    
    
    
    

    

    Wild S, 
    
    
    
    
    
    

    

    Rudan I, 
    
    
    
    
    
    

    

    Igl W, 
    
    
    
    
    
    

    

    Milaneschi Y, 
    
    
    
    
    
    

    

    Parker A, 
    
    
    
    
    
    

    

    Fava C, 
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Fox E, 
    
    
    
    
    
    

    

    Kumari M, 
    
    
    
    
    
    

    

    Go M, 
    
    
    
    
    
    

    

    van der Harst P, 
    
    
    
    
    
    

    

    Kao W, 
    
    
    
    
    
    

    

    SjÃ\textparagraphgren M, 
    
    
    
    
    
    

    

    Vinay D, 
    
    
    
    
    
    

    

    Alexander M, 
    
    
    
    
    
    

    

    Tabara Y, 
    
    
    
    
    
    

    

    Shaw-Hawkins S, 
    
    
    
    
    
    

    

    Whincup P, 
    
    
    
    
    
    

    

    Liu Y, 
    
    
    
    
    
    

    

    Shi G, 
    
    
    
    
    
    

    

    Kuusisto J, 
    
    
    
    
    
    

    

    Tayo B, 
    
    
    
    
    
    

    

    Seielstad M, 
    
    
    
    
    
    

    

    Sim X, 
    
    
    
    
    
    

    

    Nguyen K, 
    
    
    
    
    
    

    

    LehtimÃ\textcurrencyki T, 
    
    
    
    
    
    

    

    Matullo G, 
    
    
    
    
    
    

    

    Wu Y, 
    
    
    
    
    
    

    

    Gaunt T, 
    
    
    
    
    
    

    

    Onland-Moret N, 
    
    
    
    
    
    

    

    Cooper M, 
    
    
    
    
    
    

    

    Platou C, 
    
    
    
    
    
    

    

    Org E, 
    
    
    
    
    
    

    

    Hardy R, 
    
    
    
    
    
    

    

    Dahgam S, 
    
    
    
    
    
    

    

    Palmen J, 
    
    
    
    
    
    

    

    Vitart V, 
    
    
    
    
    
    

    

    Braund P, 
    
    
    
    
    
    

    

    Kuznetsova T, 
    
    
    
    
    
    

    

    Uiterwaal C, 
    
    
    
    
    
    

    

    Adeyemo A, 
    
    
    
    
    
    

    

    Palmas W, 
    
    
    
    
    
    

    

    Campbell H, 
    
    
    
    
    
    

    

    Ludwig B, 
    
    
    
    
    
    

    

    Tomaszewski M, 
    
    
    
    
    
    

    

    Tzoulaki I, 
    
    
    
    
    
    

    

    Palmer N, 
    
    
    
    
    
    

    

    consortium C, 
    
    
    
    
    
    

    

    Consortium C, 
    
    
    
    
    
    

    

    Consortium K, 
    
    
    
    
    
    

    

    consortium E, 
    
    
    
    
    
    

    

    consortium C, 
    
    
    
    
    
    

    

    Aspelund T, 
    
    
    
    
    
    

    

    Garcia M, 
    
    
    
    
    
    

    

    Chang Y, 
    
    
    
    
    
    

    

    O’Connell J, 
    
    
    
    
    
    

    

    Steinle N, 
    
    
    
    
    
    

    

    Grobbee D, 
    
    
    
    
    
    

    

    Arking D, 
    
    
    
    
    
    

    

    Kardia S, 
    
    
    
    
    
    

    

    Morrison A, 
    
    
    
    
    
    

    

    Hernandez D, 
    
    
    
    
    
    

    

    Najjar S, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    Hadley D, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Hingorani A, 
    
    
    
    
    
    

    

    Day I, 
    
    
    
    
    
    

    

    Lawlor D, 
    
    
    
    
    
    

    

    Beilby J, 
    
    
    
    
    
    

    

    Lawrence R, 
    
    
    
    
    
    

    

    Clarke R, 
    
    
    
    
    
    

    

    Hopewell J, 
    
    
    
    
    
    

    

    Ongen H, 
    
    
    
    
    
    

    

    Dreisbach A, 
    
    
    
    
    
    

    

    Li Y, 
    
    
    
    
    
    

    

    Young J, 
    
    
    
    
    
    

    

    Bis J, 
    
    
    
    
    
    

    

    KÃ\textcurrencyhÃ\textparagraphnen M, 
    
    
    
    
    
    

    

    Viikari J, 
    
    
    
    
    
    

    

    Adair L, 
    
    
    
    
    
    

    

    Lee N, 
    
    
    
    
    
    

    

    Chen M, 
    
    
    
    
    
    

    

    Olden M, 
    
    
    
    
    
    

    

    Pattaro C, 
    
    
    
    
    
    

    

    Bolton J, 
    
    
    
    
    
    

    

    KÃ\textparagraphttgen A, 
    
    
    
    
    
    

    

    Bergmann S, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Chaturvedi N, 
    
    
    
    
    
    

    

    Frayling T, 
    
    
    
    
    
    

    

    Islam M, 
    
    
    
    
    
    

    

    Jafar T, 
    
    
    
    
    
    

    

    Erdmann J, 
    
    
    
    
    
    

    

    Kulkarni S, 
    
    
    
    
    
    

    

    Bornstein S, 
    
    
    
    
    
    

    

    GrÃ\textcurrencyssler J, 
    
    
    
    
    
    

    

    Groop L, 
    
    
    
    
    
    

    

    Voight B, 
    
    
    
    
    
    

    

    Kettunen J, 
    
    
    
    
    
    

    

    Howard P, 
    
    
    
    
    
    

    

    Taylor A, 
    
    
    
    
    
    

    

    Guarrera S, 
    
    
    
    
    
    

    

    Ricceri F, 
    
    
    
    
    
    

    

    Emilsson V, 
    
    
    
    
    
    

    

    Plump A, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    Khaw K, 
    
    
    
    
    
    

    

    Weder A, 
    
    
    
    
    
    

    

    Hunt S, 
    
    
    
    
    
    

    

    Sun Y, 
    
    
    
    
    
    

    

    Bergman R, 
    
    
    
    
    
    

    

    Collins F, 
    
    
    
    
    
    

    

    Bonnycastle L, 
    
    
    
    
    
    

    

    Scott L, 
    
    
    
    
    
    

    

    Stringham H, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Perola M, 
    
    
    
    
    
    

    

    Vartiainen E, 
    
    
    
    
    
    

    

    Brand S, 
    
    
    
    
    
    

    

    Staessen J, 
    
    
    
    
    
    

    

    Wang T, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Artigas M, 
    
    
    
    
    
    

    

    Dong Y, 
    
    
    
    
    
    

    

    Snieder H, 
    
    
    
    
    
    

    

    Wang X, 
    
    
    
    
    
    

    

    Zhu H, 
    
    
    
    
    
    

    

    Lohman K, 
    
    
    
    
    
    

    

    Rudock M, 
    
    
    
    
    
    

    

    Heckbert S, 
    
    
    
    
    
    

    

    Smith N, 
    
    
    
    
    
    

    

    Wiggins K, 
    
    
    
    
    
    

    

    Doumatey A, 
    
    
    
    
    
    

    

    Shriner D, 
    
    
    
    
    
    

    

    Veldre G, 
    
    
    
    
    
    

    

    Viigimaa M, 
    
    
    
    
    
    

    

    Kinra S, 
    
    
    
    
    
    

    

    Prabhakaran D, 
    
    
    
    
    
    

    

    Tripathy V, 
    
    
    
    
    
    

    

    Langefeld C, 
    
    
    
    
    
    

    

    Rosengren A, 
    
    
    
    
    
    

    

    Thelle D, 
    
    
    
    
    
    

    

    Corsi A, 
    
    
    
    
    
    

    

    Singleton A, 
    
    
    
    
    
    

    

    Forrester T, 
    
    
    
    
    
    

    

    Hilton G, 
    
    
    
    
    
    

    

    McKenzie C, 
    
    
    
    
    
    

    

    Salako T, 
    
    
    
    
    
    

    

    Iwai N, 
    
    
    
    
    
    

    

    Kita Y, 
    
    
    
    
    
    

    

    Ogihara T, 
    
    
    
    
    
    

    

    Ohkubo T, 
    
    
    
    
    
    

    

    Okamura T, 
    
    
    
    
    
    

    

    Ueshima H, 
    
    
    
    
    
    

    

    Umemura S, 
    
    
    
    
    
    

    

    Eyheramendy S, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Cho Y, 
    
    
    
    
    
    

    

    Kim H, 
    
    
    
    
    
    

    

    Lee J, 
    
    
    
    
    
    

    

    Scott J, 
    
    
    
    
    
    

    

    Sehmi J, 
    
    
    
    
    
    

    

    Zhang W, 
    
    
    
    
    
    

    

    Hedblad B, 
    
    
    
    
    
    

    

    Nilsson P, 
    
    
    
    
    
    

    

    Smith G, 
    
    
    
    
    
    

    

    Wong A, 
    
    
    
    
    
    

    

    Narisu N, 
    
    
    
    
    
    

    

    StanÄÃ\textexclamdownkovÃ\textexclamdown A, 
    
    
    
    
    
    

    

    Raffel L, 
    
    
    
    
    
    

    

    Yao J, 
    
    
    
    
    
    

    

    Kathiresan S, 
    
    
    
    
    
    

    

    O’Donnell C, 
    
    
    
    
    
    

    

    Schwartz S, 
    
    
    
    
    
    

    

    Ikram M, 
    
    
    
    
    
    

    

    Longstreth W, 
    
    
    
    
    
    

    

    Mosley T, 
    
    
    
    
    
    

    

    Seshadri S, 
    
    
    
    
    
    

    

    Shrine N, 
    
    
    
    
    
    

    

    Wain L, 
    
    
    
    
    
    

    

    Morken M, 
    
    
    
    
    
    

    

    Swift A, 
    
    
    
    
    
    

    

    Laitinen J, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Zitting P, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Humphries S, 
    
    
    
    
    
    

    

    Danesh J, 
    
    
    
    
    
    

    

    Rasheed A, 
    
    
    
    
    
    

    

    Goel A, 
    
    
    
    
    
    

    

    Hamsten A, 
    
    
    
    
    
    

    

    Watkins H, 
    
    
    
    
    
    

    

    Bakker S, 
    
    
    
    
    
    

    

    van Gilst W, 
    
    
    
    
    
    

    

    Janipalli C, 
    
    
    
    
    
    

    

    Mani K, 
    
    
    
    
    
    

    

    Yajnik C, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Mattace-Raso F, 
    
    
    
    
    
    

    

    Oostra B, 
    
    
    
    
    
    

    

    Demirkan A, 
    
    
    
    
    
    

    

    Isaacs A, 
    
    
    
    
    
    

    

    Rivadeneira F, 
    
    
    
    
    
    

    

    Lakatta E, 
    
    
    
    
    
    

    

    Orru M, 
    
    
    
    
    
    

    

    Scuteri A, 
    
    
    
    
    
    

    

    Ala-Korpela M, 
    
    
    
    
    
    

    

    Kangas A, 
    
    
    
    
    
    

    

    LyytikÃ\textcurrencyinen L, 
    
    
    
    
    
    

    

    Soininen P, 
    
    
    
    
    
    

    

    Tukiainen T, 
    
    
    
    
    
    

    

    WÃ\frac14rtz P, 
    
    
    
    
    
    

    

    Ong R, 
    
    
    
    
    
    

    

    DÃ\textparagraphrr M, 
    
    
    
    
    
    

    

    Kroemer H, 
    
    
    
    
    
    

    

    VÃ\textparagraphlker U, 
    
    
    
    
    
    

    

    VÃ\textparagraphlzke H, 
    
    
    
    
    
    

    

    Galan P, 
    
    
    
    
    
    

    

    Hercberg S, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Zelenika D, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Zhai G, 
    
    
    
    
    
    

    

    Meschia J, 
    
    
    
    
    
    

    

    Nalls M, 
    
    
    
    
    
    

    

    Sharma P, 
    
    
    
    
    
    

    

    Terzic J, 
    
    
    
    
    
    

    

    Kumar M, 
    
    
    
    
    
    

    

    Denniff M, 
    
    
    
    
    
    

    

    Zukowska-Szczechowska E, 
    
    
    
    
    
    

    

    Wagenknecht L, 
    
    
    
    
    
    

    

    Fowkes F, 
    
    
    
    
    
    

    

    Charchar F, 
    
    
    
    
    
    

    

    Schwarz P, 
    
    
    
    
    
    

    

    Hayward C, 
    
    
    
    
    
    

    

    Guo X, 
    
    
    
    
    
    

    

    Rotimi C, 
    
    
    
    
    
    

    

    Bots M, 
    
    
    
    
    
    

    

    Brand E, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Polasek O, 
    
    
    
    
    
    

    

    Talmud P, 
    
    
    
    
    
    

    

    Nyberg F, 
    
    
    
    
    
    

    

    Kuh D, 
    
    
    
    
    
    

    

    Laan M, 
    
    
    
    
    
    

    

    Hveem K, 
    
    
    
    
    
    

    

    Palmer L, 
    
    
    
    
    
    

    

    van der Schouw Y, 
    
    
    
    
    
    

    

    Casas J, 
    
    
    
    
    
    

    

    Mohlke K, 
    
    
    
    
    
    

    

    Vineis P, 
    
    
    
    
    
    

    

    Raitakari O, 
    
    
    
    
    
    

    

    Ganesh S, 
    
    
    
    
    
    

    

    Wong T, 
    
    
    
    
    
    

    

    Tai E, 
    
    
    
    
    
    

    

    Cooper R, 
    
    
    
    
    
    

    

    Laakso M, 
    
    
    
    
    
    

    

    Rao D, 
    
    
    
    
    
    

    

    Harris T, 
    
    
    
    
    
    

    

    Morris R, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Kivimaki M, 
    
    
    
    
    
    

    

    Marmot M, 
    
    
    
    
    
    

    

    Miki T, 
    
    
    
    
    
    

    

    Saleheen D, 
    
    
    
    
    
    

    

    Chandak G, 
    
    
    
    
    
    

    

    Coresh J, 
    
    
    
    
    
    

    

    Navis G, 
    
    
    
    
    
    

    

    Salomaa V, 
    
    
    
    
    
    

    

    Han B, 
    
    
    
    
    
    

    

    Zhu X, 
    
    
    
    
    
    

    

    Kooner J, 
    
    
    
    
    
    

    

    Melander O, 
    
    
    
    
    
    

    

    Ridker P, 
    
    
    
    
    
    

    

    Bandinelli S, 
    
    
    
    
    
    

    

    Gyllensten U, 
    
    
    
    
    
    

    

    Wright A, 
    
    
    
    
    
    

    

    Wilson J, 
    
    
    
    
    
    

    

    Ferrucci L, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Tuomilehto J, 
    
    
    
    
    
    

    

    Pramstaller P, 
    
    
    
    
    
    

    

    Elosua R, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Sijbrands E, 
    
    
    
    
    
    

    

    Altshuler D, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Shuldiner A, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    Meneton P, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Gudnason V, 
    
    
    
    
    
    

    

    Rotter J, 
    
    
    
    
    
    

    

    Rettig R, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Witteman J, 
    
    
    
    
    
    

    

    Hartikainen A, 
    
    
    
    
    
    

    

    Beckmann J, 
    
    
    
    
    
    

    

    Boerwinkle E, 
    
    
    
    
    
    

    

    Vasan R, 
    
    
    
    
    
    

    

    Boehnke M, 
    
    
    
    
    
    

    

    Larson M, 
    
    
    
    
    
    

    

    JÃ\textcurrencyrvelin M, 
    
    
    
    
    
    

    

    Psaty B, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Chakravarti A, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    van Duijn C, 
    
    
    
    
    
    

    

    Newton-Cheh C, 
    
    
    
    
    
    

    

    Levy D, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Tang H, 
    
    
    
    
    
    

    

    Knowles J, 
    
    
    
    
    
    

    

    Hlatky M, 
    
    
    
    
    
    

    

    Fortmann S, 
    
    
    
    
    
    

    

    Assimes T, 
    
    
    
    
    
    

    

    Quertermous T, 
    
    
    
    
    
    

    

    Go A, 
    
    
    
    
    
    

    

    Iribarren C, 
    
    
    
    
    
    

    

    Absher D, 
    
    
    
    
    
    

    

    Risch N, 
    
    
    
    
    
    

    

    Myers R, 
    
    
    
    
    
    

    

    Sidney S, 
    
    
    
    
    
    

    

    Ziegler A, 
    
    
    
    
    
    

    

    Schillert A, 
    
    
    
    
    
    

    

    Bickel C, 
    
    
    
    
    
    

    

    Sinning C, 
    
    
    
    
    
    

    

    Rupprecht H, 
    
    
    
    
    
    

    

    Lackner K, 
    
    
    
    
    
    

    

    Wild P, 
    
    
    
    
    
    

    

    Schnabel R, 
    
    
    
    
    
    

    

    Blankenberg S, 
    
    
    
    
    
    

    

    Zeller T, 
    
    
    
    
    
    

    

    MÃ\frac14nzel T, 
    
    
    
    
    
    

    

    Perret C, 
    
    
    
    
    
    

    

    Cambien F, 
    
    
    
    
    
    

    

    Tiret L, 
    
    
    
    
    
    

    

    Nicaud V, 
    
    
    
    
    
    

    

    Proust C, 
    
    
    
    
    
    

    

    Dehghan A, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    van Duijn C, 
    
    
    
    
    
    

    

    Levy D, 
    
    
    
    
    
    

    

    Whitteman J, 
    
    
    
    
    
    

    

    Cupples L, 
    
    
    
    
    
    

    

    Demissie-Banjaw S, 
    
    
    
    
    
    

    

    Ramachandran V, 
    
    
    
    
    
    

    

    Smith A, 
    
    
    
    
    
    

    

    Gudnason V, 
    
    
    
    
    
    

    

    Boerwinkle E, 
    
    
    
    
    
    

    

    Folsom A, 
    
    
    
    
    
    

    

    Morrison A, 
    
    
    
    
    
    

    

    Psaty B, 
    
    
    
    
    
    

    

    Chen I, 
    
    
    
    
    
    

    

    Rotter J, 
    
    
    
    
    
    

    

    Bis J, 
    
    
    
    
    
    

    

    Volcik K, 
    
    
    
    
    
    

    

    Rice K, 
    
    
    
    
    
    

    

    Taylor K, 
    
    
    
    
    
    

    

    Marciante K, 
    
    
    
    
    
    

    

    Smith N, 
    
    
    
    
    
    

    

    Glazer N, 
    
    
    
    
    
    

    

    Heckbert S, 
    
    
    
    
    
    

    

    Harris T, 
    
    
    
    
    
    

    

    Lumley T, 
    
    
    
    
    
    

    

    Kong A, 
    
    
    
    
    
    

    

    Thorleifsson G, 
    
    
    
    
    
    

    

    Thorgeirsson G, 
    
    
    
    
    
    

    

    Holm H, 
    
    
    
    
    
    

    

    Gulcher J, 
    
    
    
    
    
    

    

    Stefansson K, 
    
    
    
    
    
    

    

    Andersen K, 
    
    
    
    
    
    

    

    Gretarsdottir S, 
    
    
    
    
    
    

    

    Thorsteinsdottir U, 
    
    
    
    
    
    

    

    Preuss M, 
    
    
    
    
    
    

    

    Schreiber S, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    KÃ\textparagraphnig I, 
    
    
    
    
    
    

    

    Lieb W, 
    
    
    
    
    
    

    

    Hengstenberg C, 
    
    
    
    
    
    

    

    Schunkert H, 
    
    
    
    
    
    

    

    Erdmann J, 
    
    
    
    
    
    

    

    Fischer M, 
    
    
    
    
    
    

    

    Grosshennig A, 
    
    
    
    
    
    

    

    Medack A, 
    
    
    
    
    
    

    

    Stark K, 
    
    
    
    
    
    

    

    Linsel-Nitschke P, 
    
    
    
    
    
    

    

    Bruse P, 
    
    
    
    
    
    

    

    Aherrahrou Z, 
    
    
    
    
    
    

    

    Peters A, 
    
    
    
    
    
    

    

    Loley C, 
    
    
    
    
    
    

    

    Willenborg C, 
    
    
    
    
    
    

    

    Nahrstedt J, 
    
    
    
    
    
    

    

    Freyer J, 
    
    
    
    
    
    

    

    Gulde S, 
    
    
    
    
    
    

    

    Doering A, 
    
    
    
    
    
    

    

    Meisinger C, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Klopp N, 
    
    
    
    
    
    

    

    Illig T, 
    
    
    
    
    
    

    

    Meinitzer A, 
    
    
    
    
    
    

    

    Tomaschitz A, 
    
    
    
    
    
    

    

    Halperin E, 
    
    
    
    
    
    

    

    Dobnig H, 
    
    
    
    
    
    

    

    Scharnagl H, 
    
    
    
    
    
    

    

    Kleber M, 
    
    
    
    
    
    

    

    Laaksonen R, 
    
    
    
    
    
    

    

    Pilz S, 
    
    
    
    
    
    

    

    Grammer T, 
    
    
    
    
    
    

    

    Stojakovic T, 
    
    
    
    
    
    

    

    Renner W, 
    
    
    
    
    
    

    

    MÃ\textcurrencyrz W, 
    
    
    
    
    
    

    

    BÃ\textparagraphhm B, 
    
    
    
    
    
    

    

    Winkelmann B, 
    
    
    
    
    
    

    

    Winkler K, 
    
    
    
    
    
    

    

    Hoffmann M, 
    
    
    
    
    
    

    

    O’Donnell C, 
    
    
    
    
    
    

    

    Voight B, 
    
    
    
    
    
    

    

    Altshuler D, 
    
    
    
    
    
    

    

    Siscovick D, 
    
    
    
    
    
    

    

    Musunuru K, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Barbalic M, 
    
    
    
    
    
    

    

    Melander O, 
    
    
    
    
    
    

    

    Elosua R, 
    
    
    
    
    
    

    

    Kathiresan S, 
    
    
    
    
    
    

    

    Schwartz S, 
    
    
    
    
    
    

    

    Salomaa V, 
    
    
    
    
    
    

    

    Guiducci C, 
    
    
    
    
    
    

    

    Burtt N, 
    
    
    
    
    
    

    

    Gabriel S, 
    
    
    
    
    
    

    

    Stewart A, 
    
    
    
    
    
    

    

    Wells G, 
    
    
    
    
    
    

    

    Chen L, 
    
    
    
    
    
    

    

    Jarinova O, 
    
    
    
    
    
    

    

    Roberts R, 
    
    
    
    
    
    

    

    McPherson R, 
    
    
    
    
    
    

    

    Dandona S, 
    
    
    
    
    
    

    

    Pichard A, 
    
    
    
    
    
    

    

    Rader D, 
    
    
    
    
    
    

    

    Devaney J, 
    
    
    
    
    
    

    

    Lindsay J, 
    
    
    
    
    
    

    

    Kent K, 
    
    
    
    
    
    

    

    Qu L, 
    
    
    
    
    
    

    

    Satler L, 
    
    
    
    
    
    

    

    Burnett M, 
    
    
    
    
    
    

    

    Li M, 
    
    
    
    
    
    

    

    Reilly M, 
    
    
    
    
    
    

    

    Wilensky R, 
    
    
    
    
    
    

    

    Waksman R, 
    
    
    
    
    
    

    

    Epstein S, 
    
    
    
    
    
    

    

    Matthai W, 
    
    
    
    
    
    

    

    Knouff C, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Hakonarson H, 
    
    
    
    
    
    

    

    Walker M, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Balmforth A, 
    
    
    
    
    
    

    

    Wright B, 
    
    
    
    
    
    

    

    Nelson C, 
    
    
    
    
    
    

    

    Thompson J, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Braund P, 
    
    
    
    
    
    

    

    Ball S, 
    
    
    
    
    
    

    

    Smith N, 
    
    
    
    
    
    

    

    Felix J, 
    
    
    
    
    
    

    

    Morrison A, 
    
    
    
    
    
    

    

    Demissie S, 
    
    
    
    
    
    

    

    Glazer N, 
    
    
    
    
    
    

    

    Loehr L, 
    
    
    
    
    
    

    

    Cupples L, 
    
    
    
    
    
    

    

    Dehghan A, 
    
    
    
    
    
    

    

    Lumley T, 
    
    
    
    
    
    

    

    Rosamond W, 
    
    
    
    
    
    

    

    Lieb W, 
    
    
    
    
    
    

    

    Rivadeneira F, 
    
    
    
    
    
    

    

    Bis J, 
    
    
    
    
    
    

    

    Folsom A, 
    
    
    
    
    
    

    

    Benjamin E, 
    
    
    
    
    
    

    

    Aulchenko Y, 
    
    
    
    
    
    

    

    Haritunians T, 
    
    
    
    
    
    

    

    Couper D, 
    
    
    
    
    
    

    

    Murabito J, 
    
    
    
    
    
    

    

    Wang Y, 
    
    
    
    
    
    

    

    Stricker B, 
    
    
    
    
    
    

    

    Gottdiener J, 
    
    
    
    
    
    

    

    Chang P, 
    
    
    
    
    
    

    

    Wang T, 
    
    
    
    
    
    

    

    Rice K, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Heckbert S, 
    
    
    
    
    
    

    

    Fox E, 
    
    
    
    
    
    

    

    O’Donnell C, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    Rotter J, 
    
    
    
    
    
    

    

    Willerson J, 
    
    
    
    
    
    

    

    Levy D, 
    
    
    
    
    
    

    

    van Duijn C, 
    
    
    
    
    
    

    

    Psaty B, 
    
    
    
    
    
    

    

    Witteman J, 
    
    
    
    
    
    

    

    Boerwinkle E, 
    
    
    
    
    
    

    

    Vasan R, 
    
    
    
    
    
    

    

    KÃ\textparagraphttgen A, 
    
    
    
    
    
    

    

    Pattaro C, 
    
    
    
    
    
    

    

    BÃ\textparagraphger C, 
    
    
    
    
    
    

    

    Fuchsberger C, 
    
    
    
    
    
    

    

    Olden M, 
    
    
    
    
    
    

    

    Glazer N, 
    
    
    
    
    
    

    

    Parsa A, 
    
    
    
    
    
    

    

    Gao X, 
    
    
    
    
    
    

    

    Yang Q, 
    
    
    
    
    
    

    

    Smith A, 
    
    
    
    
    
    

    

    O’Connell J, 
    
    
    
    
    
    

    

    Li M, 
    
    
    
    
    
    

    

    Schmidt H, 
    
    
    
    
    
    

    

    Tanaka T, 
    
    
    
    
    
    

    

    Isaacs A, 
    
    
    
    
    
    

    

    Ketkar S, 
    
    
    
    
    
    

    

    Hwang S, 
    
    
    
    
    
    

    

    Johnson A, 
    
    
    
    
    
    

    

    Dehghan A, 
    
    
    
    
    
    

    

    Teumer A, 
    
    
    
    
    
    

    

    ParÃ\textcopyright G, 
    
    
    
    
    
    

    

    Atkinson E, 
    
    
    
    
    
    

    

    Zeller T, 
    
    
    
    
    
    

    

    Lohman K, 
    
    
    
    
    
    

    

    Cornelis M, 
    
    
    
    
    
    

    

    Probst-Hensch N, 
    
    
    
    
    
    

    

    Kronenberg F, 
    
    
    
    
    
    

    

    TÃ\textparagraphnjes A, 
    
    
    
    
    
    

    

    Hayward C, 
    
    
    
    
    
    

    

    Aspelund T, 
    
    
    
    
    
    

    

    Eiriksdottir G, 
    
    
    
    
    
    

    

    Launer L, 
    
    
    
    
    
    

    

    Harris T, 
    
    
    
    
    
    

    

    Rampersaud E, 
    
    
    
    
    
    

    

    Mitchell B, 
    
    
    
    
    
    

    

    Arking D, 
    
    
    
    
    
    

    

    Boerwinkle E, 
    
    
    
    
    
    

    

    Struchalin M, 
    
    
    
    
    
    

    

    Cavalieri M, 
    
    
    
    
    
    

    

    Singleton A, 
    
    
    
    
    
    

    

    Giallauria F, 
    
    
    
    
    
    

    

    Metter J, 
    
    
    
    
    
    

    

    de Boer J, 
    
    
    
    
    
    

    

    Haritunians T, 
    
    
    
    
    
    

    

    Lumley T, 
    
    
    
    
    
    

    

    Siscovick D, 
    
    
    
    
    
    

    

    Psaty B, 
    
    
    
    
    
    

    

    Zillikens M, 
    
    
    
    
    
    

    

    Oostra B, 
    
    
    
    
    
    

    

    Feitosa M, 
    
    
    
    
    
    

    

    Province M, 
    
    
    
    
    
    

    

    de Andrade M, 
    
    
    
    
    
    

    

    Turner S, 
    
    
    
    
    
    

    

    Schillert A, 
    
    
    
    
    
    

    

    Ziegler A, 
    
    
    
    
    
    

    

    Wild P, 
    
    
    
    
    
    

    

    Schnabel R, 
    
    
    
    
    
    

    

    Wilde S, 
    
    
    
    
    
    

    

    Munzel T, 
    
    
    
    
    
    

    

    Leak T, 
    
    
    
    
    
    

    

    Illig T, 
    
    
    
    
    
    

    

    Klopp N, 
    
    
    
    
    
    

    

    Meisinger C, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Koenig W, 
    
    
    
    
    
    

    

    Zgaga L, 
    
    
    
    
    
    

    

    Zemunik T, 
    
    
    
    
    
    

    

    Kolcic I, 
    
    
    
    
    
    

    

    Minelli C, 
    
    
    
    
    
    

    

    Hu F, 
    
    
    
    
    
    

    

    Johansson A, 
    
    
    
    
    
    

    

    Igl W, 
    
    
    
    
    
    

    

    Zaboli G, 
    
    
    
    
    
    

    

    Wild S, 
    
    
    
    
    
    

    

    Wright A, 
    
    
    
    
    
    

    

    Campbell H, 
    
    
    
    
    
    

    

    Ellinghaus D, 
    
    
    
    
    
    

    

    Schreiber S, 
    
    
    
    
    
    

    

    Aulchenko Y, 
    
    
    
    
    
    

    

    Felix J, 
    
    
    
    
    
    

    

    Rivadeneira F, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Imboden M, 
    
    
    
    
    
    

    

    Nitsch D, 
    
    
    
    
    
    

    

    BrandstÃ\textcurrencytter A, 
    
    
    
    
    
    

    

    Kollerits B, 
    
    
    
    
    
    

    

    Kedenko L, 
    
    
    
    
    
    

    

    MÃ\textcurrencygi R, 
    
    
    
    
    
    

    

    Stumvoll M, 
    
    
    
    
    
    

    

    Kovacs P, 
    
    
    
    
    
    

    

    Boban M, 
    
    
    
    
    
    

    

    Campbell S, 
    
    
    
    
    
    

    

    Endlich K, 
    
    
    
    
    
    

    

    VÃ\textparagraphlzke H, 
    
    
    
    
    
    

    

    Kroemer H, 
    
    
    
    
    
    

    

    Nauck M, 
    
    
    
    
    
    

    

    VÃ\textparagraphlker U, 
    
    
    
    
    
    

    

    Polasek O, 
    
    
    
    
    
    

    

    Vitart V, 
    
    
    
    
    
    

    

    Badola S, 
    
    
    
    
    
    

    

    Parker A, 
    
    
    
    
    
    

    

    Ridker P, 
    
    
    
    
    
    

    

    Kardia S, 
    
    
    
    
    
    

    

    Blankenberg S, 
    
    
    
    
    
    

    

    Liu Y, 
    
    
    
    
    
    

    

    Curhan G, 
    
    
    
    
    
    

    

    Franke A, 
    
    
    
    
    
    

    

    Rochat T, 
    
    
    
    
    
    

    

    Paulweber B, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Wang W, 
    
    
    
    
    
    

    

    Gudnason V, 
    
    
    
    
    
    

    

    Shuldiner A, 
    
    
    
    
    
    

    

    Coresh J, 
    
    
    
    
    
    

    

    Schmidt R, 
    
    
    
    
    
    

    

    Ferrucci L, 
    
    
    
    
    
    

    

    Shlipak M, 
    
    
    
    
    
    

    

    van Duijn C, 
    
    
    
    
    
    

    

    Borecki I, 
    
    
    
    
    
    

    

    KrÃ\textcurrencymer B, 
    
    
    
    
    
    

    

    Rudan I, 
    
    
    
    
    
    

    

    Gyllensten U, 
    
    
    
    
    
    

    

    Wilson J, 
    
    
    
    
    
    

    

    Witteman J, 
    
    
    
    
    
    

    

    Pramstaller P, 
    
    
    
    
    
    

    

    Rettig R, 
    
    
    
    
    
    

    

    Hastie N, 
    
    
    
    
    
    

    

    Chasman D, 
    
    
    
    
    
    

    

    Kao W, 
    
    
    
    
    
    

    

    Heid I, 
    
    
    
    
    
    

    

    Fox C, 
    
    
    
    
    
    

    

    Vasan R, 
    
    
    
    
    
    

    

    Glazer N, 
    
    
    
    
    
    

    

    Felix J, 
    
    
    
    
    
    

    

    Lieb W, 
    
    
    
    
    
    

    

    Wild P, 
    
    
    
    
    
    

    

    Felix S, 
    
    
    
    
    
    

    

    Watzinger N, 
    
    
    
    
    
    

    

    Larson M, 
    
    
    
    
    
    

    

    Smith N, 
    
    
    
    
    
    

    

    Dehghan A, 
    
    
    
    
    
    

    

    Grosshennig A, 
    
    
    
    
    
    

    

    Schillert A, 
    
    
    
    
    
    

    

    Teumer A, 
    
    
    
    
    
    

    

    Schmidt R, 
    
    
    
    
    
    

    

    Kathiresan S, 
    
    
    
    
    
    

    

    Lumley T, 
    
    
    
    
    
    

    

    Aulchenko Y, 
    
    
    
    
    
    

    

    KÃ\textparagraphnig I, 
    
    
    
    
    
    

    

    Zeller T, 
    
    
    
    
    
    

    

    Homuth G, 
    
    
    
    
    
    

    

    Struchalin M, 
    
    
    
    
    
    

    

    Aragam J, 
    
    
    
    
    
    

    

    Bis J, 
    
    
    
    
    
    

    

    Rivadeneira F, 
    
    
    
    
    
    

    

    Erdmann J, 
    
    
    
    
    
    

    

    Schnabel R, 
    
    
    
    
    
    

    

    DÃ\textparagraphrr M, 
    
    
    
    
    
    

    

    Zweiker R, 
    
    
    
    
    
    

    

    Lind L, 
    
    
    
    
    
    

    

    Rodeheffer R, 
    
    
    
    
    
    

    

    Greiser K, 
    
    
    
    
    
    

    

    Levy D, 
    
    
    
    
    
    

    

    Haritunians T, 
    
    
    
    
    
    

    

    Deckers J, 
    
    
    
    
    
    

    

    Stritzke J, 
    
    
    
    
    
    

    

    Lackner K, 
    
    
    
    
    
    

    

    VÃ\textparagraphlker U, 
    
    
    
    
    
    

    

    Ingelsson E, 
    
    
    
    
    
    

    

    Kullo I, 
    
    
    
    
    
    

    

    Haerting J, 
    
    
    
    
    
    

    

    O’Donnell C, 
    
    
    
    
    
    

    

    Heckbert S, 
    
    
    
    
    
    

    

    Stricker B, 
    
    
    
    
    
    

    

    Ziegler A, 
    
    
    
    
    
    

    

    Reffelmann T, 
    
    
    
    
    
    

    

    Redfield M, 
    
    
    
    
    
    

    

    Werdan K, 
    
    
    
    
    
    

    

    Mitchell G, 
    
    
    
    
    
    

    

    Rice K, 
    
    
    
    
    
    

    

    Arnett D, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Gottdiener J, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Blettner M, 
    
    
    
    
    
    

    

    Friedrich N, 
    
    
    
    
    
    

    

    Wang T, 
    
    
    
    
    
    

    

    Psaty B, 
    
    
    
    
    
    

    

    van Duijn C, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Munzel T, 
    
    
    
    
    
    

    

    Kroemer H, 
    
    
    
    
    
    

    

    Benjamin E, 
    
    
    
    
    
    

    

    Rotter J, 
    
    
    
    
    
    

    

    Witteman J, 
    
    
    
    
    
    

    

    Schunkert H, 
    
    
    
    
    
    

    

    Schmidt H, 
    
    
    
    
    
    

    

    Valzke H, 
    
    
    
    
    
    

    

    Blankenberg S, 
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Zhang W, 
    
    
    
    
    
    

    

    Lord G, 
    
    
    
    
    
    

    

    van der Harst P, 
    
    
    
    
    
    

    

    Lawlor D, 
    
    
    
    
    
    

    

    Sehmi J, 
    
    
    
    
    
    

    

    Gale D, 
    
    
    
    
    
    

    

    Wass M, 
    
    
    
    
    
    

    

    Ahmadi K, 
    
    
    
    
    
    

    

    Bakker S, 
    
    
    
    
    
    

    

    Beckmann J, 
    
    
    
    
    
    

    

    Bilo H, 
    
    
    
    
    
    

    

    Bochud M, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Cook H, 
    
    
    
    
    
    

    

    Cotlarciuc I, 
    
    
    
    
    
    

    

    Smith G, 
    
    
    
    
    
    

    

    de Silva R, 
    
    
    
    
    
    

    

    Deng G, 
    
    
    
    
    
    

    

    Devuyst O, 
    
    
    
    
    
    

    

    Dikkeschei L, 
    
    
    
    
    
    

    

    Dimkovic N, 
    
    
    
    
    
    

    

    Dockrell M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Ebrahim S, 
    
    
    
    
    
    

    

    Eggermann T, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Ferrucci L, 
    
    
    
    
    
    

    

    Floege J, 
    
    
    
    
    
    

    

    Forouhi N, 
    
    
    
    
    
    

    

    Gansevoort R, 
    
    
    
    
    
    

    

    Han X, 
    
    
    
    
    
    

    

    Hedblad B, 
    
    
    
    
    
    

    

    van der Heide J, 
    
    
    
    
    
    

    

    Hepkema B, 
    
    
    
    
    
    

    

    Hernandez-Fuentes M, 
    
    
    
    
    
    

    

    Hypponen E, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    de Jong P, 
    
    
    
    
    
    

    

    Kleefstra N, 
    
    
    
    
    
    

    

    Lagou V, 
    
    
    
    
    
    

    

    Lapsley M, 
    
    
    
    
    
    

    

    Li Y, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Luan J, 
    
    
    
    
    
    

    

    Luttropp K, 
    
    
    
    
    
    

    

    MarÃ\textcopyrightchal C, 
    
    
    
    
    
    

    

    Melander O, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Nordfors L, 
    
    
    
    
    
    

    

    Parsa A, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Penninx B, 
    
    
    
    
    
    

    

    Perucha E, 
    
    
    
    
    
    

    

    Pouta A, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Roderick P, 
    
    
    
    
    
    

    

    Ruokonen A, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Schalling M, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Schlieper G, 
    
    
    
    
    
    

    

    Seelen M, 
    
    
    
    
    
    

    

    Shuldiner A, 
    
    
    
    
    
    

    

    SjÃ\textparagraphgren M, 
    
    
    
    
    
    

    

    Smit J, 
    
    
    
    
    
    

    

    Snieder H, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Stenvinkel P, 
    
    
    
    
    
    

    

    Sternberg M, 
    
    
    
    
    
    

    

    Swaminathan R, 
    
    
    
    
    
    

    

    Tanaka T, 
    
    
    
    
    
    

    

    Ubink-Veltmaat L, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Zerres K, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Maxwell P, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Jarvelin M, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Lightstone L, 
    
    
    
    
    
    

    

    Scott J, 
    
    
    
    
    
    

    

    Navis G, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Kooner J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nature</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/nature10405" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Blood pressure is a heritable trait influenced by several biological pathways and responsive to environmental stimuli. Over one billion people worldwide have hypertension (â\textyen 140âmmâHg systolic blood pressure orâ â\textyen 90âmmâHg diastolic blood pressure). Even small increments in blood pressure are associated with an increased risk of cardiovascular events. This genome-wide association study of systolic and diastolic blood pressure, which used a multi-stage design in 200,000 individuals of European descent, identified sixteen novel loci: six of these loci contain genes previously known or suspected to regulate blood pressure (GUCY1A3-GUCY1B3, NPR3-C5orf23, ADM, FURIN-FES, GOSR2, GNAS-EDN3); the other ten provide new clues to blood pressure physiology. A genetic risk score based on 29 genome-wide significant variants was associated with hypertension, left ventricular wall thickness, stroke and coronary artery disease, but not kidney disease or kidney function. We also observed associations with blood pressure in East Asian, South Asian and African ancestry individuals. Our findings provide new insights into the genetics and biology of blood pressure, and suggest potential novel therapeutic pathways for cardiovascular disease prevention.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_ng_chambers">
  <span class="title">Genome-Wide Association Study Identifies Loci Influencing Concentrations of Liver Enzymes in Plasma.</span>
  <span class="author">
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Zhang W, 
    
    
    
    
    
    

    

    Sehmi J, 
    
    
    
    
    
    

    

    Li X, 
    
    
    
    
    
    

    

    Wass M, 
    
    
    
    
    
    

    

    Van der Harst P, 
    
    
    
    
    
    

    

    Holm H, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Kavousi M, 
    
    
    
    
    
    

    

    Baumeister S, 
    
    
    
    
    
    

    

    Coin L, 
    
    
    
    
    
    

    

    Deng G, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    Heard-Costa N, 
    
    
    
    
    
    

    

    Hottenga J, 
    
    
    
    
    
    

    

    Kühnel B, 
    
    
    
    
    
    

    

    Kumar V, 
    
    
    
    
    
    

    

    Lagou V, 
    
    
    
    
    
    

    

    Liang L, 
    
    
    
    
    
    

    

    Luan J, 
    
    
    
    
    
    

    

    Vidal P, 
    
    
    
    
    
    

    

    Mateo Leach I, 
    
    
    
    
    
    

    

    O’Reilly P, 
    
    
    
    
    
    

    

    Peden J, 
    
    
    
    
    
    

    

    Rahmioglu N, 
    
    
    
    
    
    

    

    Soininen P, 
    
    
    
    
    
    

    

    Speliotes E, 
    
    
    
    
    
    

    

    Yuan X, 
    
    
    
    
    
    

    

    Thorleifsson G, 
    
    
    
    
    
    

    

    Alizadeh B, 
    
    
    
    
    
    

    

    Atwood L, 
    
    
    
    
    
    

    

    Borecki I, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Charoen P, 
    
    
    
    
    
    

    

    Cucca F, 
    
    
    
    
    
    

    

    Das D, 
    
    
    
    
    
    

    

    de Geus E, 
    
    
    
    
    
    

    

    Dixon A, 
    
    
    
    
    
    

    

    Döring A, 
    
    
    
    
    
    

    

    Ehret G, 
    
    
    
    
    
    

    

    Eyjolfsson G, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Forouhi N, 
    
    
    
    
    
    

    

    Friedrich N, 
    
    
    
    
    
    

    

    Goessling W, 
    
    
    
    
    
    

    

    Gudbjartsson D, 
    
    
    
    
    
    

    

    Harris T, 
    
    
    
    
    
    

    

    Hartikainen A, 
    
    
    
    
    
    

    

    Heath S, 
    
    
    
    
    
    

    

    Hirschfield G, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Homuth G, 
    
    
    
    
    
    

    

    Hyppönen E, 
    
    
    
    
    
    

    

    Janssen H, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Kangas A, 
    
    
    
    
    
    

    

    Kema I, 
    
    
    
    
    
    

    

    Kühn J, 
    
    
    
    
    
    

    

    Lai S, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Lerch M, 
    
    
    
    
    
    

    

    Li Y, 
    
    
    
    
    
    

    

    Liang T, 
    
    
    
    
    
    

    

    Lin J, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Martin N, 
    
    
    
    
    
    

    

    Moffatt M, 
    
    
    
    
    
    

    

    Montgomery G, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Musunuru K, 
    
    
    
    
    
    

    

    Nakamura Y, 
    
    
    
    
    
    

    

    O’Donnell C, 
    
    
    
    
    
    

    

    Olafsson I, 
    
    
    
    
    
    

    

    Penninx B, 
    
    
    
    
    
    

    

    Pouta A, 
    
    
    
    
    
    

    

    Prins B, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Puls R, 
    
    
    
    
    
    

    

    Ruokonen A, 
    
    
    
    
    
    

    

    Savolainen M, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Schouten J, 
    
    
    
    
    
    

    

    Seedorf U, 
    
    
    
    
    
    

    

    Sen-Chowdhry S, 
    
    
    
    
    
    

    

    Siminovitch K, 
    
    
    
    
    
    

    

    Smit J, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Tan W, 
    
    
    
    
    
    

    

    Teslovich T, 
    
    
    
    
    
    

    

    Tukiainen T, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    Van der Klauw M, 
    
    
    
    
    
    

    

    Vasan R, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wallaschofski H, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Willemsen G, 
    
    
    
    
    
    

    

    Würtz P, 
    
    
    
    
    
    

    

    Xu C, 
    
    
    
    
    
    

    

    Yerges-Armstrong L, 
    
    
    
    
    
    

    

    Alcohol Genome-wide Association (AlcGen) Consortium , 
    
    
    
    
    
    

    

    Diabetes Genetics Replication and Meta-analyses (DIAGRAM+) Study , 
    
    
    
    
    
    

    

    Genetic Investigation of Anthropometric Traits (G. I. A. N. T) Consortium , 
    
    
    
    
    
    

    

    Global Lipids Genetics Consortium and , Genetics of Liver Disease (G. O. L. D) Consortium , 
    
    
    
    
    
    

    

    International Consortium for Blood Pressure (I. C. B. P-G. W. A. S) , 
    
    
    
    
    
    

    

    Meta-analyses of Glucose and Insulin-Related Traits Consortium (MAGIC) , 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Ahmadi K, 
    
    
    
    
    
    

    

    Boomsma D, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Cookson W, 
    
    
    
    
    
    

    

    van Duijn C, 
    
    
    
    
    
    

    

    Froguel P, 
    
    
    
    
    
    

    

    Matsuda K, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Meisinger C, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Pietiläinen K, 
    
    
    
    
    
    

    

    Schumann G, 
    
    
    
    
    
    

    

    Snieder H, 
    
    
    
    
    
    

    

    Sternberg M, 
    
    
    
    
    
    

    

    Stolk R, 
    
    
    
    
    
    

    

    Thomas H, 
    
    
    
    
    
    

    

    Thorsteinsdottir U, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Watkins H, 
    
    
    
    
    
    

    

    Whitfield J, 
    
    
    
    
    
    

    

    Witteman J, 
    
    
    
    
    
    

    

    Wolffenbuttel B, 
    
    
    
    
    
    

    

    Fox C, 
    
    
    
    
    
    

    

    Ala-Korpela M, 
    
    
    
    
    
    

    

    Stefansson K, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    Völzke H, 
    
    
    
    
    
    

    

    Schadt E, 
    
    
    
    
    
    

    

    Scott J, 
    
    
    
    
    
    

    

    Järvelin M, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Kooner J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.970" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Concentrations of liver enzymes in plasma are widely used as indicators of liver disease. We carried out a genome-wide association study in 61,089 individuals, identifying 42 loci associated with concentrations of liver enzymes in plasma, of which 32 are new associations (P = 10(-8) to P = 10(-190)). We used functional genomic approaches including metabonomic profiling and gene expression analyses to identify probable candidate genes at these regions. We identified 69 candidate genes, including genes involved in biliary transport (ATP8B1 and ABCB11), glucose, carbohydrate and lipid metabolism (FADS1, FADS2, GCKR, JMJD1C, HNF1A, MLXIPL, PNPLA3, PPP1R3B, SLC2A2 and TRIB1), glycoprotein biosynthesis and cell surface glycobiology (ABO, ASGR1, FUT2, GPLD1 and ST3GAL4), inflammation and immunity (CD276, CDH6, GCKR, HNF1A, HPR, ITGA1, RORA and STAT4) and glutathione metabolism (GSTT1, GSTT2 and GGT), as well as several genes of uncertain or unknown function (including ABHD12, EFHD1, EFNA1, EPHA2, MICAL3 and ZNF827). Our results provide new insight into genetic mechanisms and pathways influencing markers of liver function.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_saleh">
  <span class="title">Genetic Association Analyses of Atopic Illness and Proinflammatory Cytokine Genes with Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Saleh N, 
    
    
    
    
    
    

    

    Raj S, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Bell L, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetes Metab Res Rev</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1002/dmrr.1259" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The genetic basis of the autoimmune disease type 1 diabetes (T1D) has now been largely determined, so now we can compare these findings with emerging genetic knowledge of disorders and phenotypes that have been negatively or positively associated with T1D historically. Here, we assessed the role in T1D of variants previously reported to be associated with atopic diseases and epithelial barrier function, profilaggrin (), and those that affect the expression levels of the proinflammatory cytokines tumour necrosis factor (TNF)-Î\pm, interleukin (IL)-1Î^2, interferon (IFN)Î^3 and IL-18.We genotyped single nucleotide polymorphisms (SNPs): - 105/rs28665122 in SELS or SEPS1 (selenoprotein), three single nucleotide polymorphisms in IL18 (-105/rs360717, + 183/rs5744292 and + 1467/rs574456) and R501X/rs61816761 in FLG, the major locus associated with atopic dermatitis and predisposing to asthma, in a minimum of 6743 T1D cases and 7864 controls.No evidence of T1D association was found for any of the SNPs we genotyped at FLG, SELS or IL18 (p â\textyen 0.03), nor with haplotypes of IL18 (p = 0.82). Review of previous T1D genome-wide association results revealed that four (human leucocyte antigen (HLA), gasdermin B/ORM1 (Saccharomyces cerevisiae)-like/gasdermin B/, GSDMB/ORMDL3/GSDMA and IL2RB) of ten loci recently reported to be associated with asthma were associated with T1D (pâ\textcopyright\frac120.005).These results show that there are shared genetic associations for atopy-related traits and T1D, and this might help in the future to understand the mechanisms, pathways and environmental factors that underpin the rapid rise in incidence of both disorders in children. Copyright Â\textcopyright 2011 John Wiley & Sons, Ltd.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_swafford">
  <span class="title">An Allele of <i>IKZF1</i> (Ikaros) Conferring Susceptibility to Childhood Acute Lymphoblastic Leukemia Protects against Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Swafford A, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Davison L, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Schuilenburg H, 
    
    
    
    
    
    

    

    Maisuria-Armer M, 
    
    
    
    
    
    

    

    Mistry T, 
    
    
    
    
    
    

    

    Lenardo M, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetes</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.2337/db10-0446" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2046822" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>IKZF1 encoding Ikaros, an essential regulator of lymphopoiesis and immune homeostasis, has been implicated in the development of childhood acute lymphoblastic leukemia (C-ALL). Because recent genome-wide association (GWA) studies have linked a region of the 3’-UTR of IKZF1 with C-ALL susceptibility, we tested whether IKZF1 is associated with the autoimmune disease type 1 diabetes.rs10272724 (T>C) near IKZF1 at 7p12 was genotyped in 8,333 individuals with type 1 diabetes, 9,947 control subjects, and 3,997 families of European ancestry. Association was tested using logistic regression in the case-control data and by the transmission disequilibrium test in the families. Expression data for IKZF1 by rs10272724 genotype were obtained using quantitative PCR of mRNA/cDNA generated from peripheral blood mononuclear cells from 88 individuals, whereas expression data for five other neighboring genes were obtained from the online Genevar dataset.The minor allele of rs10272724 (C) was found to be protective from type 1 diabetes (odds ratio 0.87 [95% CI 0.83-0.91]; P = 1.1 Ã 10(-11)). rs10272724 was not correlated with levels of two transcripts of IKZF1 in peripheral blood mononuclear cells.The major susceptibility genotype for C-ALL confers protection from type 1 diabetes. Our finding strengthens the link between autoimmunity and lymphoid cancers. Further investigation is warranted for the genetic effect marked by rs10272724, its impact on IKZF1, and the role of Ikaros and other family members, Ailios (IKZF3) and Eos (IKZF4), in autoimmunity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Swafford2011-ku">
  <span class="title">An Allele of IKZF1 (Ikaros) Conferring Susceptibility to Childhood Acute Lymphoblastic Leukemia Protects Against Type 1 Diabetes</span>
  <span class="author">
    
    
    
    
    
    

    

    Swafford A, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Davison L, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Schuilenburg H, 
    
    
    
    
    
    

    

    Maisuria-Armer M, 
    
    
    
    
    
    

    

    Mistry T, 
    
    
    
    
    
    

    

    Lenardo M, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetes</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.2337/db10-0446" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3046822" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>OBJECTIVE: IKZF1 encoding Ikaros, an essential regulator of lymphopoiesis and immune homeostasis, has been implicated in the development of childhood acute lymphoblastic leukemia (C-ALL). Because recent genome-wide association (GWA) studies have linked a region of the 3’-UTR of IKZF1 with C-ALL susceptibility, we tested whether IKZF1 is associated with the autoimmune disease type 1 diabetes. RESEARCH DESIGN AND METHODS: rs10272724 (T>C) near IKZF1 at 7p12 was genotyped in 8,333 individuals with type 1 diabetes, 9,947 control subjects, and 3,997 families of European ancestry. Association was tested using logistic regression in the case-control data and by the transmission disequilibrium test in the families. Expression data for IKZF1 by rs10272724 genotype were obtained using quantitative PCR of mRNA/cDNA generated from peripheral blood mononuclear cells from 88 individuals, whereas expression data for five other neighboring genes were obtained from the online Genevar dataset. RESULTS: The minor allele of rs10272724 (C) was found to be protective from type 1 diabetes (odds ratio 0.87 [95% CI 0.83-0.91]; P = 1.1  10(-11)). rs10272724 was not correlated with levels of two transcripts of IKZF1 in peripheral blood mononuclear cells. CONCLUSIONS: The major susceptibility genotype for C-ALL confers protection from type 1 diabetes. Our finding strengthens the link between autoimmunity and lymphoid cancers. Further investigation is warranted for the genetic effect marked by rs10272724, its impact on IKZF1, and the role of Ikaros and other family members, Ailios (IKZF3) and Eos (IKZF4), in autoimmunity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_trynka_vanheel">
  <span class="title">Dense Genotyping Identifies and Localizes Multiple Common and Rare Variant Association Signals in Celiac Disease.</span>
  <span class="author">
    
    
    
    
    
    

    

    Trynka G, 
    
    
    
    
    
    

    

    Hunt K, 
    
    
    
    
    
    

    

    Bockett N, 
    
    
    
    
    
    

    

    Romanos J, 
    
    
    
    
    
    

    

    Mistry V, 
    
    
    
    
    
    

    

    Szperl A, 
    
    
    
    
    
    

    

    Bakker S, 
    
    
    
    
    
    

    

    Bardella M, 
    
    
    
    
    
    

    

    Bhaw-Rosun L, 
    
    
    
    
    
    

    

    Castillejo G, 
    
    
    
    
    
    

    

    de la Concha E, 
    
    
    
    
    
    

    

    de Almeida R, 
    
    
    
    
    
    

    

    Dias K, 
    
    
    
    
    
    

    

    van Diemen C, 
    
    
    
    
    
    

    

    Dubois P, 
    
    
    
    
    
    

    

    Duerr R, 
    
    
    
    
    
    

    

    Edkins S, 
    
    
    
    
    
    

    

    Franke L, 
    
    
    
    
    
    

    

    Fransen K, 
    
    
    
    
    
    

    

    Gutierrez J, 
    
    
    
    
    
    

    

    Heap G, 
    
    
    
    
    
    

    

    Hrdlickova B, 
    
    
    
    
    
    

    

    Hunt S, 
    
    
    
    
    
    

    

    Izurieta L, 
    
    
    
    
    
    

    

    Izzo V, 
    
    
    
    
    
    

    

    Joosten L, 
    
    
    
    
    
    

    

    Langford C, 
    
    
    
    
    
    

    

    Mazzilli M, 
    
    
    
    
    
    

    

    Mein C, 
    
    
    
    
    
    

    

    Midah V, 
    
    
    
    
    
    

    

    Mitrovic M, 
    
    
    
    
    
    

    

    Mora B, 
    
    
    
    
    
    

    

    Morelli M, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    NÃ\textordmasculineÃ\pmez C, 
    
    
    
    
    
    

    

    Onengut-Gumuscu S, 
    
    
    
    
    
    

    

    Pearce K, 
    
    
    
    
    
    

    

    Platteel M, 
    
    
    
    
    
    

    

    Polanco I, 
    
    
    
    
    
    

    

    Potter S, 
    
    
    
    
    
    

    

    Ribes-Koninckx C, 
    
    
    
    
    
    

    

    RicaÃ\pmo-Ponce I, 
    
    
    
    
    
    

    

    Rich S, 
    
    
    
    
    
    

    

    Rybak A, 
    
    
    
    
    
    

    

    Santiago J, 
    
    
    
    
    
    

    

    Senapati S, 
    
    
    
    
    
    

    

    Sood A, 
    
    
    
    
    
    

    

    Szajewska H, 
    
    
    
    
    
    

    

    Troncone R, 
    
    
    
    
    
    

    

    VaradÃ\textcopyright J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wolters V, 
    
    
    
    
    
    

    

    Zhernakova A, 
    
    
    
    
    
    

    

    Spanish Consortium on the Genetics of Coeliac Disease (C. E. G. E. C) , 
    
    
    
    
    
    

    

    Prevent C. D Study Group , 
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium (W. T. C. C. C) , 
    
    
    
    
    
    

    

    Thelma B, 
    
    
    
    
    
    

    

    Cukrowska B, 
    
    
    
    
    
    

    

    Urcelay E, 
    
    
    
    
    
    

    

    Bilbao J, 
    
    
    
    
    
    

    

    Mearin M, 
    
    
    
    
    
    

    

    Barisani D, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Plagnol V, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Wijmenga C, 
    
    
    
    
    
    

    

    van Heel D. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.998" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Using variants from the 1000 Genomes Project pilot European CEU dataset and data from additional resequencing studies, we densely genotyped 183 non-HLA risk loci previously associated with immune-mediated diseases in 12,041 individuals with celiac disease (cases) and 12,228 controls. We identified 13 new celiac disease risk loci reaching genome-wide significance, bringing the number of known loci (including the HLA locus) to 40. We found multiple independent association signals at over one-third of these loci, a finding that is attributable to a combination of common, low-frequency and rare genetic variants. Compared to previously available data such as those from HapMap3, our dense genotyping in a large sample collection provided a higher resolution of the pattern of linkage disequilibrium and suggested localization of many signals to finer scale regions. In particular, 29 of the 54 fine-mapped signals seemed to be localized to single genes and, in some instances, to gene regulatory elements. Altogether, we define the complex genetic architecture of the risk regions of and refine the risk signals for celiac disease, providing the next step toward uncovering the causal mechanisms of the disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_plagnol">
  <span class="title">Genome-Wide Association Analysis of Autoantibody Positivity in Type 1 Diabetes Cases.</span>
  <span class="author">
    
    
    
    
    
    

    

    Plagnol V, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Hafler J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Jackson L, 
    
    
    
    
    
    

    

    Simmonds M, 
    
    
    
    
    
    

    

    1 Diabetes Genetics Consortium T, 
    
    
    
    
    
    

    

    Bingley P, 
    
    
    
    
    
    

    

    Gough S, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1002216" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3150451" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The genetic basis of autoantibody production is largely unknown outside of associations located in the major histocompatibility complex (MHC) human leukocyte antigen (HLA) region. The aim of this study is the discovery of new genetic associations with autoantibody positivity using genome-wide association scan single nucleotide polymorphism (SNP) data in type 1 diabetes (T1D) patients with autoantibody measurements. We measured two anti-islet autoantibodies, glutamate decarboxylase (GADA, n = 2,506), insulinoma-associated antigen 2 (IA-2A, n = 2,498), antibodies to the autoimmune thyroid (Graves’) disease (AITD) autoantigen thyroid peroxidase (TPOA, n = 8,300), and antibodies against gastric parietal cells (PCA, n = 4,328) that are associated with autoimmune gastritis. Two loci passed a stringent genome-wide significance level (p<10(-10)): 1q23/FCRL3 with IA-2A and 9q34/ABO with PCA. Eleven of 52 non-MHC T1D loci showed evidence of association with at least one autoantibody at a false discovery rate of 16%: 16p11/IL27-IA-2A, 2q24/IFIH1-IA-2A and PCA, 2q32/STAT4-TPOA, 10p15/IL2RA-GADA, 6q15/BACH2-TPOA, 21q22/UBASH3A-TPOA, 1p13/PTPN22-TPOA, 2q33/CTLA4-TPOA, 4q27/IL2/TPOA, 15q14/RASGRP1/TPOA, and 12q24/SH2B3-GADA and TPOA. Analysis of the TPOA-associated loci in 2,477 cases with Graves’ disease identified two new AITD loci (BACH2 and UBASH3A).</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2011_cotsapas">
  <span class="title">Pervasive Sharing of Genetic Effects in Autoimmune Disease</span>
  <span class="author">
    
    
    
    
    
    

    

    Cotsapas C, 
    
    
    
    
    
    

    

    Voight B, 
    
    
    
    
    
    

    

    Rossin E, 
    
    
    
    
    
    

    

    Lage K, 
    
    
    
    
    
    

    

    Neale B, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Behrens T, 
    
    
    
    
    
    

    

    Cho J, 
    
    
    
    
    
    

    

    Jager P, 
    
    
    
    
    
    

    

    Elder J, 
    
    
    
    
    
    

    

    Graham R, 
    
    
    
    
    
    

    

    Gregersen P, 
    
    
    
    
    
    

    

    Klareskog L, 
    
    
    
    
    
    

    

    Siminovitch K, 
    
    
    
    
    
    

    

    van Heel D, 
    
    
    
    
    
    

    

    Wijmenga C, 
    
    
    
    
    
    

    

    Worthington J, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Hafler D, 
    
    
    
    
    
    

    

    Rich S, 
    
    
    
    
    
    

    

    Daly M, 
    
    
    
    
    
    

    

    of Consortia F. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet.</em>
    
    
    2011
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1002254" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3154137" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome-wide association (GWA) studies have identified numerous, replicable, genetic associations between common single nucleotide polymorphisms (SNPs) and risk of common autoimmune and inflammatory (immune-mediated) diseases, some of which are shared between two diseases. Along with epidemiological and clinical evidence, this suggests that some genetic risk factors may be shared across diseases-as is the case with alleles in the Major Histocompatibility Locus. In this work we evaluate the extent of this sharing for 107 immune disease-risk SNPs in seven diseases: celiac disease, Crohn’s disease, multiple sclerosis, psoriasis, rheumatoid arthritis, systemic lupus erythematosus, and type 1 diabetes. We have developed a novel statistic for Cross Phenotype Meta-Analysis (CPMA) which detects association of a SNP to multiple, but not necessarily all, phenotypes. With it, we find evidence that 47/107 (44%) immune-mediated disease risk SNPs are associated to multiple-but not all-immune-mediated diseases (SNP-wise P(CPMA)<0.01). We also show that distinct groups of interacting proteins are encoded near SNPs which predispose to the same subsets of diseases; we propose these as the mechanistic basis of shared disease risk. We are thus able to leverage genetic data across diseases to construct biological hypotheses about the underlying mechanism of pathogenesis.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2010</h2>
<ol class="bibliography"><li>

<div id="2010_downes">
  <span class="title">Reduced Expression of <i>IFIH1</i> Is Protective for Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Angus K, 
    
    
    
    
    
    

    

    Hardy M, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS One</em>
    
    
    2010
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pone.0012646" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2936573" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>IFIH1 (interferon induced with helicase C domain 1), also known as MDA5 (melanoma differentiation-associated protein 5), is one of a family of intracellular proteins known to recognise viral RNA and mediate the innate immune response. IFIH1 is causal in type 1 diabetes based on the protective associations of four rare variants, where the derived alleles are predicted to reduce gene expression or function. Originally, however, T1D protection was mapped to the common IFIH1 nsSNP, rs1990760 or Thr946Ala. This common amino acid substitution does not cause a loss of function and evidence suggests the protective allele, Ala(946), may mark a haplotype with reduced expression of IFIH1 in line with the protection conferred by the four rare loss of function alleles. We have performed allele specific expression analysis that supports this hypothesis: the T1D protective haplotype correlates with reduced IFIH1 transcription in interferon-Î^2 stimulated peripheral blood mononuclear cells (overall pâ=â0.012). In addition, we have used multiflow cytometry analysis and quantitative PCR assays to prove reduced expression of IFIH1 in individuals heterozygous for three of the T1D-associated rare alleles: a premature stop codon, rs35744605 (Glu627X) and predicted splice variants, rs35337543 (IVS8+1) and rs35732034 (IVS14+1). We also show that the nsSNP, Ile923V, does not alter pre-mRNA levels of IFIH1. These results confirm and extend the new autoimmune disease pathway of reduced IFIH1 expression and protein function protecting from T1D.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2010_ng_chambers">
  <span class="title">Genetic Loci Influencing Kidney Function and Chronic Kidney Disease.</span>
  <span class="author">
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Zhang W, 
    
    
    
    
    
    

    

    Lord G, 
    
    
    
    
    
    

    

    van der Harst P, 
    
    
    
    
    
    

    

    Lawlor D, 
    
    
    
    
    
    

    

    Sehmi J, 
    
    
    
    
    
    

    

    Gale D, 
    
    
    
    
    
    

    

    Wass M, 
    
    
    
    
    
    

    

    Ahmadi K, 
    
    
    
    
    
    

    

    Bakker S, 
    
    
    
    
    
    

    

    Beckmann J, 
    
    
    
    
    
    

    

    Bilo H, 
    
    
    
    
    
    

    

    Bochud M, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Cook H, 
    
    
    
    
    
    

    

    Cotlarciuc I, 
    
    
    
    
    
    

    

    Davey Smith G, 
    
    
    
    
    
    

    

    de Silva R, 
    
    
    
    
    
    

    

    Deng G, 
    
    
    
    
    
    

    

    Devuyst O, 
    
    
    
    
    
    

    

    Dikkeschei L, 
    
    
    
    
    
    

    

    Dimkovic N, 
    
    
    
    
    
    

    

    Dockrell M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Ebrahim S, 
    
    
    
    
    
    

    

    Eggermann T, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Ferrucci L, 
    
    
    
    
    
    

    

    Floege J, 
    
    
    
    
    
    

    

    Forouhi N, 
    
    
    
    
    
    

    

    Gansevoort R, 
    
    
    
    
    
    

    

    Han X, 
    
    
    
    
    
    

    

    Hedblad B, 
    
    
    
    
    
    

    

    Homan van der Heide J, 
    
    
    
    
    
    

    

    Hepkema B, 
    
    
    
    
    
    

    

    Hernandez-Fuentes M, 
    
    
    
    
    
    

    

    Hypponen E, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    de Jong P, 
    
    
    
    
    
    

    

    Kleefstra N, 
    
    
    
    
    
    

    

    Lagou V, 
    
    
    
    
    
    

    

    Lapsley M, 
    
    
    
    
    
    

    

    Li Y, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Luan J, 
    
    
    
    
    
    

    

    Luttropp K, 
    
    
    
    
    
    

    

    Maréchal C, 
    
    
    
    
    
    

    

    Melander O, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Nordfors L, 
    
    
    
    
    
    

    

    Parsa A, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Penninx B, 
    
    
    
    
    
    

    

    Perucha E, 
    
    
    
    
    
    

    

    Pouta A, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Roderick P, 
    
    
    
    
    
    

    

    Ruokonen A, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Schalling M, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Schlieper G, 
    
    
    
    
    
    

    

    Seelen M, 
    
    
    
    
    
    

    

    Shuldiner A, 
    
    
    
    
    
    

    

    Sjögren M, 
    
    
    
    
    
    

    

    Smit J, 
    
    
    
    
    
    

    

    Snieder H, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Stenvinkel P, 
    
    
    
    
    
    

    

    Sternberg M, 
    
    
    
    
    
    

    

    Swaminathan R, 
    
    
    
    
    
    

    

    Tanaka T, 
    
    
    
    
    
    

    

    Ubink-Veltmaat L, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Zerres K, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Maxwell P, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Jarvelin M, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Lightstone L, 
    
    
    
    
    
    

    

    Scott J, 
    
    
    
    
    
    

    

    Navis G, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Kooner J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2010
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.566" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Using genome-wide association, we identify common variants at 2p12-p13, 6q26, 17q23 and 19q13 associated with serum creatinine, a marker of kidney function (P = 10(-10) to 10(-15)). Of these, rs10206899 (near NAT8, 2p12-p13) and rs4805834 (near SLC7A9, 19q13) were also associated with chronic kidney disease (P = 5.0 x 10(-5) and P = 3.6 x 10(-4), respectively). Our findings provide insight into metabolic, solute and drug-transport pathways underlying susceptibility to chronic kidney disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="Downes2010-qi">
  <span class="title">Reduced Expression of IFIH1 Is Protective for Type 1 Diabetes</span>
  <span class="author">
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Pekalski M, 
    
    
    
    
    
    

    

    Angus K, 
    
    
    
    
    
    

    

    Hardy M, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Todd J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS One</em>
    
    
    2010
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pone.0012646" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2936573" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>IFIH1 (interferon induced with helicase C domain 1), also known as MDA5 (melanoma differentiation-associated protein 5), is one of a family of intracellular proteins known to recognise viral RNA and mediate the innate immune response. IFIH1 is causal in type 1 diabetes based on the protective associations of four rare variants, where the derived alleles are predicted to reduce gene expression or function. Originally, however, T1D protection was mapped to the common IFIH1 nsSNP, rs1990760 or Thr946Ala. This common amino acid substitution does not cause a loss of function and evidence suggests the protective allele, Ala(946), may mark a haplotype with reduced expression of IFIH1 in line with the protection conferred by the four rare loss of function alleles. We have performed allele specific expression analysis that supports this hypothesis: the T1D protective haplotype correlates with reduced IFIH1 transcription in interferon-β stimulated peripheral blood mononuclear cells (overall p = 0.012). In addition, we have used multiflow cytometry analysis and quantitative PCR assays to prove reduced expression of IFIH1 in individuals heterozygous for three of the T1D-associated rare alleles: a premature stop codon, rs35744605 (Glu627X) and predicted splice variants, rs35337543 (IVS8+1) and rs35732034 (IVS14+1). We also show that the nsSNP, Ile923V, does not alter pre-mRNA levels of IFIH1. These results confirm and extend the new autoimmune disease pathway of reduced IFIH1 expression and protein function protecting from T1D.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="wtccc_cnv10">
  <span class="title">Genome-Wide Association Study of CNVs in 16,000 Cases of Eight Common Diseases and 3,000 Shared Controls.</span>
  <span class="author">
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Craddock N, 
    
    
    
    
    
    

    

    Hurles M, 
    
    
    
    
    
    

    

    Cardin N, 
    
    
    
    
    
    

    

    Pearson R, 
    
    
    
    
    
    

    

    Plagnol V, 
    
    
    
    
    
    

    

    Robson S, 
    
    
    
    
    
    

    

    Vukcevic D, 
    
    
    
    
    
    

    

    Barnes C, 
    
    
    
    
    
    

    

    Conrad D, 
    
    
    
    
    
    

    

    Giannoulatou E, 
    
    
    
    
    
    

    

    Holmes C, 
    
    
    
    
    
    

    

    Marchini J, 
    
    
    
    
    
    

    

    Stirrups K, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Wain L, 
    
    
    
    
    
    

    

    Yau C, 
    
    
    
    
    
    

    

    Aerts J, 
    
    
    
    
    
    

    

    Ahmad T, 
    
    
    
    
    
    

    

    Andrews T, 
    
    
    
    
    
    

    

    Arbury H, 
    
    
    
    
    
    

    

    Attwood A, 
    
    
    
    
    
    

    

    Auton A, 
    
    
    
    
    
    

    

    Ball S, 
    
    
    
    
    
    

    

    Balmforth A, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    Barton A, 
    
    
    
    
    
    

    

    Bennett A, 
    
    
    
    
    
    

    

    Bhaskar S, 
    
    
    
    
    
    

    

    Blaszczyk K, 
    
    
    
    
    
    

    

    Bowes J, 
    
    
    
    
    
    

    

    Brand O, 
    
    
    
    
    
    

    

    Braund P, 
    
    
    
    
    
    

    

    Bredin F, 
    
    
    
    
    
    

    

    Breen G, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Bruce I, 
    
    
    
    
    
    

    

    Bull J, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Burton J, 
    
    
    
    
    
    

    

    Byrnes J, 
    
    
    
    
    
    

    

    Caesar S, 
    
    
    
    
    
    

    

    Clee C, 
    
    
    
    
    
    

    

    Coffey A, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Drummond H, 
    
    
    
    
    
    

    

    Dudakia D, 
    
    
    
    
    
    

    

    Dunham A, 
    
    
    
    
    
    

    

    Ebbs B, 
    
    
    
    
    
    

    

    Eccles D, 
    
    
    
    
    
    

    

    Edkins S, 
    
    
    
    
    
    

    

    Edwards C, 
    
    
    
    
    
    

    

    Elliot A, 
    
    
    
    
    
    

    

    Emery P, 
    
    
    
    
    
    

    

    Evans D, 
    
    
    
    
    
    

    

    Evans G, 
    
    
    
    
    
    

    

    Eyre S, 
    
    
    
    
    
    

    

    Farmer A, 
    
    
    
    
    
    

    

    Ferrier I, 
    
    
    
    
    
    

    

    Feuk L, 
    
    
    
    
    
    

    

    Fitzgerald T, 
    
    
    
    
    
    

    

    Flynn E, 
    
    
    
    
    
    

    

    Forbes A, 
    
    
    
    
    
    

    

    Forty L, 
    
    
    
    
    
    

    

    Franklyn J, 
    
    
    
    
    
    

    

    Freathy R, 
    
    
    
    
    
    

    

    Gibbs P, 
    
    
    
    
    
    

    

    Gilbert P, 
    
    
    
    
    
    

    

    Gokumen O, 
    
    
    
    
    
    

    

    Gordon-Smith K, 
    
    
    
    
    
    

    

    Gray E, 
    
    
    
    
    
    

    

    Green E, 
    
    
    
    
    
    

    

    Groves C, 
    
    
    
    
    
    

    

    Grozeva D, 
    
    
    
    
    
    

    

    Gwilliam R, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Hammond N, 
    
    
    
    
    
    

    

    Hardy M, 
    
    
    
    
    
    

    

    Harrison P, 
    
    
    
    
    
    

    

    Hassanali N, 
    
    
    
    
    
    

    

    Hebaishi H, 
    
    
    
    
    
    

    

    Hines S, 
    
    
    
    
    
    

    

    Hinks A, 
    
    
    
    
    
    

    

    Hitman G, 
    
    
    
    
    
    

    

    Hocking L, 
    
    
    
    
    
    

    

    Howard E, 
    
    
    
    
    
    

    

    Howard P, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Hughes D, 
    
    
    
    
    
    

    

    Hunt S, 
    
    
    
    
    
    

    

    Isaacs J, 
    
    
    
    
    
    

    

    Jain M, 
    
    
    
    
    
    

    

    Jewell D, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Jolley J, 
    
    
    
    
    
    

    

    Jones I, 
    
    
    
    
    
    

    

    Jones L, 
    
    
    
    
    
    

    

    Kirov G, 
    
    
    
    
    
    

    

    Langford C, 
    
    
    
    
    
    

    

    Lango-Allen H, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Lee J, 
    
    
    
    
    
    

    

    Lee K, 
    
    
    
    
    
    

    

    Lees C, 
    
    
    
    
    
    

    

    Lewis K, 
    
    
    
    
    
    

    

    Lindgren C, 
    
    
    
    
    
    

    

    Maisuria-Armer M, 
    
    
    
    
    
    

    

    Maller J, 
    
    
    
    
    
    

    

    Mansfield J, 
    
    
    
    
    
    

    

    Martin P, 
    
    
    
    
    
    

    

    Massey D, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    McGuffin P, 
    
    
    
    
    
    

    

    McLay K, 
    
    
    
    
    
    

    

    Mentzer A, 
    
    
    
    
    
    

    

    Mimmack M, 
    
    
    
    
    
    

    

    Morgan A, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Mowat C, 
    
    
    
    
    
    

    

    Myers S, 
    
    
    
    
    
    

    

    Newman W, 
    
    
    
    
    
    

    

    Nimmo E, 
    
    
    
    
    
    

    

    O’Donovan M, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    Onyiah I, 
    
    
    
    
    
    

    

    Ovington N, 
    
    
    
    
    
    

    

    Owen M, 
    
    
    
    
    
    

    

    Palin K, 
    
    
    
    
    
    

    

    Parnell K, 
    
    
    
    
    
    

    

    Pernet D, 
    
    
    
    
    
    

    

    Perry J, 
    
    
    
    
    
    

    

    Phillips A, 
    
    
    
    
    
    

    

    Pinto D, 
    
    
    
    
    
    

    

    Prescott N, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Quail M, 
    
    
    
    
    
    

    

    Rafelt S, 
    
    
    
    
    
    

    

    Rayner N, 
    
    
    
    
    
    

    

    Redon R, 
    
    
    
    
    
    

    

    Reid D, 
    
    
    
    
    
    

    

    Renwick , 
    
    
    
    
    
    

    

    Ring S, 
    
    
    
    
    
    

    

    Robertson N, 
    
    
    
    
    
    

    

    Russell E, 
    
    
    
    
    
    

    

    Clair D, 
    
    
    
    
    
    

    

    Sambrook J, 
    
    
    
    
    
    

    

    Sanderson J, 
    
    
    
    
    
    

    

    Schuilenburg H, 
    
    
    
    
    
    

    

    Scott C, 
    
    
    
    
    
    

    

    Scott R, 
    
    
    
    
    
    

    

    Seal S, 
    
    
    
    
    
    

    

    Shaw-Hawkins S, 
    
    
    
    
    
    

    

    Shields B, 
    
    
    
    
    
    

    

    Simmonds M, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Somaskantharajah E, 
    
    
    
    
    
    

    

    Spanova K, 
    
    
    
    
    
    

    

    Steer S, 
    
    
    
    
    
    

    

    Stephens J, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Stone M, 
    
    
    
    
    
    

    

    Su Z, 
    
    
    
    
    
    

    

    Symmons D, 
    
    
    
    
    
    

    

    Thompson J, 
    
    
    
    
    
    

    

    Thomson W, 
    
    
    
    
    
    

    

    Travers M, 
    
    
    
    
    
    

    

    Turnbull C, 
    
    
    
    
    
    

    

    Valsesia A, 
    
    
    
    
    
    

    

    Walker M, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Warren-Perry M, 
    
    
    
    
    
    

    

    Watkins N, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Weedon M, 
    
    
    
    
    
    

    

    Wilson A, 
    
    
    
    
    
    

    

    Woodburn M, 
    
    
    
    
    
    

    

    Wordsworth B, 
    
    
    
    
    
    

    

    Young A, 
    
    
    
    
    
    

    

    Zeggini E, 
    
    
    
    
    
    

    

    Carter N, 
    
    
    
    
    
    

    

    Frayling T, 
    
    
    
    
    
    

    

    Lee C, 
    
    
    
    
    
    

    

    McVean G, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Palotie A, 
    
    
    
    
    
    

    

    Sawcer S, 
    
    
    
    
    
    

    

    Scherer S, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Tyler-Smith C, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Compston A, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Gough S, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hill A, 
    
    
    
    
    
    

    

    Mathew C, 
    
    
    
    
    
    

    

    Pembrey M, 
    
    
    
    
    
    

    

    Satsangi J, 
    
    
    
    
    
    

    

    Stratton M, 
    
    
    
    
    
    

    

    Worthington J, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Duncanson A, 
    
    
    
    
    
    

    

    Kwiatkowski D, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Parkes M, 
    
    
    
    
    
    

    

    Rahman N, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Donnelly P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nature</em>
    
    
    2010
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/nature08979" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2892339" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Copy number variants (CNVs) account for a major proportion of human genetic polymorphism and have been predicted to have an important role in genetic susceptibility to common disease. To address this we undertook a large, direct genome-wide study of association between CNVs and eight common human diseases. Using a purpose-designed array we typed approximately 19,000 individuals into distinct copy-number classes at 3,432 polymorphic CNVs, including an estimated approximately 50% of all common CNVs larger than 500 base pairs. We identified several biological artefacts that lead to false-positive associations, including systematic CNV differences between DNAs derived from blood and cell lines. Association testing and follow-up re plication analyses confirmed three loci where CNVs were associated with disease-IRGM for Crohn’s disease, HLA for Crohn’s disease, rheumatoid arthritis and type 1 diabetes, and TSPAN8 for type 2 diabetes-although in each case the locus had previously been identified in single nucleotide polymorphism (SNP)-based studies, reflecting our observation that most common CNVs that are well-typed on our array are well tagged by SNPs and so have been indirectly explored through SNP studies. We conclude that common CNVs that can be typed on existing platforms are unlikely to contribute greatly to the genetic basis of common human diseases.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2010_ng_dlk1">
  <span class="title">The Imprinted <i>DLK1</i>-<i>MEG3</i> Gene Region on Chromosome 14q32.2 Alters Susceptibility to Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Maisuria-Armer M, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Clayton D. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2010
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.493" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2820243" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Genome-wide association (GWA) studies to map common disease susceptibility loci have been hugely successful, with over 300 reproducibly associated loci reported to date. However, these studies have not yet provided convincing evidence for any susceptibility locus subject to parent-of-origin effects. Using imputation to extend existing GWA datasets, we have obtained robust evidence at rs941576 for paternally inherited risk of type 1 diabetes (T1D; ratio of allelic effects for paternal versus maternal transmissions = 0.75; 95% confidence interval (CI) = 0.71-0.79). This marker is in the imprinted region of chromosome 14q32.2, which contains the functional candidate gene DLK1. Our meta-analysis also provided support at genome-wide significance for a T1D locus at chromosome 19p13.2. The highest association was at marker rs2304256 (odds ratio (OR) = 0.86; 95%CI = 0.82-0.90) in the TYK2 gene, which has previously been associated with systemic lupus erythematosus and multiple sclerosis.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2010_nat_heinig">
  <span class="title">A Trans-Acting Locus Regulates an Anti-Viral Expression Network and Type 1 Diabetes Risk.</span>
  <span class="author">
    
    
    
    
    
    

    

    Heinig M , 
    
    
    
    
    
    

    

    Petretto E, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Bottolo L, 
    
    
    
    
    
    

    

    Rotival M, 
    
    
    
    
    
    

    

    Lu H, 
    
    
    
    
    
    

    

    Li Y, 
    
    
    
    
    
    

    

    Sarwar R, 
    
    
    
    
    
    

    

    Langley S, 
    
    
    
    
    
    

    

    Bauerfeind A, 
    
    
    
    
    
    

    

    Hummel O, 
    
    
    
    
    
    

    

    Lee Y, 
    
    
    
    
    
    

    

    Paskas S, 
    
    
    
    
    
    

    

    Rintisch C, 
    
    
    
    
    
    

    

    Saar K, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Buchan R, 
    
    
    
    
    
    

    

    Gray E, 
    
    
    
    
    
    

    

    Cyster J, 
    
    
    
    
    
    

    

    Consortium C, 
    
    
    
    
    
    

    

    Erdmann J, 
    
    
    
    
    
    

    

    Hengstenberg C, 
    
    
    
    
    
    

    

    Maouche S, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Rice C, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Schunkert H, 
    
    
    
    
    
    

    

    Goodall A, 
    
    
    
    
    
    

    

    Schulz H, 
    
    
    
    
    
    

    

    Roider H, 
    
    
    
    
    
    

    

    Vingron M, 
    
    
    
    
    
    

    

    Blankenberg S, 
    
    
    
    
    
    

    

    Maenzel T, 
    
    
    
    
    
    

    

    Zeller T, 
    
    
    
    
    
    

    

    Szymczak S, 
    
    
    
    
    
    

    

    Ziegler A, 
    
    
    
    
    
    

    

    Tiret L, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Pravenec M, 
    
    
    
    
    
    

    

    Aitman T, 
    
    
    
    
    
    

    

    Cambien F, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Hubner N, 
    
    
    
    
    
    

    

    Cook S. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nature</em>
    
    
    2010
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/nature09386" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC3657719" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Combined analyses of gene networks and DNA sequence variation can provide new insights into the aetiology of common diseases that may not be apparent from genome-wide association studies alone. Recent advances in rat genomics are facilitating systems-genetics approaches. Here we report the use of integrated genome-wide approaches across seven rat tissues to identify gene networks and the loci underlying their regulation. We defined an interferon regulatory factor 7 (IRF7)-driven inflammatory network (IDIN) enriched for viral response genes, which represents a molecular biomarker for macrophages and which was regulated in multiple tissues by a locus on rat chromosome 15q25. We show that Epstein-Barr virus induced gene 2 (Ebi2, also known as Gpr183), which lies at this locus and controls B lymphocyte migration, is expressed in macrophages and regulates the IDIN. The human orthologous locus on chromosome 13q32 controlled the human equivalent of the IDIN, which was conserved in monocytes. IDIN genes were more likely to associate with susceptibility to type 1 diabetes (T1D)-a macrophage-associated autoimmune disease-than randomly selected immune response genes (P = 8.85âÃâ10(-6)). The human locus controlling the IDIN was associated with the risk of T1D at single nucleotide polymorphism rs9585056 (P = 7.0âÃâ10(-10); odds ratio, 1.15), which was one of five single nucleotide polymorphisms in this region associated with EBI2 (GPR183) expression. These data implicate IRF7 network genes and their regulatory locus in the pathogenesis of T1D.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2009</h2>
<ol class="bibliography"><li>

<div id="2009_ng_willer_bmi">
  <span class="title">Six New Loci Associated with Body Mass Index Highlight a Neuronal Influence on Body Weight Regulation.</span>
  <span class="author">
    
    
    
    
    
    

    

    Willer C, 
    
    
    
    
    
    

    

    Speliotes E, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Li S, 
    
    
    
    
    
    

    

    Lindgren C, 
    
    
    
    
    
    

    

    Heid I, 
    
    
    
    
    
    

    

    Berndt S, 
    
    
    
    
    
    

    

    Elliott A, 
    
    
    
    
    
    

    

    Jackson A, 
    
    
    
    
    
    

    

    Lamina C, 
    
    
    
    
    
    

    

    Lettre G, 
    
    
    
    
    
    

    

    Lim N, 
    
    
    
    
    
    

    

    Lyon H, 
    
    
    
    
    
    

    

    McCarroll S, 
    
    
    
    
    
    

    

    Papadakis K, 
    
    
    
    
    
    

    

    Qi L, 
    
    
    
    
    
    

    

    Randall J, 
    
    
    
    
    
    

    

    Roccasecca R, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Scheet P, 
    
    
    
    
    
    

    

    Weedon M, 
    
    
    
    
    
    

    

    Wheeler E, 
    
    
    
    
    
    

    

    Zhao J, 
    
    
    
    
    
    

    

    Jacobs L, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Tanaka T, 
    
    
    
    
    
    

    

    Timpson N, 
    
    
    
    
    
    

    

    Almgren P, 
    
    
    
    
    
    

    

    Bennett A, 
    
    
    
    
    
    

    

    Bergman R, 
    
    
    
    
    
    

    

    Bingham S, 
    
    
    
    
    
    

    

    Bonnycastle L, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Burtt N, 
    
    
    
    
    
    

    

    Chines P, 
    
    
    
    
    
    

    

    Coin L, 
    
    
    
    
    
    

    

    Collins F, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Cooper C, 
    
    
    
    
    
    

    

    Smith G, 
    
    
    
    
    
    

    

    Dennison E, 
    
    
    
    
    
    

    

    Deodhar P, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Erdos M, 
    
    
    
    
    
    

    

    Estrada K, 
    
    
    
    
    
    

    

    Evans D, 
    
    
    
    
    
    

    

    Gianniny L, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    Gillson C, 
    
    
    
    
    
    

    

    Guiducci C, 
    
    
    
    
    
    

    

    Hackett R, 
    
    
    
    
    
    

    

    Hadley D, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Havulinna A, 
    
    
    
    
    
    

    

    Hebebrand J, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Isomaa B, 
    
    
    
    
    
    

    

    Jacobs K, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Jousilahti P, 
    
    
    
    
    
    

    

    Jovanovic Z, 
    
    
    
    
    
    

    

    Khaw K, 
    
    
    
    
    
    

    

    Kraft P, 
    
    
    
    
    
    

    

    Kuokkanen M, 
    
    
    
    
    
    

    

    Kuusisto J, 
    
    
    
    
    
    

    

    Laitinen J, 
    
    
    
    
    
    

    

    Lakatta E, 
    
    
    
    
    
    

    

    Luan J, 
    
    
    
    
    
    

    

    Luben R, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Mulas A, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Narisu N, 
    
    
    
    
    
    

    

    Ness A, 
    
    
    
    
    
    

    

    Northstone K, 
    
    
    
    
    
    

    

    O’Rahilly S, 
    
    
    
    
    
    

    

    Purmann C, 
    
    
    
    
    
    

    

    Rees M, 
    
    
    
    
    
    

    

    Ridderstrale M, 
    
    
    
    
    
    

    

    Ring S, 
    
    
    
    
    
    

    

    Rivadeneira F, 
    
    
    
    
    
    

    

    Ruokonen A, 
    
    
    
    
    
    

    

    Sandhu M, 
    
    
    
    
    
    

    

    Saramies J, 
    
    
    
    
    
    

    

    Scott L, 
    
    
    
    
    
    

    

    Scuteri A, 
    
    
    
    
    
    

    

    Silander K, 
    
    
    
    
    
    

    

    Sims M, 
    
    
    
    
    
    

    

    Song K, 
    
    
    
    
    
    

    

    Stephens J, 
    
    
    
    
    
    

    

    Stevens S, 
    
    
    
    
    
    

    

    Stringham H, 
    
    
    
    
    
    

    

    Tung Y, 
    
    
    
    
    
    

    

    Valle T, 
    
    
    
    
    
    

    

    Duijn C, 
    
    
    
    
    
    

    

    Vimaleswaran K, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Watanabe R, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Watkins N, 
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Witteman J, 
    
    
    
    
    
    

    

    Zeggini E, 
    
    
    
    
    
    

    

    Zhai G, 
    
    
    
    
    
    

    

    Zillikens M, 
    
    
    
    
    
    

    

    Altshuler D, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Chanock S, 
    
    
    
    
    
    

    

    Farooqi I, 
    
    
    
    
    
    

    

    Ferrucci L, 
    
    
    
    
    
    

    

    Guralnik J, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hu F, 
    
    
    
    
    
    

    

    Jarvelin M, 
    
    
    
    
    
    

    

    Laakso M, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Ong K, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Salomaa V, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Tuomi T, 
    
    
    
    
    
    

    

    Tuomilehto J, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Frayling T, 
    
    
    
    
    
    

    

    Groop L, 
    
    
    
    
    
    

    

    Hayes R, 
    
    
    
    
    
    

    

    Hunter D, 
    
    
    
    
    
    

    

    Mohlke K, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Boehnke M, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Hirschhorn J, 
    
    
    
    
    
    

    

    Genetic Investigation of Anthropometric Traits Consortium . 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2009
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Common variants at only two loci, FTO and MC4R, have been reproducibly associated with body mass index (BMI) in humans. To identify additional loci, we conducted meta-analysis of 15 genome-wide association studies for BMI (n > 32,000) and followed up top signals in 14 additional cohorts (n > 59,000). We strongly confirm FTO and MC4R and identify six additional loci (P < 5 x 10(-8)): TMEM18, KCTD15, GNPDA2, SH2B1, MTCH2 and NEGR1 (where a 45-kb deletion polymorphism is a candidate causal variant). Several of the likely causal genes are highly expressed or known to act in the central nervous system (CNS), emphasizing, as in rare monogenic forms of obesity, the role of the CNS in predisposition to obesity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2009_pg_kolz">
  <span class="title">Meta-Analysis of 28,141 Individuals Identifies Common Variants within Five New Loci That Influence Uric Acid Concentrations</span>
  <span class="author">
    
    
    
    
    
    

    

    Kolz M, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Teumer A, 
    
    
    
    
    
    

    

    Vitart V, 
    
    
    
    
    
    

    

    Perola M, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    Albrecht E, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Johansson A, 
    
    
    
    
    
    

    

    Nyholt D, 
    
    
    
    
    
    

    

    Aulchenko Y, 
    
    
    
    
    
    

    

    Beckmann J, 
    
    
    
    
    
    

    

    Bergmann S, 
    
    
    
    
    
    

    

    Bochud M, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Campbell H, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Homuth G, 
    
    
    
    
    
    

    

    Lamina C, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Nauck M, 
    
    
    
    
    
    

    

    Peden J, 
    
    
    
    
    
    

    

    Prokisch H, 
    
    
    
    
    
    

    

    Salo P, 
    
    
    
    
    
    

    

    Salomaa V, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Valker U, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Wang-Sattler R, 
    
    
    
    
    
    

    

    Wright A, 
    
    
    
    
    
    

    

    Adamski J, 
    
    
    
    
    
    

    

    Whitfield J, 
    
    
    
    
    
    

    

    Gyllensten U, 
    
    
    
    
    
    

    

    Wilson J, 
    
    
    
    
    
    

    

    Rudan I, 
    
    
    
    
    
    

    

    Pramstaller P, 
    
    
    
    
    
    

    

    Watkins H, 
    
    
    
    
    
    

    

    Doering A, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Valzke H, 
    
    
    
    
    
    

    

    Nagaraja R, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Illig T, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    for the EUROSPAN Consortium , 
    
    
    
    
    
    

    

    for the ENGAGE Consortium , 
    
    
    
    
    
    

    

    for the PROCARDIS Consortium , 
    
    
    
    
    
    

    

    for the KORA Study , 
    
    
    
    
    
    

    

    for the WTCCC . 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet</em>
    
    
    2009
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1000504" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p><title>Author Summary</title> <p>Elevated serum uric acid levels cause gout and are a risk factor for cardiovascular disease and diabetes. The regulation of serum uric acid levels is under a strong genetic control. This study describes the first meta-analysis of genome-wide association scans from 14 studies totalling 28,141 participants of European descent. We show that common DNA variants at nine different loci are associated with uric acid concentrations, five of which are novel. These variants are located within the genes coding for organic anion transporter 4 (<italic>SLC22A11</italic>), monocarboxylic acid transporter 9 (<italic>SLC16A9</italic>), glucokinase regulatory protein (<italic>GCKR</italic>), Carmil (<italic>LRRC16A</italic>), and near PDZ domain-containing 1 (<italic>PDZK1</italic>). Gender-specific effects are shown for variants within the recently identified genes coding for glucose transporter 9 (<italic>SLC2A9</italic>) and the ATP-binding cassette transporter (<italic>ABCG2</italic>). Based on screening of 163 metabolites, we show an association of one of the identified variants within <italic>SLC16A9</italic> with DL-carnitine and propionyl-L-carnitine. Moreover, DL-carnitine and propionyl-L-carnitine were strongly correlated with serum UA levels, forming a triangle between SNP, metabolites and UA levels. Taken together, these associations highlight pathways that are important in the regulation of serum uric acid levels and point toward novel potential targets for pharmacological intervention to prevent or treat hyperuricemia.</p></p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2009_pg_lindgren">
  <span class="title">Genome-Wide Association Scan Meta-Analysis Identifies Three Loci Influencing Adiposity and Fat Distribution.</span>
  <span class="author">
    
    
    
    
    
    

    

    Lindgren C, 
    
    
    
    
    
    

    

    Heid I, 
    
    
    
    
    
    

    

    Randall J, 
    
    
    
    
    
    

    

    Lamina C, 
    
    
    
    
    
    

    

    Steinthorsdottir V, 
    
    
    
    
    
    

    

    Qi L, 
    
    
    
    
    
    

    

    Speliotes E, 
    
    
    
    
    
    

    

    Thorleifsson G, 
    
    
    
    
    
    

    

    Willer C, 
    
    
    
    
    
    

    

    Herrera B, 
    
    
    
    
    
    

    

    Jackson A, 
    
    
    
    
    
    

    

    Lim N, 
    
    
    
    
    
    

    

    Scheet P, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Amin N, 
    
    
    
    
    
    

    

    Aulchenko Y, 
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Drong A, 
    
    
    
    
    
    

    

    Luan J, 
    
    
    
    
    
    

    

    Lyon H, 
    
    
    
    
    
    

    

    Rivadeneira F, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Timpson N, 
    
    
    
    
    
    

    

    Zillikens M, 
    
    
    
    
    
    

    

    Zhao J, 
    
    
    
    
    
    

    

    Almgren P, 
    
    
    
    
    
    

    

    Bandinelli S, 
    
    
    
    
    
    

    

    Bennett A, 
    
    
    
    
    
    

    

    Bergman R, 
    
    
    
    
    
    

    

    Bonnycastle L, 
    
    
    
    
    
    

    

    Bumpstead S, 
    
    
    
    
    
    

    

    Chanock S, 
    
    
    
    
    
    

    

    Cherkas L, 
    
    
    
    
    
    

    

    Chines P, 
    
    
    
    
    
    

    

    Coin L, 
    
    
    
    
    
    

    

    Cooper C, 
    
    
    
    
    
    

    

    Crawford G, 
    
    
    
    
    
    

    

    Doering A, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Doney A, 
    
    
    
    
    
    

    

    Ebrahim S, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Erdos M, 
    
    
    
    
    
    

    

    Estrada K, 
    
    
    
    
    
    

    

    Ferrucci L, 
    
    
    
    
    
    

    

    Fischer G, 
    
    
    
    
    
    

    

    Forouhi N, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    Grallert H, 
    
    
    
    
    
    

    

    Groves C, 
    
    
    
    
    
    

    

    Grundy S, 
    
    
    
    
    
    

    

    Guiducci C, 
    
    
    
    
    
    

    

    Hadley D, 
    
    
    
    
    
    

    

    Hamsten A, 
    
    
    
    
    
    

    

    Havulinna A, 
    
    
    
    
    
    

    

    Hofman A, 
    
    
    
    
    
    

    

    Holle R, 
    
    
    
    
    
    

    

    Holloway J, 
    
    
    
    
    
    

    

    Illig T, 
    
    
    
    
    
    

    

    Isomaa B, 
    
    
    
    
    
    

    

    Jacobs L, 
    
    
    
    
    
    

    

    Jameson K, 
    
    
    
    
    
    

    

    Jousilahti P, 
    
    
    
    
    
    

    

    Karpe F, 
    
    
    
    
    
    

    

    Kuusisto J, 
    
    
    
    
    
    

    

    Laitinen J, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Lawlor D, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Morken M, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Narisu N, 
    
    
    
    
    
    

    

    NordstrÃm A, 
    
    
    
    
    
    

    

    NordstrÃm P, 
    
    
    
    
    
    

    

    Oostra B, 
    
    
    
    
    
    

    

    Palmer C, 
    
    
    
    
    
    

    

    Payne F, 
    
    
    
    
    
    

    

    Peden J, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    RenstrÃm F, 
    
    
    
    
    
    

    

    Ruokonen A, 
    
    
    
    
    
    

    

    Salomaa V, 
    
    
    
    
    
    

    

    Sandhu M, 
    
    
    
    
    
    

    

    Scott L, 
    
    
    
    
    
    

    

    Scuteri A, 
    
    
    
    
    
    

    

    Silander K, 
    
    
    
    
    
    

    

    Song K, 
    
    
    
    
    
    

    

    Yuan X, 
    
    
    
    
    
    

    

    Stringham H, 
    
    
    
    
    
    

    

    Swift A, 
    
    
    
    
    
    

    

    Tuomi T, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Walters G, 
    
    
    
    
    
    

    

    Weedon M, 
    
    
    
    
    
    

    

    WTCCC , 
    
    
    
    
    
    

    

    Witteman J, 
    
    
    
    
    
    

    

    Zhang C, 
    
    
    
    
    
    

    

    Zhang W, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Collins F, 
    
    
    
    
    
    

    

    Smith G, 
    
    
    
    
    
    

    

    Day I, 
    
    
    
    
    
    

    

    Franks P, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hu F, 
    
    
    
    
    
    

    

    Jarvelin M, 
    
    
    
    
    
    

    

    Kong A, 
    
    
    
    
    
    

    

    Kooner J, 
    
    
    
    
    
    

    

    Laakso M, 
    
    
    
    
    
    

    

    Lakatta E, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Tanaka T, 
    
    
    
    
    
    

    

    Tuomilehto J, 
    
    
    
    
    
    

    

    Uitterlinden A, 
    
    
    
    
    
    

    

    van Duijn C, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Watkins H, 
    
    
    
    
    
    

    

    Procardis Consortia , 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Boehnke M, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Groop L, 
    
    
    
    
    
    

    

    Hunter D, 
    
    
    
    
    
    

    

    Thorsteinsdottir U, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Frayling T, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Hirschhorn J, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Stefansson K, 
    
    
    
    
    
    

    

    Mohlke K, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Giant Consortium . 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Genet</em>
    
    
    2009
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pgen.1000508" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>To identify genetic loci influencing central obesity and fat distribution, we performed a meta-analysis of 16 genome-wide association studies (GWAS, N = 38,580) informative for adult waist circumference (WC) and waist-hip ratio (WHR). We selected 26 SNPs for follow-up, for which the evidence of association with measures of central adiposity (WC and/or WHR) was strong and disproportionate to that for overall adiposity or height. Follow-up studies in a maximum of 70,689 individuals identified two loci strongly associated with measures of central adiposity; these map near TFAP2B (WC, P = 1.9x10(-11)) and MSRA (WC, P = 8.9x10(-9)). A third locus, near LYPLAL1, was associated with WHR in women only (P = 2.6x10(-8)). The variants near TFAP2B appear to influence central adiposity through an effect on overall obesity/fat-mass, whereas LYPLAL1 displays a strong female-only association with fat distribution. By focusing on anthropometric measures of central obesity and fat distribution, we have identified three loci implicated in the regulation of human adiposity.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2009_po_newhouse">
  <span class="title">Polymorphisms in the WNK1 Gene Are Associated with Blood Pressure Variation and Urinary Potassium Excretion.</span>
  <span class="author">
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Hoti M, 
    
    
    
    
    
    

    

    Burke B, 
    
    
    
    
    
    

    

    Howard P, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    Lee K, 
    
    
    
    
    
    

    

    Shaw-Hawkins S, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Kooner J, 
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Clarke R, 
    
    
    
    
    
    

    

    Collins R, 
    
    
    
    
    
    

    

    Laan M, 
    
    
    
    
    
    

    

    Org E, 
    
    
    
    
    
    

    

    Juhanson P, 
    
    
    
    
    
    

    

    Veldre G, 
    
    
    
    
    
    

    

    Viigimaa M, 
    
    
    
    
    
    

    

    Eyheramendy S, 
    
    
    
    
    
    

    

    Cappuccio F, 
    
    
    
    
    
    

    

    Ji C, 
    
    
    
    
    
    

    

    Iacone R, 
    
    
    
    
    
    

    

    Strazzullo P, 
    
    
    
    
    
    

    

    Kumari M, 
    
    
    
    
    
    

    

    Marmot M, 
    
    
    
    
    
    

    

    Brunner E, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS ONE</em>
    
    
    2009
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pone.0005003" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>WNK1–a serine/threonine kinase involved in electrolyte homeostasis and blood pressure (BP) control–is an excellent candidate gene for essential hypertension (EH). We and others have previously reported association between WNK1 and BP variation. Using tag SNPs (tSNPs) that capture 100% of common WNK1 variation in HapMap, we aimed to replicate our findings with BP and to test for association with phenotypes relating to WNK1 function in the British Genetics of Hypertension (BRIGHT) study case-control resource (1700 hypertensive cases and 1700 normotensive controls). We found multiple variants to be associated with systolic blood pressure, SBP (7/28 tSNPs min-p = 0.0005), diastolic blood pressure, DBP (7/28 tSNPs min-p = 0.002) and 24 hour urinary potassium excretion (10/28 tSNPs min-p = 0.0004). Associations with SBP and urine potassium remained significant after correction for multiple testing (p = 0.02 and p = 0.01 respectively). The major allele (A) of rs765250, located in intron 1, demonstrated the strongest evidence for association with SBP, effect size 3.14 mmHg (95%CI:1.23-4.9), DBP 1.9 mmHg (95%CI:0.7-3.2) and hypertension, odds ratio (OR: 1.3 [95%CI: 1.0-1.7]).We genotyped this variant in six independent populations (n = 14,451) and replicated the association between rs765250 and SBP in a meta-analysis (p = 7 x 10(-3), combined with BRIGHT data-set p = 2 x 10(-4), n = 17,851). The associations of WNK1 with DBP and EH were not confirmed. Haplotype analysis revealed striking associations with hypertension and BP variation (global permutation p<10(-7)). We identified several common haplotypes to be associated with increased BP and multiple low frequency haplotypes significantly associated with lower BP (>10 mmHg reduction) and risk for hypertension (OR<0.60). Our data indicates that multiple rare and common WNK1 variants contribute to BP variation and hypertension, and provide compelling evidence to initiate further genetic and functional studies to explore the role of WNK1 in BP regulation and EH.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2009_po_nolte">
  <span class="title">Common Genetic Variation near the Phospholamban Gene Is Associated with Cardiac Repolarisation: Meta-Analysis of Three Genome-Wide Association Studies.</span>
  <span class="author">
    
    
    
    
    
    

    

    Nolte I, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Waggott D, 
    
    
    
    
    
    

    

    Fu J, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Gwilliam R, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Savelieva I, 
    
    
    
    
    
    

    

    Zheng D, 
    
    
    
    
    
    

    

    Dalageorgou C, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Zeggini E, 
    
    
    
    
    
    

    

    Wain L, 
    
    
    
    
    
    

    

    WTCCC , 
    
    
    
    
    
    

    

    DCCT/EDIC Research Group , 
    
    
    
    
    
    

    

    Newton-Cheh C, 
    
    
    
    
    
    

    

    Eijgelsheim M, 
    
    
    
    
    
    

    

    Rice K, 
    
    
    
    
    
    

    

    de Bakker P, 
    
    
    
    
    
    

    

    QTGEN consortium , 
    
    
    
    
    
    

    

    Pfeufer A, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Arking D, 
    
    
    
    
    
    

    

    QTSCD consortium , 
    
    
    
    
    
    

    

    Asselbergs F, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Carter N, 
    
    
    
    
    
    

    

    Jeffery S, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Snieder H, 
    
    
    
    
    
    

    

    Paterson A, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Jamshidi Y. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS One</em>
    
    
    2009
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pone.0006138" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>To identify loci affecting the electrocardiographic QT interval, a measure of cardiac repolarisation associated with risk of ventricular arrhythmias and sudden cardiac death, we conducted a meta-analysis of three genome-wide association studies (GWAS) including 3,558 subjects from the TwinsUK and BRIGHT cohorts in the UK and the DCCT/EDIC cohort from North America. Five loci were significantly associated with QT interval at P<1x10(-6). To validate these findings we performed an in silico comparison with data from two QT consortia: QTSCD (n = 15,842) and QTGEN (n = 13,685). Analysis confirmed the association between common variants near NOS1AP (P = 1.4x10(-83)) and the phospholamban (PLN) gene (P = 1.9x10(-29)). The most associated SNP near NOS1AP (rs12143842) explains 0.82% variance; the SNP near PLN (rs11153730) explains 0.74% variance of QT interval duration. We found no evidence for interaction between these two SNPs (P = 0.99). PLN is a key regulator of cardiac diastolic function and is involved in regulating intracellular calcium cycling, it has only recently been identified as a susceptibility locus for QT interval. These data offer further mechanistic insights into genetic influence on the QT interval which may predispose to life threatening arrhythmias and sudden cardiac death.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2009_ng_newtoncheh">
  <span class="title">Genome-Wide Association Study Identifies Eight Loci Associated with Blood Pressure.</span>
  <span class="author">
    
    
    
    
    
    

    

    Newton-Cheh C, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Gateva V, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Bochud M, 
    
    
    
    
    
    

    

    Coin L, 
    
    
    
    
    
    

    

    Najjar S, 
    
    
    
    
    
    

    

    Zhao J, 
    
    
    
    
    
    

    

    Heath S, 
    
    
    
    
    
    

    

    Eyheramendy S, 
    
    
    
    
    
    

    

    Papadakis K, 
    
    
    
    
    
    

    

    Voight B, 
    
    
    
    
    
    

    

    Scott L, 
    
    
    
    
    
    

    

    Zhang F, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Tanaka T, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Khaw K, 
    
    
    
    
    
    

    

    Nilsson P, 
    
    
    
    
    
    

    

    van der Harst P, 
    
    
    
    
    
    

    

    Polidoro S, 
    
    
    
    
    
    

    

    Grobbee D, 
    
    
    
    
    
    

    

    Onland-Moret N, 
    
    
    
    
    
    

    

    Bots M, 
    
    
    
    
    
    

    

    Wain L, 
    
    
    
    
    
    

    

    Elliott K, 
    
    
    
    
    
    

    

    Teumer A, 
    
    
    
    
    
    

    

    Luan J, 
    
    
    
    
    
    

    

    Lucas G, 
    
    
    
    
    
    

    

    Kuusisto J, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Hadley D, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Zeggini E, 
    
    
    
    
    
    

    

    Beckmann J, 
    
    
    
    
    
    

    

    Bergmann S, 
    
    
    
    
    
    

    

    Lim N, 
    
    
    
    
    
    

    

    Song K, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Yuan X, 
    
    
    
    
    
    

    

    Groop L, 
    
    
    
    
    
    

    

    Orho-Melander M, 
    
    
    
    
    
    

    

    Allione A, 
    
    
    
    
    
    

    

    Di Gregorio A, 
    
    
    
    
    
    

    

    Guarrera S, 
    
    
    
    
    
    

    

    Panico S, 
    
    
    
    
    
    

    

    Ricceri F, 
    
    
    
    
    
    

    

    Romanazzi V, 
    
    
    
    
    
    

    

    Sacerdote C, 
    
    
    
    
    
    

    

    Vineis P, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    Sandhu M, 
    
    
    
    
    
    

    

    Luben R, 
    
    
    
    
    
    

    

    Crawford G, 
    
    
    
    
    
    

    

    Jousilahti P, 
    
    
    
    
    
    

    

    Perola M, 
    
    
    
    
    
    

    

    Boehnke M, 
    
    
    
    
    
    

    

    Bonnycastle L, 
    
    
    
    
    
    

    

    Collins F, 
    
    
    
    
    
    

    

    Jackson A, 
    
    
    
    
    
    

    

    Mohlke K, 
    
    
    
    
    
    

    

    Stringham H, 
    
    
    
    
    
    

    

    Valle T, 
    
    
    
    
    
    

    

    Willer C, 
    
    
    
    
    
    

    

    Bergman R, 
    
    
    
    
    
    

    

    Morken M, 
    
    
    
    
    
    

    

    Döring A, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    Illig T, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Org E, 
    
    
    
    
    
    

    

    Pfeufer A, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Kathiresan S, 
    
    
    
    
    
    

    

    Marrugat J, 
    
    
    
    
    
    

    

    O’Donnell C, 
    
    
    
    
    
    

    

    Schwartz S, 
    
    
    
    
    
    

    

    Siscovick D, 
    
    
    
    
    
    

    

    Subirana I, 
    
    
    
    
    
    

    

    Freimer N, 
    
    
    
    
    
    

    

    Hartikainen A, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    O’Reilly P, 
    
    
    
    
    
    

    

    Peltonen L, 
    
    
    
    
    
    

    

    Pouta A, 
    
    
    
    
    
    

    

    de Jong P, 
    
    
    
    
    
    

    

    Snieder H, 
    
    
    
    
    
    

    

    van Gilst W, 
    
    
    
    
    
    

    

    Clarke R, 
    
    
    
    
    
    

    

    Goel A, 
    
    
    
    
    
    

    

    Hamsten A, 
    
    
    
    
    
    

    

    Peden J, 
    
    
    
    
    
    

    

    Seedorf U, 
    
    
    
    
    
    

    

    Syvänen A, 
    
    
    
    
    
    

    

    Tognoni G, 
    
    
    
    
    
    

    

    Lakatta E, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Scheet P, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Scuteri A, 
    
    
    
    
    
    

    

    Dörr M, 
    
    
    
    
    
    

    

    Ernst F, 
    
    
    
    
    
    

    

    Felix S, 
    
    
    
    
    
    

    

    Homuth G, 
    
    
    
    
    
    

    

    Lorbeer R, 
    
    
    
    
    
    

    

    Reffelmann T, 
    
    
    
    
    
    

    

    Rettig R, 
    
    
    
    
    
    

    

    Völker U, 
    
    
    
    
    
    

    

    Galan P, 
    
    
    
    
    
    

    

    Gut I, 
    
    
    
    
    
    

    

    Hercberg S, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Zelenika D, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Williams F, 
    
    
    
    
    
    

    

    Zhai G, 
    
    
    
    
    
    

    

    Salomaa V, 
    
    
    
    
    
    

    

    Laakso M, 
    
    
    
    
    
    

    

    Elosua R, 
    
    
    
    
    
    

    

    Forouhi N, 
    
    
    
    
    
    

    

    Völzke H, 
    
    
    
    
    
    

    

    Uiterwaal C, 
    
    
    
    
    
    

    

    van der Schouw Y, 
    
    
    
    
    
    

    

    Numans M, 
    
    
    
    
    
    

    

    Matullo G, 
    
    
    
    
    
    

    

    Navis G, 
    
    
    
    
    
    

    

    Berglund G, 
    
    
    
    
    
    

    

    Bingham S, 
    
    
    
    
    
    

    

    Kooner J, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Bandinelli S, 
    
    
    
    
    
    

    

    Ferrucci L, 
    
    
    
    
    
    

    

    Watkins H, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Tuomilehto J, 
    
    
    
    
    
    

    

    Altshuler D, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Laan M, 
    
    
    
    
    
    

    

    Meneton P, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Jarvelin M, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Melander O, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2009
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.361" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2891673" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Elevated blood pressure is a common, heritable cause of cardiovascular disease worldwide. To date, identification of common genetic variants influencing blood pressure has proven challenging. We tested 2.5 million genotyped and imputed SNPs for association with systolic and diastolic blood pressure in 34,433 subjects of European ancestry from the Global BPgen consortium and followed up findings with direct genotyping (N ≤ 71,225 European ancestry, N ≤ 12,889 Indian Asian ancestry) and in silico comparison (CHARGE consortium, N = 29,136). We identified association between systolic or diastolic blood pressure and common variants in eight regions near the CYP17A1 (P = 7 \texttimes 10(-24)), CYP1A2 (P = 1 \texttimes 10(-23)), FGF5 (P = 1 \texttimes 10(-21)), SH2B3 (P = 3 \texttimes 10(-18)), MTHFR (P = 2 \texttimes 10(-13)), c10orf107 (P = 1 \texttimes 10(-9)), ZNF652 (P = 5 \texttimes 10(-9)) and PLCD3 (P = 1 \texttimes 10(-8)) genes. All variants associated with continuous blood pressure were associated with dichotomous hypertension. These associations between common variants and blood pressure and hypertension offer mechanistic insights into the regulation of blood pressure and may point to novel targets for interventions to prevent cardiovascular disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2008</h2>
<ol class="bibliography"><li>

<div id="2008_ajhg">
  <span class="title">Genome-Wide Association Study Identifies Novel Genes for Biomarkers of Cardiovascular Disease: Serum Urate and Dyslipidemia</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Braund P, 
    
    
    
    
    
    

    

    Zhang F, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Falchi M, 
    
    
    
    
    
    

    

    Ahmadi K, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Marçano A, 
    
    
    
    
    
    

    

    Hajat C, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    The Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Spector T, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Am J Hum Genet</em>
    
    
    2008
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1016/j.ajhg.2007.11.001" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Many common diseases are accompanied by disturbances in biochemical traits. Identifying the genetic determinants could provide novel insights into disease mechanisms and reveal avenues for developing new therapies. Here we report a genome-wide association analysis for commonly measured serum and urine biochemical traits. As part of the WTCCC, 500,000 SNPs genomewide were genotyped in 1955 hypertensive individuals characterized for 25 serum and urine biochemical traits. For each trait we assessed association with individual SNPs, adjusting for age, sex and BMI. Lipid measurements were further examined in a meta-analysis of genome-wide data from a type 2 diabetes scan. The most promising associations were examined in two epidemiological cohorts. We discovered association between serum urate and SLC2A9, a glucose transporter (p = 2Ã10-15) and confirmed this in two independent cohorts, GRAPHIC study (p = 9Ã10-15), TwinsUK (p = 8Ã10-19). The odds ratio for hyperuricaemia (defined as urate > 0.4mMol/l) is 1.89 (95% CI = 1.36-2.61) per copy of common allele. We also replicated many genes previously associated with serum lipids and found novel association between LDL levels and SNPs close to genes encoding PSRC1 and CELSR2 (p = 1Ã10-7). The common allele was associated with a 6% increase in non-fasting serum LDL. This region showed increased association in the meta-analysis (p = 4Ã10-14) and this finding provides a potential biological mechanism for the recent association of this same allele of the same SNP with increased risk of coronary disease.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2008_jhyp_delles">
  <span class="title">Glutathione S-Transferase Variants and Hypertension.</span>
  <span class="author">
    
    
    
    
    
    

    

    Delles C, 
    
    
    
    
    
    

    

    Padmanabhan S, 
    
    
    
    
    
    

    

    Lee W, 
    
    
    
    
    
    

    

    Miller W, 
    
    
    
    
    
    

    

    McBride M, 
    
    
    
    
    
    

    

    McClure J, 
    
    
    
    
    
    

    

    Brain N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    MarÃ\textsectionano A, 
    
    
    
    
    
    

    

    Schmieder R, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dominiczak A. 
    
  </span>
  
  <div class="jblocks">
    
    <em>J Hypertens</em>
    
    
    2008
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1097/HJH.0b013e3282fe1d67" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>OBJECTIVES: Glutathione S-transferases are involved in defences against oxidative stress. We have recently demonstrated reduced expression of glutathione S-transferase mu type 1 (Gstm1) in a rat model of hypertension. Here, we examine the association between GSTM variants and hypertension in human. METHODS: We screened 83 patients with hypertension and 46 controls for single nucleotide polymorphisms in GSTM genes by TaqMan single nucleotide polymorphism genotyping assays and DNA sequencing. We then genotyped 753 trios from the Medical Research Council British Genetics of Hypertension Study transmission disequilibrium test cohort for 10 single nucleotide polymorphisms and the GSTM1 deletion and examined renal GSTM expression in a cohort of 27 hypertensive and 18 normotensive subjects. Finally, we attempted to replicate our findings in 1675 cases and 1654 controls from the Medical Research Council British Genetics of Hypertension Study case-control cohort. RESULTS: We identified two major linkage disequilibrium blocks including GSTM4/GSTM2 and GSTM5/GSTM3 separated by the GSTM1 gene. In the British Genetics of Hypertension transmission disequilibrium test resource, a single nucleotide polymorphism in the 3’ region of GSTM5 (rs11807) was found to be associated with hypertension (P = 0.01) with the T-allele being over-transmitted to hypertensive offspring. GSTM5 mRNA expression was found to be reduced in kidney tissue of subjects homozygous for the T-allele of rs11807 as compared to C-allele homozygous and CT heterozygous subjects (P = 0.02). Nevertheless, rs11807 was not associated with hypertension in the British Genetics of Hypertension case-control cohort (P = 0.61). CONCLUSION: Our studies do not provide an evidence of an association of GSTM gene variants with hypertension in humans. They, however, illustrate the essential role of replication of initial results in a second cohort.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2008_ng_weedon_height">
  <span class="title">Genome-Wide Association Analysis Identifies 20 Loci That Influence Adult Height.</span>
  <span class="author">
    
    
    
    
    
    

    

    Weedon M, 
    
    
    
    
    
    

    

    Lango H, 
    
    
    
    
    
    

    

    Lindgren C, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Evans D, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    Freathy R, 
    
    
    
    
    
    

    

    Perry J, 
    
    
    
    
    
    

    

    Stevens S, 
    
    
    
    
    
    

    

    AS H, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Shields B, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Diabetes Genetics Initiative , 
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Bergmann S, 
    
    
    
    
    
    

    

    Beckmann J, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Palmer C, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Cambridge GEM Consortium , 
    
    
    
    
    
    

    

    Zhao J, 
    
    
    
    
    
    

    

    Li S, 
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Sandhu M, 
    
    
    
    
    
    

    

    Wheeler E, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Inouye M, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Frayling T. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2008
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.121" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Adult height is a model polygenic trait, but there has been limited success in identifying the genes underlying its normal variation. To identify genetic variants influencing adult human height, we used genome-wide association data from 13,665 individuals and genotyped 39 variants in an additional 16,482 samples. We identified 20 variants associated with adult height (P < 5 x 10(-7), with 10 reaching P < 1 x 10(-10)). Combined, the 20 SNPs explain approximately 3% of height variation, with a approximately 5 cm difference between the 6.2% of people with 17 or fewer ’tall’ alleles compared to the 5.5% with 27 or more ’tall’ alleles. The loci we identified implicate genes in Hedgehog signaling (IHH, HHIP, PTCH1), extracellular matrix (EFEMP1, ADAMTSL3, ACAN) and cancer (CDK6, HMGA2, DLEU7) pathways, and provide new insights into human growth and developmental processes. Finally, our results provide insights into the genetic architecture of a classic quantitative trait.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2008_plos_slc2a9">
  <span class="title">SLC2A9 Is a High-Capacity Urate Transporter in Humans.</span>
  <span class="author">
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    O’Neill D, 
    
    
    
    
    
    

    

    Witkowska K, 
    
    
    
    
    
    

    

    Charchar F, 
    
    
    
    
    
    

    

    Doblado M, 
    
    
    
    
    
    

    

    Evans S, 
    
    
    
    
    
    

    

    Eyheramendy S, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    Howard P, 
    
    
    
    
    
    

    

    Shaw-Hawkins S, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Kumari M, 
    
    
    
    
    
    

    

    Marmot M, 
    
    
    
    
    
    

    

    Brunner E, 
    
    
    
    
    
    

    

    Chambers J, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Kooner J, 
    
    
    
    
    
    

    

    Laan M, 
    
    
    
    
    
    

    

    Org E, 
    
    
    
    
    
    

    

    Veldre G, 
    
    
    
    
    
    

    

    Viigimaa M, 
    
    
    
    
    
    

    

    Cappuccio F, 
    
    
    
    
    
    

    

    Ji C, 
    
    
    
    
    
    

    

    Iacone R, 
    
    
    
    
    
    

    

    Strazzullo P, 
    
    
    
    
    
    

    

    Moley K, 
    
    
    
    
    
    

    

    Cheeseman C. 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS Med</em>
    
    
    2008
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pmed.0050197" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>BACKGROUND: Serum uric acid levels in humans are influenced by diet, cellular breakdown, and renal elimination, and correlate with blood pressure, metabolic syndrome, diabetes, gout, and cardiovascular disease. Recent genome-wide association scans have found common genetic variants of SLC2A9 to be associated with increased serum urate level and gout. The SLC2A9 gene encodes a facilitative glucose transporter, and it has two splice variants that are highly expressed in the proximal nephron, a key site for urate handling in the kidney. We investigated whether SLC2A9 is a functional urate transporter that contributes to the longstanding association between urate and blood pressure in man. METHODS AND FINDINGS: We expressed both SLC2A9 splice variants in Xenopus laevis oocytes and found both isoforms mediate rapid urate fluxes at concentration ranges similar to physiological serum levels (200-500 muM). Because SLC2A9 is a known facilitative glucose transporter, we also tested whether glucose or fructose influenced urate transport. We found that urate is transported by SLC2A9 at rates 45- to 60-fold faster than glucose, and demonstrated that SLC2A9-mediated urate transport is facilitated by glucose and, to a lesser extent, fructose. In addition, transport is inhibited by the uricosuric benzbromarone in a dose-dependent manner (Ki = 27 muM). Furthermore, we found urate uptake was at least 2-fold greater in human embryonic kidney (HEK) cells overexpressing SLC2A9 splice variants than nontransfected kidney cells. To confirm that our findings were due to SLC2A9, and not another urate transporter, we showed that urate transport was diminished by SLC2A9-targeted siRNA in a second mammalian cell line. In a cohort of men we showed that genetic variants of SLC2A9 are associated with reduced urinary urate clearance, which fits with common variation at SLC2A9 leading to increased serum urate. We found no evidence of association with hypertension (odds ratio 0.98, 95% confidence interval [CI] 0.9 to 1.05, p > 0.33) by meta-analysis of an SLC2A9 variant in six case-control studies including 11,897 participants. In a separate meta-analysis of four population studies including 11,629 participants we found no association of SLC2A9 with systolic (effect size -0.12 mm Hg, 95% CI -0.68 to 0.43, p = 0.664) or diastolic blood pressure (effect size -0.03 mm Hg, 95% CI -0.39 to 0.31, p = 0.82). CONCLUSIONS: This study provides evidence that SLC2A9 splice variants act as high-capacity urate transporters and is one of the first functional characterisations of findings from genome-wide association scans. We did not find an association of the SLC2A9 gene with blood pressure in this study. Our findings suggest potential pathogenic mechanisms that could offer a new drug target for gout.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2008_plagnol">
  <span class="title">Extreme Clonality in Lymphoblastoid Cell Lines with Implications for Allele Specific Expression Analyses</span>
  <span class="author">
    
    
    
    
    
    

    

    Plagnol, Vincent AND Uz, Elif AND C Wallace AND Stevens, Helen AND Clayton, David AND Ozcelik, Tayfun AND Todd, John A. . 
    
  </span>
  
  <div class="jblocks">
    
    <em>PLoS ONE</em>
    
    
    2008
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1371/journal.pone.0002966" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2494943" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Lymphoblastoid cell lines (LCL) are being actively and extensively used to examine the expression of specific genes and genome-wide expression profiles, including allele specific expression assays. However, it has recently been shown that approximately 10% of human genes exhibit random patterns of monoallelic expression within single clones of LCLs. Consequently allelic imbalance studies could be significantly compromised if bulk populations of donor cells are clonal, or near clonal. Here, using X chromosome inactivation as a readout, we confirm and quantify widespread near monoclonality in two independent sets of cell lines. Consequently, we recommend where possible the use of bulk, non cell line, ex vivo cells for allele specific expression assays.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2008_ng_loos_mc4r">
  <span class="title">Common Variants near MC4R Are Associated with Fat Mass, Weight and Risk of Obesity</span>
  <span class="author">
    
    
    
    
    
    

    

    Loos R, 
    
    
    
    
    
    

    

    Lindgren C, 
    
    
    
    
    
    

    

    Li S, 
    
    
    
    
    
    

    

    Wheeler E, 
    
    
    
    
    
    

    

    Zhao J, 
    
    
    
    
    
    

    

    Prokopenko I, 
    
    
    
    
    
    

    

    Inouye M, 
    
    
    
    
    
    

    

    Freathy R, 
    
    
    
    
    
    

    

    Attwood A, 
    
    
    
    
    
    

    

    Beckmann J, 
    
    
    
    
    
    

    

    Berndt S, 
    
    
    
    
    
    

    

    Prostate , 
    
    
    
    
    
    

    

    Trial O, 
    
    
    
    
    
    

    

    Jacobs K, 
    
    
    
    
    
    

    

    Chanock S, 
    
    
    
    
    
    

    

    Hayes R, 
    
    
    
    
    
    

    

    Bergmann S, 
    
    
    
    
    
    

    

    Bennett A, 
    
    
    
    
    
    

    

    Bingham S, 
    
    
    
    
    
    

    

    Bochud M, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Cauchi S, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Cooper C, 
    
    
    
    
    
    

    

    Smith G, 
    
    
    
    
    
    

    

    Day I, 
    
    
    
    
    
    

    

    Dina C, 
    
    
    
    
    
    

    

    De S, 
    
    
    
    
    
    

    

    Dermitzakis E, 
    
    
    
    
    
    

    

    Doney A, 
    
    
    
    
    
    

    

    Elliott K, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Evans D, 
    
    
    
    
    
    

    

    Farooqi I, 
    
    
    
    
    
    

    

    Froguel P, 
    
    
    
    
    
    

    

    Ghori J, 
    
    
    
    
    
    

    

    Groves C, 
    
    
    
    
    
    

    

    Gwilliam R, 
    
    
    
    
    
    

    

    Hadley D, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hebebrand J, 
    
    
    
    
    
    

    

    Heid I, 
    
    
    
    
    
    

    

    A. K, 
    
    
    
    
    
    

    

    Lamina C, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    Illig T, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Herrera B, 
    
    
    
    
    
    

    

    Hinney A, 
    
    
    
    
    
    

    

    Hunt S, 
    
    
    
    
    
    

    

    Jarvelin M, 
    
    
    
    
    
    

    

    Johnson T, 
    
    
    
    
    
    

    

    Jolley J, 
    
    
    
    
    
    

    

    Karpe F, 
    
    
    
    
    
    

    

    Keniry A, 
    
    
    
    
    
    

    

    Khaw K, 
    
    
    
    
    
    

    

    Luben R, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    Marchini J, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    McGinnis R, 
    
    
    
    
    
    

    

    Meyre D, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Ness A, 
    
    
    
    
    
    

    

    Neville M, 
    
    
    
    
    
    

    

    Nica A, 
    
    
    
    
    
    

    

    Ong K, 
    
    
    
    
    
    

    

    O’Rahilly S, 
    
    
    
    
    
    

    

    Owen K, 
    
    
    
    
    
    

    

    Palmer C, 
    
    
    
    
    
    

    

    Papadakis K, 
    
    
    
    
    
    

    

    Potter S, 
    
    
    
    
    
    

    

    Pouta A, 
    
    
    
    
    
    

    

    Qi L, 
    
    
    
    
    
    

    

    Study N, 
    
    
    
    
    
    

    

    Randall J, 
    
    
    
    
    
    

    

    Rayner N, 
    
    
    
    
    
    

    

    Ring S, 
    
    
    
    
    
    

    

    Sandhu M, 
    
    
    
    
    
    

    

    Scherag A, 
    
    
    
    
    
    

    

    Sims M, 
    
    
    
    
    
    

    

    Song K, 
    
    
    
    
    
    

    

    Soranzo N, 
    
    
    
    
    
    

    

    Speliotes E, 
    
    
    
    
    
    

    

    Initiative D, 
    
    
    
    
    
    

    

    Syddall H, 
    
    
    
    
    
    

    

    Teichmann S, 
    
    
    
    
    
    

    

    Timpson N, 
    
    
    
    
    
    

    

    Tobias J, 
    
    
    
    
    
    

    

    Uda M, 
    
    
    
    
    
    

    

    Study S, 
    
    
    
    
    
    

    

    Vogel C, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Waterworth D, 
    
    
    
    
    
    

    

    Weedon M, 
    
    
    
    
    
    

    

    Consortium W, 
    
    
    
    
    
    

    

    Willer C, 
    
    
    
    
    
    

    

    N. F, 
    
    
    
    
    
    

    

    Wraight , 
    
    
    
    
    
    

    

    Yuan X, 
    
    
    
    
    
    

    

    Zeggini E, 
    
    
    
    
    
    

    

    Hirschhorn J, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Frayling T, 
    
    
    
    
    
    

    

    Vollenweider P, 
    
    
    
    
    
    

    

    Waeber G, 
    
    
    
    
    
    

    

    Mooser V, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Wareham N, 
    
    
    
    
    
    

    

    Barroso I, 
    
    
    
    
    
    

    

    Jacobs K, 
    
    
    
    
    
    

    

    Chanock S, 
    
    
    
    
    
    

    

    Hayes R, 
    
    
    
    
    
    

    

    Lamina C, 
    
    
    
    
    
    

    

    Gieger C, 
    
    
    
    
    
    

    

    Illig T, 
    
    
    
    
    
    

    

    Meitinger T, 
    
    
    
    
    
    

    

    Wichmann H, 
    
    
    
    
    
    

    

    Kraft P, 
    
    
    
    
    
    

    

    Hankinson S, 
    
    
    
    
    
    

    

    Hunter D, 
    
    
    
    
    
    

    

    Hu F, 
    
    
    
    
    
    

    

    Lyon H, 
    
    
    
    
    
    

    

    Voight B, 
    
    
    
    
    
    

    

    Ridderstrale M, 
    
    
    
    
    
    

    

    Groop L, 
    
    
    
    
    
    

    

    Scheet P, 
    
    
    
    
    
    

    

    Sanna S, 
    
    
    
    
    
    

    

    Abecasis G, 
    
    
    
    
    
    

    

    Albai G, 
    
    
    
    
    
    

    

    Nagaraja R, 
    
    
    
    
    
    

    

    Schlessinger D, 
    
    
    
    
    
    

    

    Jackson A, 
    
    
    
    
    
    

    

    Tuomilehto J, 
    
    
    
    
    
    

    

    Collins F, 
    
    
    
    
    
    

    

    Boehnke M, 
    
    
    
    
    
    

    

    Mohlke K. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Genet.</em>
    
    
    2008
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.140" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2669167" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>To identify common variants influencing body mass index (BMI), we analyzed genome-wide association data from 16,876 individuals of European descent. After previously reported variants in FTO, the strongest association signal (rs17782313, P = 2.9 x 10(-6)) mapped 188 kb downstream of MC4R (melanocortin-4 receptor), mutations of which are the leading cause of monogenic severe childhood-onset obesity. We confirmed the BMI association in 60,352 adults (per-allele effect = 0.05 Z-score units; P = 2.8 x 10(-15)) and 5,988 children aged 7-11 (0.13 Z-score units; P = 1.5 x 10(-8)). In case-control analyses (n = 10,583), the odds for severe childhood obesity reached 1.30 (P = 8.0 x 10(-11)). Furthermore, we observed overtransmission of the risk allele to obese offspring in 660 families (P (pedigree disequilibrium test average; PDT-avg) = 2.4 x 10(-4)). The SNP location and patterns of phenotypic associations are consistent with effects mediated through altered MC4R function. Our findings establish that common variants near MC4R influence fat mass, weight and obesity risk at the population level and reinforce the need for large-scale data integration to identify variants influencing continuous biomedical traits.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2007</h2>
<ol class="bibliography"><li>

<div id="2007_genres">
  <span class="title">Information Capture Using SNPs from HapMap and Whole-Genome Chips Differs in a Sample of Inflammatory and Cardiovascular Gene-Centric Regions from Genome-Wide Estimates.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Caulfield M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genome Res</em>
    
    
    2007
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1101/gr.5996407" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Large-scale genetic association studies are now widely conducted using SNPs selected from the International HapMap Project or provided on commercial "whole genome" chips. As only a subset of human genetic variation has been identified, it is unknown what proportion of the total genetic variation can be captured in this way, although recent genome-wide estimates of SNP capture rates have been encouraging. We estimated the expected gene-centric information capture for whole-genome chips using sequence data from 306 inflammatory/cardiovascular genes and found SNP capture rates notably lower than previous genome-wide estimates. Further investigation indicates that a major explanation for these lower capture rates is the aggregation of particular sequence features that influence both linkage disequilibrium and the amenability of SNPs for genotyping within the broad class of inflammatory/ cardiovascular genes. This suggests that the power of genetic association studies in some complex traits will depend not only upon established factors, such as allele frequency and penetrance, but may also be influenced by the distribution of sequence features in the class of genes expected to underlie the disease of interest.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2007_marcano">
  <span class="title">Genetic Association Analysis of Inositol Polyphosphate Phosphatase-like 1 (INPPL1, SHIP2) Variants with Essential Hypertension.</span>
  <span class="author">
    
    
    
    
    
    

    

    Marçano A, 
    
    
    
    
    
    

    

    Burke B, 
    
    
    
    
    
    

    

    Gungadoo J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Kaisaki P, 
    
    
    
    
    
    

    

    Woon P, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Gauguier D, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>J Med Genet</em>
    
    
    2007
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1136/jmg.2007.049718" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Inositol polyphosphate phosphatase-like 1 (INPPL1, SHIP2) is a negative regulator of insulin signalling and was previously found to be associated with hypertension, obesity and type 2 diabetes in a cohort of British diabetic families presenting features of the metabolic syndrome. In particular, a haplotype of three genetic polymorphisms (rs2276047, rs9886 and an insertion/deletion polymorphism in intron 1) was found to be strongly associated with increased susceptibility to hypertension. To assess if INPPL1 variants play a direct role in the development of essential hypertension, we genotyped the three previously associated INPPL1 polymorphisms in a cohort of 712 severely hypertensive TDT families from the BRIGHT study. We found no evidence of significant association between hypertension and any of the three INPPL1 polymorphisms or haplotypes (p > 0.1). These results suggest that INPPL1 variants may be involved in mechanisms causing hypertension specifically in the context of the metabolic syndrome.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="burton_genome-wide_2007">
  <span class="title">Genome-Wide Association Study of 14,000 Cases of Seven Common Diseases and 3,000 Shared Controls</span>
  <span class="author">
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Cardon L, 
    
    
    
    
    
    

    

    Craddock N, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Duncanson A, 
    
    
    
    
    
    

    

    Kwiatkowski D, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Donnelly P, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Davison D, 
    
    
    
    
    
    

    

    Donnelly P, 
    
    
    
    
    
    

    

    Easton D, 
    
    
    
    
    
    

    

    Evans D, 
    
    
    
    
    
    

    

    Leung H, 
    
    
    
    
    
    

    

    Marchini J, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Spencer C, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Cardon L, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Attwood A, 
    
    
    
    
    
    

    

    Boorman J, 
    
    
    
    
    
    

    

    Cant B, 
    
    
    
    
    
    

    

    Everson U, 
    
    
    
    
    
    

    

    Hussey J, 
    
    
    
    
    
    

    

    Jolley J, 
    
    
    
    
    
    

    

    Knight A, 
    
    
    
    
    
    

    

    Koch K, 
    
    
    
    
    
    

    

    Meech E, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Prowse C, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Taylor N, 
    
    
    
    
    
    

    

    Walters G, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Watkins N, 
    
    
    
    
    
    

    

    Winzer T, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Jones R, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    Ring S, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Pembrey M, 
    
    
    
    
    
    

    

    Breen G, 
    
    
    
    
    
    

    

    St Clair D, 
    
    
    
    
    
    

    

    Caesar S, 
    
    
    
    
    
    

    

    Gordon-Smith K, 
    
    
    
    
    
    

    

    Jones L, 
    
    
    
    
    
    

    

    Fraser C, 
    
    
    
    
    
    

    

    Green E, 
    
    
    
    
    
    

    

    Grozeva D, 
    
    
    
    
    
    

    

    Hamshere M, 
    
    
    
    
    
    

    

    Holmans P, 
    
    
    
    
    
    

    

    Jones I, 
    
    
    
    
    
    

    

    Kirov G, 
    
    
    
    
    
    

    

    Moskvina V, 
    
    
    
    
    
    

    

    Nikolov I, 
    
    
    
    
    
    

    

    O’Donovan M, 
    
    
    
    
    
    

    

    Owen M, 
    
    
    
    
    
    

    

    Craddock N, 
    
    
    
    
    
    

    

    Collier D, 
    
    
    
    
    
    

    

    Elkin A, 
    
    
    
    
    
    

    

    Farmer A, 
    
    
    
    
    
    

    

    Williamson R, 
    
    
    
    
    
    

    

    McGuffin P, 
    
    
    
    
    
    

    

    Young A, 
    
    
    
    
    
    

    

    Ferrier I, 
    
    
    
    
    
    

    

    Ball S, 
    
    
    
    
    
    

    

    Balmforth A, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Bishop D, 
    
    
    
    
    
    

    

    Iles M, 
    
    
    
    
    
    

    

    Maqbool A, 
    
    
    
    
    
    

    

    Yuldasheva N, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Braund P, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Dixon R, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    Stevens S, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Thompson J, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Bredin F, 
    
    
    
    
    
    

    

    Tremelling M, 
    
    
    
    
    
    

    

    Parkes M, 
    
    
    
    
    
    

    

    Drummond H, 
    
    
    
    
    
    

    

    Lees C, 
    
    
    
    
    
    

    

    Nimmo E, 
    
    
    
    
    
    

    

    Satsangi J, 
    
    
    
    
    
    

    

    Fisher S, 
    
    
    
    
    
    

    

    Forbes A, 
    
    
    
    
    
    

    

    Lewis C, 
    
    
    
    
    
    

    

    Onnie C, 
    
    
    
    
    
    

    

    Prescott N, 
    
    
    
    
    
    

    

    Sanderson J, 
    
    
    
    
    
    

    

    Mathew C, 
    
    
    
    
    
    

    

    Barbour J, 
    
    
    
    
    
    

    

    Mohiuddin M, 
    
    
    
    
    
    

    

    Todhunter C, 
    
    
    
    
    
    

    

    Mansfield J, 
    
    
    
    
    
    

    

    Ahmad T, 
    
    
    
    
    
    

    

    Cummings F, 
    
    
    
    
    
    

    

    Jewell D, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Marcano C, 
    
    
    
    
    
    

    

    Burke B, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Gungadoo J, 
    
    
    
    
    
    

    

    Lee K, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Xue M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Barton A, 
    
    
    
    
    
    

    

    Genomics (BRAGGS) T, 
    
    
    
    
    
    

    

    Bruce I, 
    
    
    
    
    
    

    

    Donovan H, 
    
    
    
    
    
    

    

    Eyre S, 
    
    
    
    
    
    

    

    Gilbert P, 
    
    
    
    
    
    

    

    Hider S, 
    
    
    
    
    
    

    

    Hinks A, 
    
    
    
    
    
    

    

    John S, 
    
    
    
    
    
    

    

    Potter C, 
    
    
    
    
    
    

    

    Silman A, 
    
    
    
    
    
    

    

    Symmons D, 
    
    
    
    
    
    

    

    Thomson W, 
    
    
    
    
    
    

    

    Worthington J, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Widmer B, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Frayling T, 
    
    
    
    
    
    

    

    Freathy R, 
    
    
    
    
    
    

    

    Lango H, 
    
    
    
    
    
    

    

    Perry J, 
    
    
    
    
    
    

    

    Shields B, 
    
    
    
    
    
    

    

    Weedon M, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hitman G, 
    
    
    
    
    
    

    

    Walker M, 
    
    
    
    
    
    

    

    Elliott K, 
    
    
    
    
    
    

    

    Groves C, 
    
    
    
    
    
    

    

    Lindgren C, 
    
    
    
    
    
    

    

    Rayner N, 
    
    
    
    
    
    

    

    Timpson N, 
    
    
    
    
    
    

    

    Zeggini E, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Newport M, 
    
    
    
    
    
    

    

    Sirugo G, 
    
    
    
    
    
    

    

    Lyons E, 
    
    
    
    
    
    

    

    Vannberg F, 
    
    
    
    
    
    

    

    Hill A, 
    
    
    
    
    
    

    

    Bradbury L, 
    
    
    
    
    
    

    

    Farrar C, 
    
    
    
    
    
    

    

    Pointon J, 
    
    
    
    
    
    

    

    Wordsworth P, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Franklyn J, 
    
    
    
    
    
    

    

    Heward J, 
    
    
    
    
    
    

    

    Simmonds M, 
    
    
    
    
    
    

    

    Gough S, 
    
    
    
    
    
    

    

    Seal S, 
    
    
    
    
    
    

    

    Susceptibility Collaboration (UK) B, 
    
    
    
    
    
    

    

    Stratton M, 
    
    
    
    
    
    

    

    Rahman N, 
    
    
    
    
    
    

    

    Ban M, 
    
    
    
    
    
    

    

    Goris A, 
    
    
    
    
    
    

    

    Sawcer S, 
    
    
    
    
    
    

    

    Compston A, 
    
    
    
    
    
    

    

    Conway D, 
    
    
    
    
    
    

    

    Jallow M, 
    
    
    
    
    
    

    

    Newport M, 
    
    
    
    
    
    

    

    Sirugo G, 
    
    
    
    
    
    

    

    Rockett K, 
    
    
    
    
    
    

    

    Kwiatkowski D, 
    
    
    
    
    
    

    

    Bumpstead S, 
    
    
    
    
    
    

    

    Chaney A, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Ghori M, 
    
    
    
    
    
    

    

    Gwilliam R, 
    
    
    
    
    
    

    

    Hunt S, 
    
    
    
    
    
    

    

    Inouye M, 
    
    
    
    
    
    

    

    Keniry A, 
    
    
    
    
    
    

    

    King E, 
    
    
    
    
    
    

    

    McGinnis R, 
    
    
    
    
    
    

    

    Potter S, 
    
    
    
    
    
    

    

    Ravindrarajah R, 
    
    
    
    
    
    

    

    Whittaker P, 
    
    
    
    
    
    

    

    Widden C, 
    
    
    
    
    
    

    

    Withers D, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Leung H, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Easton D, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Evans D, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Cardon L, 
    
    
    
    
    
    

    

    Cardin N, 
    
    
    
    
    
    

    

    Davison D, 
    
    
    
    
    
    

    

    Ferreira T, 
    
    
    
    
    
    

    

    Pereira-Gale J, 
    
    
    
    
    
    

    

    Hallgrimsdóttir I, 
    
    
    
    
    
    

    

    Howie B, 
    
    
    
    
    
    

    

    Marchini J, 
    
    
    
    
    
    

    

    Spencer C, 
    
    
    
    
    
    

    

    Su Z, 
    
    
    
    
    
    

    

    Teo Y, 
    
    
    
    
    
    

    

    Vukcevic D, 
    
    
    
    
    
    

    

    Donnelly P, 
    
    
    
    
    
    

    

    Bentley D, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Cardon L, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Compston A, 
    
    
    
    
    
    

    

    Craddock N, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Donnelly P, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Gough S, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hill A, 
    
    
    
    
    
    

    

    Kwiatkowski D, 
    
    
    
    
    
    

    

    Mathew C, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Parkes M, 
    
    
    
    
    
    

    

    Pembrey M, 
    
    
    
    
    
    

    

    Rahman N, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Stratton M, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Worthington J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nature</em>
    
    
    2007
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/nature05911" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>There is increasing evidence that genome-wide association (GWA) studies represent a powerful approach to the identification of genes involved in common human diseases. We describe a joint GWA study (using the Affymetrix GeneChip 500K Mapping Array Set) undertaken in the British population, which has examined approximately 2,000 individuals for each of 7 major diseases and a shared set of approximately 3,000 controls. Case-control comparisons identified 24 independent association signals at P < 5 x 10(-7): 1 in bipolar disorder, 1 in coronary artery disease, 9 in Crohn’s disease, 3 in rheumatoid arthritis, 7 in type 1 diabetes and 3 in type 2 diabetes. On the basis of prior findings and replication studies thus-far completed, almost all of these signals reflect genuine susceptibility effects. We observed association at many previously identified loci, and found compelling evidence that some loci confer risk for more than one of the diseases studied. Across all diseases, we identified a large number of further signals (including 58 loci with single-point P values between 10(-5) and 5 x 10(-7)) likely to yield additional susceptibility loci. The importance of appropriately large samples was confirmed by the modest effect sizes observed at most loci identified. This study thus represents a thorough validation of the GWA approach. It has also demonstrated that careful use of a shared control group represents a safe and effective approach to GWA analyses of multiple disease phenotypes; has generated a genome-wide genotype database for future studies of common diseases in the British population; and shown that, provided individuals with non-European ancestry are excluded, the extent of population stratification in the British population is generally modest. Our findings offer new avenues for exploring the pathophysiology of these important disorders. We anticipate that our data, results and software, which will be widely available to other investigators, will provide a powerful resource for human genetics research.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2007_wtccc">
  <span class="title">Association Scan of 14,500 Nonsynonymous SNPs in Four Diseases Identifies Autoimmunity Variants</span>
  <span class="author">
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Australo-Anglo-American Spondylitis Consortium (TASC) , 
    
    
    
    
    
    

    

    Burton P, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Cardon L, 
    
    
    
    
    
    

    

    Craddock N, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Duncanson A, 
    
    
    
    
    
    

    

    Kwiatkowski D, 
    
    
    
    
    
    

    

    McCarthy M, 
    
    
    
    
    
    

    

    Ouwehand W, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Donnelly P, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Davison D, 
    
    
    
    
    
    

    

    Easton D, 
    
    
    
    
    
    

    

    Evans D, 
    
    
    
    
    
    

    

    Leung H, 
    
    
    
    
    
    

    

    Marchini J, 
    
    
    
    
    
    

    

    Morris A, 
    
    
    
    
    
    

    

    Spencer C, 
    
    
    
    
    
    

    

    Tobin M, 
    
    
    
    
    
    

    

    Attwood A, 
    
    
    
    
    
    

    

    Boorman J, 
    
    
    
    
    
    

    

    Cant B, 
    
    
    
    
    
    

    

    Everson U, 
    
    
    
    
    
    

    

    Hussey J, 
    
    
    
    
    
    

    

    Jolley J, 
    
    
    
    
    
    

    

    Knight A, 
    
    
    
    
    
    

    

    Koch K, 
    
    
    
    
    
    

    

    Meech E, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Prowse C, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Taylor N, 
    
    
    
    
    
    

    

    Walters G, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Watkins N, 
    
    
    
    
    
    

    

    Winzer T, 
    
    
    
    
    
    

    

    Jones R, 
    
    
    
    
    
    

    

    McArdle W, 
    
    
    
    
    
    

    

    Ring S, 
    
    
    
    
    
    

    

    Strachan D, 
    
    
    
    
    
    

    

    Pembrey M, 
    
    
    
    
    
    

    

    Breen G, 
    
    
    
    
    
    

    

    St Clair D, 
    
    
    
    
    
    

    

    Caesar S, 
    
    
    
    
    
    

    

    Gordon-Smith K, 
    
    
    
    
    
    

    

    Jones L, 
    
    
    
    
    
    

    

    Fraser C, 
    
    
    
    
    
    

    

    Green E, 
    
    
    
    
    
    

    

    Grozeva D, 
    
    
    
    
    
    

    

    Hamshere M, 
    
    
    
    
    
    

    

    Holmans P, 
    
    
    
    
    
    

    

    Jones I, 
    
    
    
    
    
    

    

    Kirov G, 
    
    
    
    
    
    

    

    Moskivina V, 
    
    
    
    
    
    

    

    Nikolov I, 
    
    
    
    
    
    

    

    O’Donovan M, 
    
    
    
    
    
    

    

    Owen M, 
    
    
    
    
    
    

    

    Collier D, 
    
    
    
    
    
    

    

    Elkin A, 
    
    
    
    
    
    

    

    Farmer A, 
    
    
    
    
    
    

    

    Williamson R, 
    
    
    
    
    
    

    

    McGuffin P, 
    
    
    
    
    
    

    

    Young A, 
    
    
    
    
    
    

    

    Ferrier I, 
    
    
    
    
    
    

    

    Ball S, 
    
    
    
    
    
    

    

    Balmforth A, 
    
    
    
    
    
    

    

    Barrett J, 
    
    
    
    
    
    

    

    Bishop T, 
    
    
    
    
    
    

    

    Iles M, 
    
    
    
    
    
    

    

    Maqbool A, 
    
    
    
    
    
    

    

    Yuldasheva N, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Braund P, 
    
    
    
    
    
    

    

    Dixon R, 
    
    
    
    
    
    

    

    Mangino M, 
    
    
    
    
    
    

    

    Stevens S, 
    
    
    
    
    
    

    

    Thompson J, 
    
    
    
    
    
    

    

    Bredin F, 
    
    
    
    
    
    

    

    Tremelling M, 
    
    
    
    
    
    

    

    Parkes M, 
    
    
    
    
    
    

    

    Drummond H, 
    
    
    
    
    
    

    

    Lees C, 
    
    
    
    
    
    

    

    Nimmo E, 
    
    
    
    
    
    

    

    Satsangi J, 
    
    
    
    
    
    

    

    Fisher S, 
    
    
    
    
    
    

    

    Forbes A, 
    
    
    
    
    
    

    

    Lewis C, 
    
    
    
    
    
    

    

    Onnie C, 
    
    
    
    
    
    

    

    Prescott N, 
    
    
    
    
    
    

    

    Sanderson J, 
    
    
    
    
    
    

    

    Matthew C, 
    
    
    
    
    
    

    

    Barbour J, 
    
    
    
    
    
    

    

    Mohiuddin M, 
    
    
    
    
    
    

    

    Todhunter C, 
    
    
    
    
    
    

    

    Mansfield J, 
    
    
    
    
    
    

    

    Ahmad T, 
    
    
    
    
    
    

    

    Cummings F, 
    
    
    
    
    
    

    

    Jewell D, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Marcano C, 
    
    
    
    
    
    

    

    Burke B, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Gungadoo J, 
    
    
    
    
    
    

    

    Lee K, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Xue M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Barton A, 
    
    
    
    
    
    

    

    Biologics in RA Genetics and Genomics Study Syndicate (BRAGGS) Steering Committee , 
    
    
    
    
    
    

    

    Bruce I, 
    
    
    
    
    
    

    

    Donovan H, 
    
    
    
    
    
    

    

    Eyre S, 
    
    
    
    
    
    

    

    Gilbert P, 
    
    
    
    
    
    

    

    Hilder S, 
    
    
    
    
    
    

    

    Hinks A, 
    
    
    
    
    
    

    

    John S, 
    
    
    
    
    
    

    

    Potter C, 
    
    
    
    
    
    

    

    Silman A, 
    
    
    
    
    
    

    

    Symmons D, 
    
    
    
    
    
    

    

    Thomson W, 
    
    
    
    
    
    

    

    Worthington J, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Widmer B, 
    
    
    
    
    
    

    

    Frayling T, 
    
    
    
    
    
    

    

    Freathy R, 
    
    
    
    
    
    

    

    Lango H, 
    
    
    
    
    
    

    

    Perry J, 
    
    
    
    
    
    

    

    Shields B, 
    
    
    
    
    
    

    

    Weedon M, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hitman G, 
    
    
    
    
    
    

    

    Walker M, 
    
    
    
    
    
    

    

    Elliott K, 
    
    
    
    
    
    

    

    Groves C, 
    
    
    
    
    
    

    

    Lindgren C, 
    
    
    
    
    
    

    

    Rayner N, 
    
    
    
    
    
    

    

    Timpson N, 
    
    
    
    
    
    

    

    Zeggini E, 
    
    
    
    
    
    

    

    Newport M, 
    
    
    
    
    
    

    

    Sirugo G, 
    
    
    
    
    
    

    

    Lyons E, 
    
    
    
    
    
    

    

    Vannberg F, 
    
    
    
    
    
    

    

    Hill A, 
    
    
    
    
    
    

    

    Bradbury L, 
    
    
    
    
    
    

    

    Farrar C, 
    
    
    
    
    
    

    

    Pointon J, 
    
    
    
    
    
    

    

    Wordsworth P, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Franklyn J, 
    
    
    
    
    
    

    

    Heward J, 
    
    
    
    
    
    

    

    Simmonds M, 
    
    
    
    
    
    

    

    Gough S, 
    
    
    
    
    
    

    

    Seal S, 
    
    
    
    
    
    

    

    Breast Cancer Susceptibility Collaboration (UK) , 
    
    
    
    
    
    

    

    Stratton M, 
    
    
    
    
    
    

    

    Rahman N, 
    
    
    
    
    
    

    

    Ban M, 
    
    
    
    
    
    

    

    Goris A, 
    
    
    
    
    
    

    

    Sawcer S, 
    
    
    
    
    
    

    

    Compston A, 
    
    
    
    
    
    

    

    Conway D, 
    
    
    
    
    
    

    

    Jallow M, 
    
    
    
    
    
    

    

    Newport M, 
    
    
    
    
    
    

    

    Sirugo G, 
    
    
    
    
    
    

    

    Rockett K, 
    
    
    
    
    
    

    

    Bumpstead S, 
    
    
    
    
    
    

    

    Chaney A, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Ghori M, 
    
    
    
    
    
    

    

    Gwilliam R, 
    
    
    
    
    
    

    

    Hunt S, 
    
    
    
    
    
    

    

    Inouye M, 
    
    
    
    
    
    

    

    Keniry A, 
    
    
    
    
    
    

    

    King E, 
    
    
    
    
    
    

    

    McGinnis R, 
    
    
    
    
    
    

    

    Potter S, 
    
    
    
    
    
    

    

    Ravindrarajah R, 
    
    
    
    
    
    

    

    Whittaker P, 
    
    
    
    
    
    

    

    Widden C, 
    
    
    
    
    
    

    

    Withers D, 
    
    
    
    
    
    

    

    Cardin N, 
    
    
    
    
    
    

    

    Davison D, 
    
    
    
    
    
    

    

    Ferreira T, 
    
    
    
    
    
    

    

    Pereira-Gale J, 
    
    
    
    
    
    

    

    Hallgrimsdo’ttir I, 
    
    
    
    
    
    

    

    Howie B, 
    
    
    
    
    
    

    

    Su Z, 
    
    
    
    
    
    

    

    Teo Y, 
    
    
    
    
    
    

    

    Vukcevic D, 
    
    
    
    
    
    

    

    Bentley D, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Compston A, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Hall A, 
    
    
    
    
    
    

    

    Hattersley A, 
    
    
    
    
    
    

    

    Hill A, 
    
    
    
    
    
    

    

    Parkes M, 
    
    
    
    
    
    

    

    Pembrey M, 
    
    
    
    
    
    

    

    Stratton M, 
    
    
    
    
    
    

    

    Mitchell S, 
    
    
    
    
    
    

    

    Newby P, 
    
    
    
    
    
    

    

    Brand O, 
    
    
    
    
    
    

    

    Carr-Smith J, 
    
    
    
    
    
    

    

    Pearce S, 
    
    
    
    
    
    

    

    McGinnis R, 
    
    
    
    
    
    

    

    Keniry A, 
    
    
    
    
    
    

    

    Deloukas P, 
    
    
    
    
    
    

    

    Reveille J, 
    
    
    
    
    
    

    

    Zhou X, 
    
    
    
    
    
    

    

    Sims A, 
    
    
    
    
    
    

    

    Dowling A, 
    
    
    
    
    
    

    

    Taylor J, 
    
    
    
    
    
    

    

    Doan T, 
    
    
    
    
    
    

    

    Davis J, 
    
    
    
    
    
    

    

    Savage L, 
    
    
    
    
    
    

    

    Ward M, 
    
    
    
    
    
    

    

    Learch T, 
    
    
    
    
    
    

    

    Weisman M, 
    
    
    
    
    
    

    

    Brown M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat. Genet.</em>
    
    
    2007
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng.2007.17" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2680141" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>We have genotyped 14,436 nonsynonymous SNPs (nsSNPs) and 897 major histocompatibility complex (MHC) tag SNPs from 1,000 independent cases of ankylosing spondylitis (AS), autoimmune thyroid disease (AITD), multiple sclerosis (MS) and breast cancer (BC). Comparing these data against a common control dataset derived from 1,500 randomly selected healthy British individuals, we report initial association and independent replication in a North American sample of two new loci related to ankylosing spondylitis, ARTS1 and IL23R, and confirmation of the previously reported association of AITD with TSHR and FCRL3. These findings, enabled in part by increased statistical power resulting from the expansion of the control reference group to include individuals from the other disease groups, highlight notable new possibilities for autoimmune regulation and suggest that IL23R may be a common susceptibility factor for the major ’seronegative’ diseases.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2007_todd_ng">
  <span class="title">Robust Associations of Four New Chromosome Regions from Genome-Wide Analyses of Type 1 Diabetes.</span>
  <span class="author">
    
    
    
    
    
    

    

    Todd J, 
    
    
    
    
    
    

    

    Walker N, 
    
    
    
    
    
    

    

    Cooper J, 
    
    
    
    
    
    

    

    Smyth D, 
    
    
    
    
    
    

    

    Downes K, 
    
    
    
    
    
    

    

    Plagnol V, 
    
    
    
    
    
    

    

    Bailey R, 
    
    
    
    
    
    

    

    Nejentsev S, 
    
    
    
    
    
    

    

    Field S, 
    
    
    
    
    
    

    

    Payne F, 
    
    
    
    
    
    

    

    Lowe C, 
    
    
    
    
    
    

    

    Szeszko J, 
    
    
    
    
    
    

    

    Hafler J, 
    
    
    
    
    
    

    

    Zeitels L, 
    
    
    
    
    
    

    

    Yang J, 
    
    
    
    
    
    

    

    Vella A, 
    
    
    
    
    
    

    

    Nutland S, 
    
    
    
    
    
    

    

    Stevens H, 
    
    
    
    
    
    

    

    Schuilenburg H, 
    
    
    
    
    
    

    

    <b>Coleman G</b>, 
    
    
    
    
    
    

    

    Maisuria M, 
    
    
    
    
    
    

    

    Meadows W, 
    
    
    
    
    
    

    

    Smink L, 
    
    
    
    
    
    

    

    Healy B, 
    
    
    
    
    
    

    

    <b>Burren O</b>, 
    
    
    
    
    
    

    

    Lam A, 
    
    
    
    
    
    

    

    Ovington N, 
    
    
    
    
    
    

    

    Allen J, 
    
    
    
    
    
    

    

    Adlem E, 
    
    
    
    
    
    

    

    Leung H, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Howson J, 
    
    
    
    
    
    

    

    Guja C, 
    
    
    
    
    
    

    

    Ionescu-TÃ&amp;#0174rgoviste C, 
    
    
    
    
    
    

    

    of Type 1 Diabetes in Finland G, 
    
    
    
    
    
    

    

    Simmonds M, 
    
    
    
    
    
    

    

    Heward J, 
    
    
    
    
    
    

    

    Gough S, 
    
    
    
    
    
    

    

    Dunger D, 
    
    
    
    
    
    

    

    Wellcome Trust Case Control Consortium , 
    
    
    
    
    
    

    

    Wicker L, 
    
    
    
    
    
    

    

    Clayton D. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Nat Genet</em>
    
    
    2007
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/ng2068" target="_blank" class="pure-button doi">DOI</a>
  
  
  <a href="http://europepmc.org/articles/PMC2492393" target="_blank" class="pure-button pmc ">PMC <i class="fa fa-unlock-alt"></i></a>
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The Wellcome Trust Case Control Consortium (WTCCC) primary genome-wide association (GWA) scan on seven diseases, including the multifactorial autoimmune disease type 1 diabetes (T1D), shows associations at P < 5 x 10(-7) between T1D and six chromosome regions: 12q24, 12q13, 16p13, 18p11, 12p13 and 4q27. Here, we attempted to validate these and six other top findings in 4,000 individuals with T1D, 5,000 controls and 2,997 family trios independent of the WTCCC study. We confirmed unequivocally the associations of 12q24, 12q13, 16p13 and 18p11 (P(follow-up)</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2006</h2>
<ol class="bibliography"><li>

<div id="2006_ahg">
  <span class="title">Appropriate Use of Information on Family History of Disease in Recruitment for Linkage Analysis Studies</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Clayton D. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Ann Hum Genet</em>
    
    
    2006
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1111/j.1529-8817.2005.00233.x" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Summary When conducting genetic studies for complex traits, large samples are commonly required to detect any of the number of genes with relatively low effect thought to underly such traits. This is because, in contrast to monogenic diseases, complex traits typically result from a number of different genetic pathways (genetic heterogeneity) and any sample is likely to contain a considerable fraction of sporadic cases (phenocopies). Such samples are time-consuming and costly to recruit and analyse. Methods which might be used to decrease sample size include attempting to select families, with the aim of reducing genetic heterogeneity or phenocopy rate within the sample. Selecting cases with positive family history of disease should reduce the phenocopy rate, and this strategy has been employed in linkage studies of complex disease, although evaluations of such a strategy have been equivocal. This paper shows how identity by descent (IBD) distributions may be calculated for affected relative pairs recruited conditional on the affection status of a third relative. These distributions are then used to calculate expected power in affected sib and half-sib linkage studies when recruitment is conditional on family history of disease. We consider the proxy conditions of recruitment conditional on disease in an affected parent or third sibling with single-locus and additive multilocus genetic models. We show that while such selection strategies can reduce power if disease risk alleles are common and environmental heterogeneity low, under models more likely to underly common complex diseases power will generally be increased, and that this effect is greater as more loci are involved. Though the proxy cases studied are more extreme than a general strategy of asking potential recruits whether they have any family history of disease, these results suggest that conditional recruitment is more generally useful than previous studies have suggested.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2006_ajhg_a">
  <span class="title">Improved Power Offered by a Score Test for Linkage Disequilibrium Mapping of Quantitative-Trait Loci by Selective Genotyping.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Chapman J, 
    
    
    
    
    
    

    

    Clayton D. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Am J Hum Genet</em>
    
    
    2006
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1086/500562" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Selective genotyping is used to increase efficiency in genetic association studies of quantitative traits by genotyping only those individuals who deviate from the population mean. However, selection distorts the conditional distribution of the trait given genotype, and such data sets are usually analyzed using case-control methods, quantitative analysis within selected groups, or a combination of both. We show that Hotelling’s T(2) test, recently proposed for association studies of one or several tagging single-nucleotide polymorphisms in a prospective (i.e., trait given genotype) design, can also be applied to the retrospective (i.e., genotype given trait) selective-genotyping design, and we use simulation to demonstrate its improved power over existing methods.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="2006_ajhg_b">
  <span class="title">Linkage Analysis Using Co-Phenotypes in the BRIGHT Study Reveals Novel Potential Susceptibility Loci for Hypertension</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Xue M, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Marçano A, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    Burke B, 
    
    
    
    
    
    

    

    Gungadoo J, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Mein C, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Am J Hum Genet</em>
    
    
    2006
    
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li>
<li>

<div id="2006_hyp_munroe">
  <span class="title">Increased Support for Linkage of a Novel Locus on Chromosome 5q13 for Essential Hypertension in the British Genetics of Hypertension Study.</span>
  <span class="author">
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Xue M, 
    
    
    
    
    
    

    

    MarÃ\textsectionano A, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Onipinla A, 
    
    
    
    
    
    

    

    Burke B, 
    
    
    
    
    
    

    

    Gungadoo J, 
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    Pembroke J, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Mein C, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Medical Research Council British Genetics of Hypertension Study . 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hypertension</em>
    
    
    2006
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Human hypertension arises from a combination of genetic factors and lifestyle influences. With cardiovascular disease set to become the number 1 cause of death worldwide, it is important to understand the etiologic mechanisms for hypertension, because these might provide new routes to improved treatment. The British Genetics of Hypertension Study has recently published a primary genome screen that identified 4 chromosomal regions of interest. We have now genotyped additional markers to confirm the most promising regions for follow-up studies. Thirty-four additional microsatellites were genotyped in our severely hypertensive affected sibling pair resource (now 1635 families with 2142 affected sibling pairs), leading to a substantial increase in information content in the regions of interest. We found increased support for linkage of chromosome 5q13 to human hypertension (multipoint logarithm of odds=2.50) with 3 adjacent markers yielding single point logarithm of odds scores of 3.22, 2.84, and 2.51. The placement of additional markers on 2q, 6q, and 9q diminished support for linkage in these regions. However, the addition of new data and families identified novel regions of interest on chromosomes 1q and 11q. The 3 positive markers in the chromosome 5 region were also genotyped in 712 distinct parent-offspring trios with the same severe phenotype to replicate linkage and association. Borderline support for replication was found (P=0.07). We found increased evidence for linkage and borderline-significant evidence for association for a hypertension susceptibility locus on chromosome 5q13 that is worthy of detailed fine mapping and assessment of candidate genes.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="bell06">
  <span class="title">Two-Dimensional Genome Scan Identifies Novel Epistatic Loci for Essential Hypertension.</span>
  <span class="author">
    
    
    
    
    
    

    

    Bell J, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Wiltshire S, 
    
    
    
    
    
    

    

    Mein C, 
    
    
    
    
    
    

    

    Pembroke J, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Farrall M. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hum Mol Genet</em>
    
    
    2006
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/hmg/ddl058" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>It is well established that gene interactions influence common human diseases, but to date linkage studies have been constrained to searching for single genes across the genome. We applied a novel approach to uncover significant gene-gene interactions in a systematic two-dimensional (2D) genome scan of essential hypertension. The study cohort comprised 2076 affected sib-pairs and 66 affected half-sib-pairs from the BRItish Genetics of HyperTension study. Extensive simulations were used to establish significance thresholds in the context of 2D genome scans. Our analyses found significant and suggestive evidence for loci on chromosomes 5, 9, 11, 15, 16, and 19 that influence hypertension when gene-gene interactions are taken into account (5q13.1 and 11q22.1, two-locus lod score = 5.72; 5q13.1 and 19q12, two-locus lod score = 5.35; 9q22.3 and 15q12, two-locus lod score = 4.80; 16p12.3 and 16q23, two-locus lod score = 4.50). For each significant and suggestive pairwise interaction, the two-locus genetic model that best fitted the data was determined. Regions that were not detected using single-locus linkage analysis were identified in the 2D scan as contributing significant epistatic effects. This approach has discovered novel loci for hypertension and offers a unique potential to use existing data to uncover novel regions involved in complex human diseases.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="binder06">
  <span class="title">Haplotypes of the Beta-2 Adrenergic Receptor Associate with High Diastolic Blood Pressure in the Caerphilly Prospective Study.</span>
  <span class="author">
    
    
    
    
    
    

    

    Binder A, 
    
    
    
    
    
    

    

    Garcia E, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Gbenga K, 
    
    
    
    
    
    

    

    Ben-Shlomo Y, 
    
    
    
    
    
    

    

    Yarnell J, 
    
    
    
    
    
    

    

    Brown P, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Skrabal F, 
    
    
    
    
    
    

    

    Kotanko P, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>J Hypertens</em>
    
    
    2006
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1097/01.hjh.0000209983.28735.33" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>OBJECTIVES: Current evidence demonstrates that both genetic and environmental factors influence blood pressure. The sympathetic nervous system is a key player in blood pressure control and functional genetic variants of the beta-2 adrenergic receptor (B2AR) have been identified and implicated in the pathogenesis of hypertension. The present study aimed to determine the effects of common haplotypes of the B2AR gene upon blood pressure in the Caerphilly Prospective Study. DESIGN: Two thousand five hundred and twelve men (aged 45-59 years) participated in the study. We selected individuals in the upper (n = 347) and lower (n = 279) quintiles of the diastolic blood pressure distribution fixed at two time points [phase 2 (1984-88) or phase 3 (1989-93)] as cases and controls. METHODS: We analysed two functional polymorphisms (Arg16Gly and Gln27Glu) of B2AR and their haplotypes. RESULTS: We found a higher risk of hypertension in individuals homozygous for the Gln27 compared to those individuals homozygous for Glu27 [odds ratio (OR) = 1.94; 95% confidence interval (CI) = 1.34-2.81; P = 0.001]. Three haplotypes (Gly16Gln27, Gly16Glu27 and Arg16Gln27) were present in both quintile groups. Logistic regression analysis showed that haplotypes with a Gln27 allele (Gly16Gln27 and Arg16Gln27) conferred a significantly higher risk for hypertension than the Gly16Glu27 haplotype (OR = 1.55; 95% CI = 1.11-2.17, OR = 1.37; 95% CI = 1.04-1.81; P = 0.009 and P = 0.027, respectively). However, there was no evidence to support a statistically significant difference in odds ratios for the Gly16Gln27 and Arg16Gln27 haplotypes (P = 0.477), suggesting that it is the Gln27 allele alone, rather than any haplotype, which best explains the association. CONCLUSIONS: In a prospectively studied Caucasian male cohort, high diastolic blood pressure was associated with B2AR haplotypes containing the pro-downregulatory Gln27 variant.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="padmanabhan06">
  <span class="title">Chromosome 2p Shows Significant Linkage to Antihypertensive Response in the BRItish Genetics of HyperTension Study.</span>
  <span class="author">
    
    
    
    
    
    

    

    Padmanabhan S, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Munroe P, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Lathrop M, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Connell J. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hypertension</em>
    
    
    2006
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1161/01.HYP.0000197947.62601.9d" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>There is a lack of consistently linked loci influencing blood pressure and hypertension status, and this may be because of genetic or phenotypic heterogeneity. We hypothesize that stratification of subjects by response to antihypertensive drug groups could be used to stringently define subsets that will have reduced genetic and etiologic heterogeneity, by partitioning contrasting mechanisms of hypertension and, thus, enhancing gene finding. We investigated the British Genetics of Hypertension Study population, which is composed of 2142 severely hypertensive white affected sibling pairs. Nonresponse to antihypertensive therapy was defined as an on-treatment blood pressure of >140/90 mm Hg or a difference between prediagnosis and on-treatment blood pressure of <20 mm Hg. Of the nonresponders, there were 89 sibling pairs (AB) who were both on antihypertensive therapy that inhibit the renin-angiotensin system (angiotensin-converting enzyme inhibitors, angiotensin II type-1 receptor blockers, or beta-blockers), and 76 sibling pairs (CD) who were both on drugs that do not (calcium channel blockers or diuretics). Nonparametric linkage analysis carried out using markers from a 10-cM genome scan and additional "grid tightening" markers showed significant linkage in the AB group on chromosome 2p (logarithm of odds=4.84 at 90.68 Kosambi cM) and suggestive linkage for the CD group on chromosome 10q (logarithm of odds=2.83 at 125.96 Kosambi cM). The AB linkage locus attained genomewide significance after simulation using 10 000 replicates (P=0.005). This locus may contain a gene for the salt-sensitive form of hypertension and/or a pharmacogenetic locus affecting drug response. We have demonstrated for the first time identification of a significant locus by partitioning different pathways of hypertension using drug response.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2005</h2>
<ol class="bibliography"><li>

<div id="newhouse05">
  <span class="title">Haplotypes of the WNK1 Gene Associate with Blood Pressure Variation in a Severely Hypertensive Population from the British Genetics of Hypertension Study.</span>
  <span class="author">
    
    
    
    
    
    

    

    Newhouse S, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Dobson R, 
    
    
    
    
    
    

    

    Mein C, 
    
    
    
    
    
    

    

    Pembroke J, 
    
    
    
    
    
    

    

    Farrall M, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Brown M, 
    
    
    
    
    
    

    

    Samani N, 
    
    
    
    
    
    

    

    Dominiczak A, 
    
    
    
    
    
    

    

    Connell J, 
    
    
    
    
    
    

    

    Webster J, 
    
    
    
    
    
    

    

    Lathrop G, 
    
    
    
    
    
    

    

    Caulfield M, 
    
    
    
    
    
    

    

    Munroe P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Hum Mol Genet</em>
    
    
    2005
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1093/hmg/ddi187" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Mutations in the WNK1 gene cause Gordon’s syndrome, a rare Mendelian form of hypertension. We assessed whether common WNK1 variants might also contribute to essential hypertension (EH), a multifactorial disorder affecting > 25% of the adult population worldwide. A panel of 19 single nucleotide polymorphisms (SNPs) spanning the gene was selected from public databases and was genotyped in 100 white European families to determine the pattern of linkage disequilibrium, haplotype structure and tagging SNPs for the WNK1 locus. Eight tagging SNPs were identified with 90% power to predict common WNK1 haplotypes and SNPs. Family-based association tests were used to test for association with EH and severity of hypertension in 712 severely hypertensive families from the MRC British Genetics of Hypertension study resource. No association was found between WNK1 polymorphisms or haplotypes with hypertension; however, one SNP rs1468326, located 3 kb from the WNK1 promoter, was found to be nominally associated with severity of hypertension, with both systolic blood pressure (BP) (Z = +2.24, P = 0.025) and diastolic BP (Z = +1.99, P = 0.046). We also found nominal support for association of one common WNK1 haplotype with increased systolic BP (Z = +1.91, P = 0.053). This is the first study to perform haplotype association analysis of the WNK1 gene with EH. This finding of association between a SNP near the promoter region and the severity of hypertension suggests that increased expression of WNK1 might contribute to BP variability and susceptibility to EH similar to the mechanism of hypertension observed in Gordon’s syndrome.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2004</h2>
<ol class="bibliography"><li>

<div id="wallace04">
  <span class="title">Linkage Analysis of Susceptibility to Leprosy Type Using an IBD Regression Method.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Fitness J, 
    
    
    
    
    
    

    

    Hennig B, 
    
    
    
    
    
    

    

    Sichali L, 
    
    
    
    
    
    

    

    Mwaungulu L, 
    
    
    
    
    
    

    

    Pönnighaus J, 
    
    
    
    
    
    

    

    Warndorff D, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Fine P, 
    
    
    
    
    
    

    

    Hill A. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genes Immun</em>
    
    
    2004
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1038/sj.gene.6364062" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Leprosy is a chronic disease caused by infection with Mycobacterium leprae, which is manifested across a wide clinical spectrum. There is evidence that susceptibility both to leprosy per se and to the clinical type of leprosy is influenced by host genetic factors. This paper describes the application of an identity by descent regression search for genetic determinants of leprosy type among families from Karonga District, Northern Malawi. Suggestive evidence was found for linkage to leprosy type on chr 21q22 (P<0.001). The methodological implications of the approach and the findings are discussed.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2003</h2>
<ol class="bibliography"><li>

<div id="hickman03">
  <span class="title">Trends in Drug Overdose Deaths in England and Wales 1993-98: Methadone Does Not Kill More People than Heroin.</span>
  <span class="author">
    
    
    
    
    
    

    

    Hickman M, 
    
    
    
    
    
    

    

    Madden P, 
    
    
    
    
    
    

    

    Henry J, 
    
    
    
    
    
    

    

    Baker A, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Wakefield J, 
    
    
    
    
    
    

    

    Stimson G, 
    
    
    
    
    
    

    

    Elliott P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Addiction</em>
    
    
    2003
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>AIMS: To test the hypothesis that methadone is responsible for a greater increase in overdose deaths than heroin, and causes proportionally more overdose deaths than heroin at weekends. DESIGN AND SETTING: Multivariate analysis of 3961 death certificates mentioning heroin, morphine and/or methadone held on the Office for National Statistics drug-related poisoning mortality database from 1993 to 1998 in England and Wales. MEASUREMENTS: Percentage increase in deaths by year by drug, odds ratio (OR) of dying at the weekend from methadone-related overdose compared to dying from heroin/morphine overdose. FINDINGS: From 1993 to 1998, annual opiate overdose deaths increased from 378 to 909. There was a 24.7% (95% confidence interval (CI) 22-28%) yearly increase in heroin deaths compared to 9.4% (95% CI 6-13%) for methadone only. This difference was significant (P < 0.001 by test of interaction) after adjustment for sex, age group, polydrug use, area of residence and underlying cause of death. The largest number of deaths occurred on Saturday (673). The OR of death from methadone overdose on Saturday and Sunday was 1.48 (95% CI 1.29-1.71) for methadone-only deaths compared to dying from heroin/morphine at the weekend after adjustment for other covariates, but the OR was not significant (1.09, 95% CI 0.95-1.25) if the weekend was defined as Friday and Saturday. CONCLUSIONS: There was no evidence that the threefold increase in deaths over time was due to methadone. There was equivocal support only for the hypothesis that there was an excess of deaths from methadone at weekends. Increased interventions to prevent overdose among injectors in England and Wales are long overdue.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="wallace03">
  <span class="title">Estimating the Relative Recurrence Risk Ratio Using a Global Cross-Ratio Model.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Clayton D. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Genet Epidemiol</em>
    
    
    2003
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  <a href="http://doi.org/10.1002/gepi.10270" target="_blank" class="pure-button doi">DOI</a>
  
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>The relative recurrence risk ratio lambdaR (and particularly the sibling recurrence risk ratio, lambdaS) is often of interest to those wanting to quantify the genetic contribution towards risk of disease or to discriminate between different genetic models. However, estimating lambdaR for complex diseases for which genetic and environmental risk factors are both involved is not straightforward. Ignoring environmental factors may lead to inflated estimates of lambdaR. We present a marginal model which uses a copula function to model the association in cumulative incidence rates between pairs of relatives. This model is applicable to present-state data and allows estimation of risk of disease in a pair of relatives (and hence lambdaR), given measured environmental covariates. We apply the model to leprosy among sibling pairs from the Karonga district, Malawi. If risk factors are ignored, the apparent lambdaS in this population is over 3. Accounting for known nongenetic risk factors reduces it to just under 2.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li>
<li>

<div id="wallace03a">
  <span class="title">Estimating the Relative Recurrence Risk Ratio for Leprosy in Karonga District, Malawi.</span>
  <span class="author">
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Clayton D, 
    
    
    
    
    
    

    

    Fine P. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Lepr Rev</em>
    
    
    2003
    
  <!--  -->
  <!--   [<a class="abstract">Abs</a>] -->
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
  <!-- <span class="abstract hidden"> -->
  <!--   <p>Leprosy is a chronic disease caused by infection with Mycobacterium leprae. Susceptibility to leprosy is influenced by both genetic and non-genetic factors and the disease is known to cluster in families. One measure of genetic effect is the relative recurrence risk ratio, lambdaR. Estimates of this parameter can be inflated if environmental risk factors which also cluster in families, such as household contact, are not properly accounted for. We present the results of fitting a cross ratio model that allows estimation of the odds ratio of disease conditional on disease or no disease in a given relative, given measured covariates. From this model we can predict fitted values for lambdaR that represent the familial risk not accounted for by other covariates including observed household contact. If all covariates could be measured, this would be the ’genetic relative risk ratio’. We find lambdaR > 1 for all relative pairs except grandparent-grandchild, and lambdaR > 2 for siblings. Though not in itself evidence for a strong genetic susceptibility to leprosy, this result is consistent with much other evidence which suggests susceptibility to leprosy is under the control of many factors, the strongest of which may be non-genetic, with host genetics playing a small but significant role.</p> -->
  <!-- </span> -->
  <!--  -->
</div>
</li></ol>
<h2 class="bibliography">2001</h2>
<ol class="bibliography"><li>

<div id="new01">
  <span class="title">Measuring Differences in the Provision of Diabetes Care</span>
  <span class="author">
    
    
    
    
    
    

    

    New J, 
    
    
    
    
    
    

    

    Roxburgh M, 
    
    
    
    
    
    

    

    Vaughan N, 
    
    
    
    
    
    

    

    <b>Wallace C</b>, 
    
    
    
    
    
    

    

    Elliott P, 
    
    
    
    
    
    

    

    Young R. 
    
  </span>
  
  <div class="jblocks">
    
    <em>Diabetes Nutrition &amp; Metabolism</em>
    
    
    2001
    
  <!--  -->
   
  
 
  
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  <!--  -->
  </div>

  <!-- Hidden abstract block -->
  <!--  -->
</div>
</li></ol>

  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <!-- <h2 class="footer-heading">Wallace group</h2> -->

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            
              Chris Wallace
            
<a href="https://orcid.org/0000-0001-9755-1703"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon"></a>
</li>
	  <li><a href="/contact/">Contact details</a>
            <!--  -->
            <!-- <li><a href="mailto:cew54-at-cam-dot-ac-dot-uk">cew54-at-cam-dot-ac-dot-uk</a></li> -->
            <!--  -->
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/chr1swallace"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/chr1swallace"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

        </ul>
      </div>

      <div class="footer-col footer-col-1">
        <p><b>Statistics</b> applied to <b>genomics</b> to understand <b>autoimmune diseases</b></p>
	<!-- &lt;b&gt;Statistics&lt;/b&gt; applied to genomics to understand autoimmune diseases</p> -->
      </div>
    </div>

  </div>

</footer>

    <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-134236872-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-134236872-1');
</script>



  </body>

</html>
