<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Data</title>
  <meta name="description" content="Statistics applied to genomics to understand autoimmune diseases">

  <link rel="stylesheet" href="/assets/main.css">
  <!-- <link rel="stylesheet" href="/assets/css/style.css"> -->
  <link rel="stylesheet" href="/assets/css/beautifuljekyll.css">
  <link rel="stylesheet" href="/assets/css/style.css">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
￼
  <link rel="canonical" href="http://chr1swallace.github.io/data/">
  <link rel="alternate" type="application/rss+xml" title="Wallace group" href="/feed.xml">
  
  
</head>


  <body>

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Wallace group</a>
  
    
      <nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
           
	  <a class="page-link" href="/people/">People</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/research/">Research</a>
	  
            
          
            
            
           
	  <a class="page-link" style="font-weight:bold;" href="/data/">Data</a>
	  <!-- <a class="page-link" style="color:#099;font-weight:bold;" href="/data/">Data</a> -->
	  
            
          
            
            
           
	  <a class="page-link" href="/code/">Software</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/papers/">Papers</a>
	  
            
          
            
            
           
	  <a class="page-link" href="/talks/">Presentations</a>
	  
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
            
            
          
        </div>
      </nav>
    
  </div>
</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <!-- <header class="post-header"> -->
  <!--   <h1 class="post-title">Data</h1> -->
  <!-- </header> -->

  <div class="post-content">
    <p>This page holds information about datasets and code used in our papers.  Our intention
is to make the data and code we have used accessible.  If anything is
unclear, please <a href="https://github.com/chr1swallace/chr1swallace.github.io/issues">report it as an issue</a> so I (and you) can track it.  If
you really don’t want to use github, as a last resort, email
=cew54= at =cam.ac.uk=.</p>

<!-- # Code and data organised by papers -->

<!-- ## 2015 -->

<!-- *Paper*: } -->

<!-- *Code*: Uses the R package [GUESSFM](https://github.com/chr1swallace/GUESSFM).  For detailed pipeline of steps to run GUESSFM, see the [vignettes](https://github.com/chr1swallace/GUESSFM/tree/master/vignettes). -->

<!-- *Data*: [ | [T1D ImmunoChip Data) -->

<!-- *Paper*: } -->

<!-- *Data*: [ | [[MS ImmunoChip Data](T1D ImmunoChip Data) | [ | [[T1D ImmunoChip Data](RA ImmunoChip Data) -->

<!-- Complete results are available in the [Supplementary Tables](http://www.nature.com/ng/journal/v47/n7/full/ng.3330.html#supplementary-information). -->

<!-- *Paper*: } -->

<!-- *Data*: [ | [[T1D gene expression](BABYDIET gene expression) | [ -->

<!-- *Paper*: } -->

<!-- *Code*: R code used to perform -->
<!--   numerical calculations exploring the appropriate p value thresholds -->
<!--   for calling genetic association in one disease given genomewide -->
<!--   significant genetic association at the same SNP in a related -->
<!--   disease. https://github.com/chr1swallace/condpp.   -->

<!-- *Results*: Detailed results are curated at the [[http://www.immunobase.org/poster/type-1-diabetes-immunochip-study-onengut-gumuscu/][ImmunoBase poster page](Asthma gene expression), including interactive figures, T1D regions studied and identified, and a full list of the T1D credible SNP set. -->

<!-- *Data*: [ -->

<!-- *Paper*: } -->

<!-- *Code*: Uses the R package [[https://github.com/chr1swallace/coloc/][coloc](T1D ImmunoChip Data) (available also from [CRAN](http://cran.r-project.org/web/packages/coloc/index.html)). -->

<!-- *Data*: [ | [[Fairfax eQTL data](ImmunoChip summary statistics) | [ -->

<!-- *Paper*: } -->

<!-- *Code*: R script for fine mapping https://github.com/chr1swallace/finemap-psa  -->

<!-- *Data*: RA p values required for input were downloaded from [[http://www.immunobase.org][ImmunoBase](Cardiogenics eQTL data). -->

<!-- *Paper*: } -->

<!-- *Code*: James' code is on https://github.com/jamesliley/cFDR-common-controls. -->

<!-- *Data*: GWAS summary statistics (p values) were downloaded from [. Also uses [[T1D ImmunoChip Data](Immunobase) for simulations to examine properties of the method. -->

<!-- ## 2014 -->
<!-- *Paper*: } -->

<!-- *Code*: Olly's pipeline is at [. -->

<!-- *Data*: We used data from [[Cusanovich][Cusanovich et al](http://github.com/ollyburren/vseams)'s transcription factor knockdowns.  In fact, I think our paper was submitted only a month after theirs was published, because the preprint was shared on arXiv (and indexed on [Haldane's Sieve](http://haldanessieve.org/2013/10/22/the-functional-consequences-of-variation-in-transcription-factor-binding), which is where we saw it), and the authors readily shared their data pre-publication.  Open science working at its best. -->

<!-- We also used [. -->

<h2 id="gwas">GWAS</h2>

<h1 id="atd-immunochip">ATD ImmunoChip</h1>

<p>Need to be requested from the original study authors <a href="https://www.ncbi.nlm.nih.gov/pubmed/22922229">https://www.ncbi.nlm.nih.gov/pubmed/22922229</a></p>

<h1 id="coeliac-immunochip">Coeliac ImmunoChip</h1>

<p>Primary analysis is by <a href="http://www.nature.com/ng/journal/v43/n12/full/ng.998.html">Trynka et al</a>.  Genotype data  is hosted by the European Bioinformatics Institute, under accession number <a href="http://www.ebi.ac.uk/ega/studies/EGAS00000000053">EGAS00000000053</a>.</p>

<h1 id="ms-immunochip">MS ImmunoChip</h1>

<p>MS ImmunoChip data was accessed through application to the <a href="http://www.imsgenetics.org/">International Multiple Sclerosis Genetic Consortium (IMSGC)</a>.  Primary analysis of these data is presented in  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882906/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882906/</a> and the dbGaP accession number for the  US cohort is phs000357.v2.p2.</p>

<h1 id="ra-immunochip">RA ImmunoChip</h1>

<p>Primary analysis was by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882906/">Eyre et al</a> and data were accessed through a request to the authors.</p>

<h1 id="t1d-immunochip">T1D ImmunoChip</h1>

<p>is available from <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v3.p2">dbGaP</a></p>

<h1 id="t1d-gwas">T1D GWAS</h1>

<p>2000 T1D samples were genotyped as part of the WTCCC (and controls) - data access  is described  <a href="https://www.wtccc.org.uk/info/access_to_data_samples.html">https://www.wtccc.org.uk/info/access_to_data_samples.html</a>. An additional 4000 cases were genotyped by the T1DGC, available at <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v3.p2">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000180.v3.p2</a></p>

<p>Autoantibody data on T1D subjects is available by contacting study authors from <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002216">http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002216</a>.</p>

<h1 id="immunochip-summary-statistics">ImmunoChip summary statistics</h1>

<p><em>ImmunoBase</em>: ImmunoChip summary GWAS statistics from multiple studies <em>used to be</em> nicely curated and available from <a href="http://www.immunobase.org">ImmunoBase</a> if you first register (free) then go to =Data..downloads= on the right, then click through to =data/immunochip/association_stats=.  I’m not sure how to access it now.</p>

<h2 id="expression-data">Expression data</h2>

<h1 id="fairfax-eqtl-data">Fairfax eQTL data</h1>

<p>Ben Fairfax has published eQTL papers on sorted cells (monocytes, stimulated monocytes, B cells) from close to 300 samples.  The expression data are freely available, but the genotype data require a data access agreement, available as detailed in either paper.</p>

<p><em>Fairfax et al. (2012) Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. Nat. Genet. 44, 502-510</em></p>

<p>gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-945</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>.</p>

<p><em>Fairfax et al. (2014) Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression. Science 343, 1246949</em></p>

<p>gene expression data: <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232">http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2232</a>; genotyping data: <a href="http://www.ebi.ac.uk/ega/EGAD00010000144">http://www.ebi.ac.uk/ega/EGAD00010000144</a>, <a href="http://www.ebi.ac.uk/ega/EGAD00010000520">http://www.ebi.ac.uk/ega/EGAD00010000520</a>.</p>

<h1 id="cusanovich-et-als-transcription-factor-knockdown">Cusanovich et al’s transcription factor knockdown</h1>

<p>See <a href="http://europepmc.org/abstract/MED/24603674">http://europepmc.org/abstract/MED/24603674</a> for the paper.  This
dataset has a special place in my heart, as we requested it after
Cusanovich posted a preprint and it was shared quickly and freely.  As
a result, I /think/ our paper using this data was actually preprinted and submitted
before the Cusanovich paper was formally published in a journal.  A
true example of preprint publishing furthering research.</p>

<h1 id="babydiet-gene-expression">BABYDIET gene expression</h1>

<p>PBMC gene expression from longitudinally sampled children at risk of type 1 diabetes, described by <a href="http://www.ncbi.nlm.nih.gov/pubmed/24561305">Ferreira et al.</a></p>

<p>http://www.ebi.ac.uk/arrayexpress, accession no. E-MTAB-1724</p>

<h1 id="t1d-gene-expression">T1D gene expression</h1>

<p>The T1D PBMCs were collected as part of the Genetic Resource
Investigating Diabetes (GRID) cohort collection
(<a href="http://www.childhood-diabetes.org.uk/grid.shtml">http://www.childhood-diabetes.org.uk/grid.shtml</a>).  Expression data at
<a href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a>, accession no. E-MTAB-1724</p>

<h1 id="asthma-gene-expression">Asthma gene expression</h1>

<p>Asthma cohort described by <a href="http://www.ncbi.nlm.nih.gov/pubmed/21779351">Bjornsdottir et al. (2011)</a>. Raw and normalized data are deposited with ArrayExpress ( <a href="http://www.ebi.ac.uk/arrayexpress/">http://www.ebi.ac.uk/arrayexpress/</a>, E-GEOD-19301).</p>

<h1 id="cardiogenics-eqtl-data">Cardiogenics eQTL data</h1>

<p><em>Garnier S et al. (2013) Genome-wide haplotype analysis of cis expression quantitative trait Loci in monocytes <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003240">PLoS Genet. 9(1):e1003240</a></em></p>

<p>Monocytes eQTL study available from EGA: <a href="https://www.ebi.ac.uk/ega/studies/EGAS00001000411">https://www.ebi.ac.uk/ega/studies/EGAS00001000411</a></p>


  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <!-- <h2 class="footer-heading">Wallace group</h2> -->

    <div class="footer-col-wrapper">
    <!-- <div class="container-md beautiful-jekyll-footer"> -->
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            <!--  -->
              Chris Wallace
            <!--  -->
<a href="https://orcid.org/0000-0001-9755-1703"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon"></a>
</li>
	  <li><a href="/contact/">Contact details</a>
            <!--  -->
            <!-- <li><a href="mailto:cew54-at-cam-dot-ac-dot-uk">cew54-at-cam-dot-ac-dot-uk</a></li> -->
            <!--  -->
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/chr1swallace"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/chr1swallace"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">chr1swallace</span></a>

          </li>
          

        </ul>
      </div>

      <div class="footer-col footer-col-1">
        <p><b>Statistics</b> applied to <b>genomics</b> to understand <b>autoimmune diseases</b></p>
	<!-- &lt;b&gt;Statistics&lt;/b&gt; applied to genomics to understand autoimmune diseases</p> -->
      </div>
    </div>

  </div>

</footer>

    <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-134236872-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-134236872-1');
</script>



  </body>

</html>
