var store = [{
        "title": "Chris talking at MRC BSU Armitage Workshop",
        "excerpt":"The MRC BSU Armitage Workshop is an annual event.  This year, the invited lecture was by Barbara Engelhardt and one of the work shop talks was given by Chris Wallace Needles, haystacks and genetic causes of disease. See slides. ","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2018/11/armitage"
      },{
        "title": "simGWAS paper published",
        "excerpt":"simGWAS paper published in Bioinformatics simGWAS allows fast simulation of GWAS summary statistics for large case control studies. See Vignette for usage ","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2018/11/simgwas"
      },{
        "title": "Martin Keleman joins the group",
        "excerpt":"Martin Keleman is a PhD student on the Mathematical Genomics and Medicine programme, supervised by Carl Anderson at the Wellcome Trust Sanger Institute. His research focuses on developing practical machine learning methods for GWAS data, that will improve our understanding of how genetic differences influence variation in complex disease risk....","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/01/mk"
      },{
        "title": "Elena, Olly and Chris all presenting at Immunogenomics of Disease",
        "excerpt":"The Immunogenomics of Disease is happening this week in Hinxton. We will be there in force, with Olly Burren talking about dimension reduction to study autoimmune disease genetics, Elena presenting her work on allele specific expression, and Chris talking about joint fine mapping of immune mediated disease. ","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/02/immg"
      },{
        "title": "New preprint on fine mapping multiple diseases",
        "excerpt":"Sharing information between related diseases using Bayesian joint fine mapping increases accuracy and identifies novel associations in six immune mediated diseases Jennifer Asimit, Daniel B Rainbow, Mary Fortune, Nastasiya F Grinberg, Linda S Wicker, Chris Wallace Thousands of genetic variants have been associated with human disease risk, but linkage disequilibrium...","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/02/mfm"
      },{
        "title": "Elena and Anna speak at MASAMB",
        "excerpt":"Elena Vigorito and Anna Hutchinson both presented their work at MASAMB 2019 today. There is increasing evidence that gene regulation differs between patients with active disease and healthy controls, but eQTL studies still focus in the majority on healthy individuals. Elena spoke on ‚ÄúMapping cis-eQTL from RNA-seq data with no...","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/04/masamb"
      },{
        "title": "Anna and Chris speak at Quantitative Genomics 2019",
        "excerpt":"Quantitative Genomics is a fantastic PhD student organised annual conference that has been running for a few years now. This year, Chris Wallace gave a keynote talk describing our work leveraging large GWAS datasets to understand the genetics of rare immune-mediated diseases for which standard GWAS approaches require a prohibitive...","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/06/quantgen"
      },{
        "title": "New publication Stochastic search and joint fine-mapping",
        "excerpt":"Work on improving fine mapping by borrowing information from related diseases now published. /Abstract/ Thousands of genetic variants are associated with human disease risk, but linkage disequilibrium (LD) hinders fine-mapping the causal variants. Both lack of power, and joint tagging of two or more distinct causal variants by a single...","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/08/mfm"
      },{
        "title": "SimGWAS now on CRAN",
        "excerpt":"The simGWAS package, authored to accompany our GWAS simulation paper (1), is now on CRAN. ","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/08/2018-08-26-simgwas"
      },{
        "title": "new preprint on GWAS fine mapping credible sets",
        "excerpt":"Our paper is now avaliable! Credible set coverage is higher than you think and we‚Äôve developed a statistical method and user-friendly software to correct for this! https://t.co/FKT9BQxdI6 @chr1swallace https://t.co/H00FW00ygJ &mdash; Anna Hutchinson (@AnnaHutchinson6) September 26, 2019","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2019/10/credsets"
      },{
        "title": "A genetic basis for immune mediated diseases",
        "excerpt":"By reducing the genome to a radically few 13 immune-focused dimensions, we perform comparative study of the genetics of rare and common immune-mediated diseases, and identify biological co-factors for specific disease clusters, work led by @ollyburren If you want to explore your GWAS summary data in our immune-mediated disease basis,...","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2020/02/imd-basis"
      },{
        "title": "Improving the coverage of credible sets in Bayesian genetic fine-mapping",
        "excerpt":"Our work on credible set coverage in genetic fine-mapping is now available! @PLOSCompBiol https://t.co/nVOiIWUlYh We identify systematic bias, discuss the source of the bias and provide a method with user-friendly software to improve coverage @chr1swallace @MRC_BSU @Cambridge_Uni &mdash; Anna Hutchinson (@AnnaHutchinson6) April 14, 2020","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2020/04/credset"
      },{
        "title": "Eliciting priors and relaxing the single causal variant assumption in colocalisation analyses",
        "excerpt":"The coloc package has had a new release, incorporating new functionality. https://t.co/Q6GasFdt9vWhat are the developments? &mdash; Chris Wallace üá™üá∫ üè≥Ô∏è‚Äçüåà (@chr1swallace) April 28, 2020","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2020/05/coloc-v4"
      },{
        "title": "New preprint on consensus clustering for Bayesian mixture models",
        "excerpt":"Our new preprint is out! By applying consensus clustering to Bayesian clustering models, we sidestep convergence and runtime issues which enables integrative clustering for multiple &#39;omics datasets @chr1swallace @pauldwkirk @MRC_BSU https://t.co/2n7p8dfobv &mdash; Stephen Coleman (@stcolema1) December 19, 2020","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2020/12/consensus-clustering"
      },{
        "title": "New preprint evaluation of biclustering in gene expression data",
        "excerpt":"New preprint by Kath Nicholls carefully compares 4 sets of biclustering algorithms according to their ability to identify structure in gene expression data. Using real data to evaluate biclusters helps go beyond gene-only or sample-only evaluation. https://t.co/cRNNZqUSnl &mdash; Chris Wallace üá™üá∫ üè≥Ô∏è‚Äçüåà (@chr1swallace) December 19, 2020","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2020/12/biclustering-preprint"
      },{
        "title": "Our guide to Capture Hi-C analysis using Chicago and companion tools",
        "excerpt":"Our guide to Capture Hi-C analysis using Chicago and companion tools is finally out.Lots of new tips, tricks and scripts!@MalyValerie @HelenR_J @paulafpritchett @MonDllRs C.Eijsbouts @williamrorchard @StevenWingett @chr1swallace @jonathancairns https://t.co/HE777kYyyR &mdash; Mikhail Spivakov (@mikhailspivakov) August 9, 2021","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2021/09/chicago-and-friends"
      },{
        "title": "Update to coloc",
        "excerpt":"If you like coloc, but hate making a single causal variant assumption, check out this update now published https://t.co/EWdL7yF6Ul with the package available from CRAN https://t.co/jJ5EWB3DDI and vignette https://t.co/sWouPEk5c2. https://t.co/SmjSRDdXTg &mdash; Chris Wallace üá™üá∫ üè≥Ô∏è‚Äçüåà (@chr1swallace) September 29, 2021","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2021/09/update-to-coloc"
      },{
        "title": "new BSU director",
        "excerpt":"We are delighted to announce that Professor John Whittaker will become our new Director‚ÄúI couldn‚Äôt be more excited to be taking up this position at this time.&quot;Read more: https://t.co/xLux86oKZA@The_MRC @Cambridge_Uni #biostatistics &mdash; MRC Biostatistics Unit (@MRC_BSU) September 30, 2021","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2021/09/new-bsu-director"
      },{
        "title": "new binary cFDR preprint",
        "excerpt":"We have now extended the cFDR framework for binary covariates (e.g. binary functional genomic data) and introduce an all-encompassing user-friendly R package to implement both Flexible cFDR and Binary cFDR! See preprint here: https://t.co/aFw9o8g09y @chr1swallace @MRC_BSU ü•≥ &mdash; Anna Hutchinson (@AnnaHutchinson6) October 25, 2021","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2021/10/new-binary-cfdr-preprint"
      },{
        "title": "Hutchinson et al, flexible cFDR published in PLoS Genetics",
        "excerpt":"Absolutely thrilled to see Flexible cFDR published in PLOS Genetics this morning - my third first author paper during my PhD ü•≥üçæhttps://t.co/U7ok70h3g1 &mdash; Anna Hutchinson (@AnnaHutchinson6) October 21, 2021","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2021/10/hutchinson-et-al-flexible-cfdr-published-in-plos-genetics"
      },{
        "title": "Hernandez et al, flashfm fine mapping published in Nat Comms",
        "excerpt":"This study introduces flasfm, a method for general quantitative multi-trait fine-mapping that  allows for related individuals and missing trait measurements.  https://t.co/PhX4u0Dwnp @JennAsimit @NBanadik @chr1swallace @InesBarroso4 @JanaSoenksen @MRC_BSU @The_MRC @ExeterMed &mdash; Nature Communications (@NatureComms) October 25, 2021","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2021/10/hernandez-et-al-flashfm-fine-mapping-published-in-nat-comms"
      },{
        "title": "post-doc position available",
        "excerpt":"3  year statistical post-doc job, pls retweet!Join our group in Cambridge UK using  genetics and genomics to understand the human immune system and its dysregulation in relation to diseasehttps://t.co/x4X0gEW41x closes 24 Nov &mdash; Chris Wallace üá™üá∫ üè≥Ô∏è‚Äçüåà (@chr1swallace) October 27, 2021","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2021/10/post-doc-position-available"
      },{
        "title": "EMGM2022 abstract deadline approaching",
        "excerpt":"#EMGM2022 is a friendly conference in maths, genetics, statistics. Lots of ECR and more senior speakers. What are you working on? Now is the time to write your abstract!Hoping to hold a hybrid meeting (f2f + remote) in Cambridge this year, with contingency in case not possible. https://t.co/g4hdXHKQCh &mdash; Chris...","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2022/01/emgm2022-abstract-deadline-approaching"
      },{
        "title": "Group members give talks at the European Mathematical Genetics Meeting",
        "excerpt":"The 50th European Mathematical Genetics Meeting  was held in Cambridge and three group members, Guillermo Reales, Ichcha Manipur, and Alan O‚ÄôCallaghan gave talks. See the talks here. ","categories": [],
        "tags": [],
        "url": "http://chr1swallace.github.io/blog/2022/05/emgm-talks"
      },]
